
<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Research Directions - Zitnik Lab</title>
    <link rel="stylesheet" href="/assets/css/app.css">
    <link rel="shortcut icon" type="image/png"
           href="/favicon.png" 
    />
    <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>
    <!-- Begin Jekyll SEO tag v2.6.1 -->
<title>Research Directions | Zitnik Lab</title>
<meta name="generator" content="Jekyll v3.8.6" />
<meta property="og:title" content="Research Directions" />
<meta name="author" content="Marinka Zitnik" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Artificial Intelligence (AI), Medicine, Science, and Drug Discovery at Harvard" />
<meta property="og:description" content="Artificial Intelligence (AI), Medicine, Science, and Drug Discovery at Harvard" />
<link rel="canonical" href="https://zitniklab.hms.harvard.edu/research/" />
<meta property="og:url" content="https://zitniklab.hms.harvard.edu/research/" />
<meta property="og:site_name" content="Zitnik Lab" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="Research Directions" />
<meta name="twitter:site" content="@marinkazitnik" />
<meta name="twitter:creator" content="@Marinka Zitnik" />
<script type="application/ld+json">
{"url":"https://zitniklab.hms.harvard.edu/research/","author":{"@type":"Person","name":"Marinka Zitnik"},"headline":"Research Directions","description":"Artificial Intelligence (AI), Medicine, Science, and Drug Discovery at Harvard","@type":"WebPage","@context":"https://schema.org"}</script>
<!-- End Jekyll SEO tag -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-162129505-1"></script>
<script>
  window['ga-disable-UA-162129505-1'] = window.doNotTrack === "1" || navigator.doNotTrack === "1" || navigator.doNotTrack === "yes" || navigator.msDoNotTrack === "1";
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());
  gtag('config', 'UA-162129505-1');
</script><!-- head scripts --></head>

  <body>
    
<nav class="navbar is-primary" >
    <div class="container">
        <div class="navbar-brand">
            <a href="/" class="navbar-item"><b>
                Zitnik Lab
            </b></a>
            <a role="button" class="navbar-burger burger" aria-label="menu" aria-expanded="false" data-target="navMenu">
                <span aria-hidden="true"></span>
                <span aria-hidden="true"></span>
                <span aria-hidden="true"></span>
            </a>
        </div>
        <div class="navbar-menu" id="navMenu">
            <div class="navbar-start">
<!--                <a href="/" class="navbar-item "><b>Home</b></a>-->
                
                
                    
                    <div class="navbar-item has-dropdown is-hoverable">
                        <a href="/#" class="navbar-link "><b>About</b></a>
                        <div class="navbar-dropdown">
                            
                            <a href="/bio/" class="navbar-item "><b>Bio</b></a>
                            
                            <a href="/contact/" class="navbar-item "><b>Contact</b></a>
                            
                            <a href="/talks/" class="navbar-item "><b>Recent Talks</b></a>
                            
                        </div>
                    </div>
                    
                
                    
                <a href="/research/" class="navbar-item is-active"><b>Research</b></a>
                    
                
                    
                <a href="/publications/" class="navbar-item "><b>Publications</b></a>
                    
                
                    
                <a href="/people/" class="navbar-item "><b>Members</b></a>
                    
                
                    
                <a href="/meetings/" class="navbar-item "><b>Conferences</b></a>
                    
                
                    
                <a href="/DMAI/" class="navbar-item "><b>DMAI</b></a>
                    
                
                    
                <a href="/data/" class="navbar-item "><b>Datasets</b></a>
                    
                
                    
                <a href="/software/" class="navbar-item "><b>ML Tools</b></a>
                    
                
                    
                <a href="https://zitniklab.hms.harvard.edu/TDC/" class="navbar-item "><b>TDC</b></a>
                    
                
                    
                <a href="/news/" class="navbar-item "><b>News</b></a>
                    
                
                    
                <a href="/jobs/" class="navbar-item "><b>Join Us</b></a>
                    
                
                
            </div>
        </div>
    </div>
</nav>

    
        <section class="hero  is-medium  is-bold is-primary"  style="background: url('/hero.jpg') no-repeat center center; background-size: cover;" >
    <div class="hero-body">
        <div class="container">
            <p class="title is-2">Research Directions</p>
            <p class="subtitle is-3"></p>
            
        </div>
    </div>
</section>
    
    


    <section class="section">
        <div class="container">
            <div class="columns">
                
                <div class="column is-12">
                    
                    
                    
<div class="content">
    <p>Our research develops AI and machine learning methods to address grand challenges in science and medicine. In doing so, we:</p>
<ol>
  <li>Invent ways to infuse knowledge &amp; structure into AI models to reduce uncertainty and enable generalization to new scenarios not seen during training.</li>
  <li>Develop methods that produce actionable &amp; trustworthy representations and can reason over massive datasets.</li>
  <li>Translate machine learning research into innovative applications.</li>
</ol>

<p>Our research strategy is to create <strong>foundational models (e.g., pre-trained models, self-supervised models, general-purpose models, multi-purpose models, and multi-modal models)</strong> trained on broad data at scale and can be adapted to a wide range of downstream tasks. This research opens up new avenues for understanding network and disease biology, developing safe &amp; effective medicines, and can impact the way predictive modeling is performed today at the fundamental level.</p>

<div align="center">
<img style="margin-bottom:20px;margin-top:20px;width:80%" src="/img/our-approach.png" />
</div>

<hr />

<h2 id="knowledge-guided-ai-fusing-biomedical-knowledge-and-patient-data">Knowledge-guided AI: Fusing biomedical knowledge and patient data</h2>

<p><em>Methods: Knowledge graphs, multi-modal learning, foundational models</em></p>

<p>Soon, the state of a person will be characterized with increasing precision by incorporating data modalities like genetic code, behaviors, therapeutics, nutrients, and the environment—the challenge is how to computationally operationalize these data to make them amenable to decision making.</p>

<p>Further, data are of many different types, including experimental readouts, curated annotations, and metadata—no single data type can capture all the factors necessary to understand a phenomenon such as a disease. These high-dimensional datasets lead to far more complex characterizations than are currently in use, requiring fundamentally new approaches.</p>

<p>To this end, we invent methods to fuse rich, heterogeneous data into knowledge graphs to bridge the divide between biomedical research and patient data. This approach combines data in their broadest sense to reduce redundancy, resolve contradictory observations, and model uncertainty. Using our methods, we could, for example, construct one of the largest biological networks ever created, with over 2.3 billion edges and more than 2,000 modes.</p>

<p>Ultimately, we believe that science and medicine are among the most exciting areas for machine learning, with many hard problems and applications of immense impact. For this reason, we build high-quality open-source data repositories whenever possible to bring biomedical data closer to other scientists who can now readily use these datasets in their research.</p>

<div align="center">
<img style="margin-top:20px;margin-bottom:20px;width:100%" src="/img/research-fuse.png" />
</div>

<h4 id="example-related-publications">Example related publications:</h4>

<p><a href="https://doi.org/10.1016/j.inffus.2018.09.012">Machine Learning for Integrating Data in Biology and Medicine: Principles, Practice, and Opportunities</a><br />
<a href="https://www.nature.com/articles/s41467-017-01642-w">A Comprehensive Structural, Biochemical and Biological Profiling of the Human NUDIX Hydrolase Family</a><br />
<a href="http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004552">Gene Prioritization by Compressive Data Fusion and Chaining</a><br />
<a href="http://bioinformatics.oxfordjournals.org/content/31/12/i230.full">Gene Network Inference by Fusing Data from Diverse Distributions</a><br />
<a href="http://dx.doi.org/10.1109/TPAMI.2014.2343973">Data Fusion by Matrix Factorization</a></p>

<hr />

<h2 id="graph-ai-learning-trustworthy-representations-for-complex-and-networked-systems">Graph AI: Learning trustworthy representations for complex and networked systems</h2>

<p><em>Methods: Graph representation learning, graph neural networks, geometric deep learning, few-shot learning, transfer learning</em></p>

<p>The success of machine learning is heavily dependent on the choice of data features to which the methods are applied. For that reason, much of the actual efforts in deploying algorithms go into the engineering of features that support effective machine learning. We have already made substantial progress in developing representation learning methods that expand the scope and ease the applicability of machine learning in sciences and medicine.</p>

<p>The challenge, however, is that prevailing deep and representation learning algorithms are designed for data with a regular, grid-like structure (e.g., images have a 2D grid structure, and sequences have a linear 1D structure). As a result, these algorithms cannot truly exploit complex, interconnected data with irregular interactions between entities, i.e., edges, the essence of graphs. We are developing methods to address these challenges. The notion of vector space embeddings is at the technical core of these methods. We formalize this idea by specifying deep transformation functions, or graph neural networks, that map nodes, or larger graph structures, to points in a low-dimensional space, termed embeddings. Importantly, these functions are optimized to embed the input network to perform algebraic operations in this learned space to reflect the network’s topology.</p>

<p>Our research has pioneered graph neural networks in bioinformatics and deep learning for network biology and medicine. This allowed us to apply neural networks much more broadly and set sights on new frontiers beyond classic applications of neural networks that learn from images and sequences. We show, for example, how embeddings enable the repurposing of drugs for new indications and the discovery of dozens of safe drug combinations in patients with considerably fewer unwanted side effects than today’s treatments. Further, embeddings allow for accurate molecular phenotyping by identifying drug targets, disease proteins, molecular functions, and other phenotypes better than complex algorithms.</p>

<p>We are also actively developing machine learning methods for learning actionable representations—lending themselves to actionable hypotheses—and allow users of our models to ask what-if questions and receive predictions that are accurate, precise, robust, and can be interpreted meaningfully.</p>

<p>Within the context of biomedical data, we are looking to advance the algorithms to train more with less data, exploit the ability of models to apply prediction prowess acquired from one data type to another type, and design contextually adaptive AI for classes of phenomena that can learn and reason about never-before-seen systems as they encounter new tasks and situations (e.g., new patients, diseases, or cell types).</p>

<div align="center">
<img style="margin-top:20px;margin-bottom:20px;width:100%" src="/img/research-learn.png" />
</div>

<h4 id="perspective">Perspective</h4>

<p><a href="https://www.nature.com/articles/s41551-022-00942-x">Graph Representation Learning for Biomedicine and Healthcare in Nature Biomedical Engineering</a><br /></p>

<h4 id="example-related-publications-1">Example related publications:</h4>

<p><a href="https://academic.oup.com/bioinformatics/article/34/13/i457/5045770">Modeling Polypharmacy Side Effects with Graph Convolutional Networks</a><br />
<a href="https://doi.org/10.3389/fgene.2019.00381">To Embed or Not: Network Embedding as a Paradigm in Computational Biology</a><br />
<a href="https://academic.oup.com/bioinformatics/article/33/14/i190/3953967/Predicting-multicellular-function-through-multi">Predicting Multicellular Function Through Multi-Layer Tissue Networks</a><br />
<a href="https://papers.nips.cc/paper/9123-gnnexplainer-generating-explanations-for-graph-neural-networks">GNN Explainer: Generating Explanations for Graph Neural Networks</a><br />
<a href="https://papers.nips.cc/paper/7473-embedding-logical-queries-on-knowledge-graphs">Embedding Logical Queries on Knowledge Graphs</a><br />
<a href="https://dl.acm.org/citation.cfm?id=3220025">Learning Structural Node Embeddings via Diffusion Wavelets</a></p>

<h4 id="podcasts">Podcasts:</h4>

<p><a href="https://mitharvardwai.github.io/">MITxHarvard Women in Artificial Intelligence</a> [<a href="https://www.youtube.com/watch?v=myv4E5gBnmQ">YouTube Interview</a>]</p>

<h4 id="lectures">Lectures:</h4>

<p><a href="../publications/slides/marinka-zitnik-cs224w.pdf">Slides: Graph Neural Networks in Computational Biology</a><br />
<a href="https://youtu.be/_hy9AgZXhbQ">Recording: Graph Neural Networks in Computational Biology</a></p>

<hr />

<h2 id="actionable-ai-reasoning-about-interconnected-biology-and-medicine">Actionable AI: Reasoning about interconnected biology and medicine</h2>

<p><em>Methods: Heterogeneous and multi-scale networks, model interpretability, graph explainability</em></p>

<p>Networks, or graphs, pervade biomedical data—from the molecular level to the level of connections between diseases in a person and the societal level encompassing all human interactions. These interactions at different levels give rise to a bewildering degree of complexity which is only likely to be fully understood through a holistic and integrated systems view and the study of combined, multi-level networks.</p>

<p>We focus on designing algorithmic solutions to optimize and manipulate networked systems for useful purposes and to predict their behavior, such as how genomics influences human traits in a particular environment. For example, using protein-protein interaction data that have only recently become available, we composed and analyzed interactome networks from 1,840 species across the tree of life, expanding the number of species from about 5 in previous studies to 1,840. This unique dataset allowed us to conduct the largest ever study of protein interactomes and quantify the resilience of interactomes—a critical property as the breakdown of proteins may lead to cell death or disease, showing, for the first time, how interactomes change during evolution, and how these changes impact their response to environmental unpredictability.</p>

<p>We also actively develop methods to harness rich interaction data and network dynamics. We scale up the analyses to see structure in massive amounts of data that are too complex to be detected with other methods.</p>

<div align="center">
<img style="margin-top:20px;margin-bottom:20px;width:100%" src="/img/research-reason.png" />
</div>

<h4 id="example-related-publications-2">Example related publications:</h4>

<p><a href="https://doi.org/10.1073/pnas.1818013116">Evolution of Resilience in Protein Interactomes Across the Tree of Life</a><br />
<a href="https://www.nature.com/articles/s41467-018-05469-x">Network Enhancement as a General Method to Denoise Weighted Biological Networks</a><br />
<a href="http://psb.stanford.edu/psb-online/proceedings/psb18/agrawal.pdf">Large-Scale Analysis of Disease Pathways in the Human Interactome</a></p>

<hr />

<h2 id="ai4science-accelerating-the-discovery-of-safe-and-effective-medicines">AI4Science: Accelerating the discovery of safe and effective medicines</h2>

<p>For centuries, the scientific method—the fundamental practice of science that scientists use to explain the natural world systematically and logically—has remained essentially the same. We have already made substantial progress in using machine learning to change that.</p>

<p>Within the context of drug development, we develop methods that accelerate the discovery of safer and more effective medicines. For example, it can take 15 years and cost $1 billion for a new drug to reach patients, as the question of identifying which diseases a new drug (compound) could treat is tremendously complex. However, diseases are not independent of each other, and many genes are shared between often quite distinct conditions. Similarly, the effects of drugs are not limited to the molecules to which they directly bind in the body; instead, these effects spread throughout the molecular networks in which they act. Therefore, the effect of a drug on a disease is inherently a network phenomenon. We leverage this understanding to develop AI assistants for drug discovery and development.</p>

<div align="center">
<img style="margin-bottom:20px;margin-top:20px;width:100%" src="/img/research-accelerate1.png" />
</div>

<h4 id="example-related-publications-3">Example related publications:</h4>

<p><a href="https://arxiv.org/abs/2004.07229">Network Medicine Framework for Identifying Drug Repurposing Opportunities for COVID-19</a><br />
<a href="https://academic.oup.com/bioinformatics/article/34/13/i457/5045770">Modeling Polypharmacy Side Effects with Graph Convolutional Networks</a><br />
<a href="https://www.nature.com/articles/s41467-018-04948-5">Prioritizing Network Communities</a><br />
<a href="https://openreview.net/pdf?id=HJlWWJSFDH">Pre-training Graph Neural Networks</a><br />
<a href="https://www.biorxiv.org/content/10.1101/2020.02.25.960302v1">Discovering Novel Cell Types Across Heterogeneous Single-Cell Experiments</a><br />
<a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004552">Gene Prioritization by Compressive Data Fusion and Chaining</a><br />
<a href="https://www.tandfonline.com/doi/full/10.4161/sysb.29072">Matrix Factorization-based Data Fusion for Drug-induced Liver Injury Prediction</a><br />
<a href="https://www.nature.com/articles/srep03202">Discovering Disease-disease Associations by Fusing Systems-level Molecular Data</a><br />
<a href="https://psb.stanford.edu/psb-online/proceedings/psb16/zitnik.pdf">Collective Pairwise Classification for Multi-way Analysis of Disease and Drug Data</a></p>

<h4 id="podcasts-1">Podcasts:</h4>

<p><a href="http://www.vision-concept.com/aktuell/Bayer_Foundation_Report_2020/">Bayer Foundation Report (Bayer’s Early Excellence in Science Award)</a><br /> 
<a href="https://bioinformatics.chat/drug-development">Machine Learning for Drug Development</a><br />
<a href="https://omny.fm/shows/pharmacy-podcast-network/data-science-a-i-futuredose-tech">FutureDose Tech</a><br />
<a href="https://www.jclinic.mit.edu/post/a-sweeping-computational-search-for-drugs-against-covid-19">MIT AI Cures Blog</a></p>

<hr />

<h2 id="modern-data-management-ai-infrastructure-and-ai-ready-datasets-for-biomedical-discovery">Modern data management: AI infrastructure and AI-ready datasets for biomedical discovery</h2>

<p>The attention of the ML community to therapeutics remains relatively limited compared to application areas such as natural language processing and computer vision, even though therapeutics offer many challenging problems and applications of immense impact. We posit that this is due to the following key challenges: (1) The lack of high-quality therapeutic datasets and biomedical knowledge prevents researchers from formulating relevant therapeutic tasks as solvable machine-learning problems—the challenge is how to computationally operationalize these data to make them amenable to ML; (2) Datasets are of many different types, including experimental readouts, curated annotations, and metadata, and are scattered around the biorepositories—the challenge for non-domain experts is how to identify, process, and curate datasets relevant to a task of interest; and (3) Despite promising prediction accuracies of computational models, their use in practice, such as for rare diseases and novel drugs in development, is hindered—the challenge is how to assess algorithmic advances in a manner that allows for robust and fair model comparison and represents what one would expect in a  real-world deployment or clinical implementation.</p>

<p>To address these challenges, we founded <a href="https://zitniklab.hms.harvard.edu/TDC/">Therapeutics Data Commons (TDC)</a> <a href="/publications/slides/TDC-futuretx20.pdf">[Slides]</a>, an open-science data platform with AI/ML-ready datasets and learning tasks for therapeutics. Therapeutics Data Commons is a collection of machine learning tasks spread across the entire range of therapeutics. Datasets and benchmarks in TDC provide a systematic model development and evaluation framework that allows more machine learning researchers to contribute to the field. We envision that TDC can considerably accelerate machine-learning model development, validation, and transition into production and clinical implementation.</p>

<div align="center">
<a href="https://zitniklab.hms.harvard.edu/TDC/"><img style="margin-bottom:20px;margin-top:20px;width:100%" src="/img/futuretx20-TDC.png" /></a>
</div>

<h4 id="perspective-1">Perspective</h4>

<p><a href="https://www.nature.com/articles/s41589-022-01131-2">Artificial Intelligence Foundation for Therapeutic Science in Nature Chemical Biology</a><br /></p>

<h4 id="example-related-publications-4">Example related publications:</h4>

<p><a href="https://arxiv.org/abs/2102.09548">Therapeutics Data Commons: Machine Learning Datasets and Tasks for Therapeutics</a><br />
<a href="https://jmlr.csail.mit.edu/papers/volume13/zitnik12a/zitnik12a.pdf">NIMFA: A Python Library for Non-negative Matrix Factorization</a></p>

<h4 id="initiatives">Initiatives:</h4>

<p><a href="https://zitniklab.hms.harvard.edu/TDC/">Therapeutics Data Commons</a></p>

</div>
                </div>
                
            </div>
        </div>
    </section>
    
    <script src="/assets/js/app.js" type="text/javascript"></script><!-- footer scripts -->
<div style="background-color:#A41034">
        <div class="content is-normal has-text-centered">
            <p style="color:white;padding-top:20px;padding-bottom:20px;"><a href="https://scholar.harvard.edu/marinka" style="color:white"><b>Zitnik Lab</b></a>
                &nbsp;&middot;&nbsp; <a href="#" style="color:white"><b>Artificial Intelligence in Medicine and Science</b></a>
                &nbsp;&middot;&nbsp; <a href="https://harvard.edu" style="color:white"><b>Harvard</b></a>
                &nbsp;&middot;&nbsp; <a href="https://dbmi.hms.harvard.edu/" style="color:white"><b>Department of Biomedical Informatics</b></a></p>
        </div>
</div></body>
</html>

