
<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Organizing Conferences, Workshops, and Tutorials - MARINKA ZITNIK</title>
    <link rel="stylesheet" href="/assets/css/app.css">
    <link rel="shortcut icon" type="image/png"
           href="/favicon.png" 
    />
    <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>
    <!-- Begin Jekyll SEO tag v2.6.1 -->
<title>Organizing Conferences, Workshops, and Tutorials | MARINKA ZITNIK</title>
<meta name="generator" content="Jekyll v3.8.6" />
<meta property="og:title" content="Organizing Conferences, Workshops, and Tutorials" />
<meta name="author" content="Marinka Zitnik" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Machine Learning for Medicine and Science" />
<meta property="og:description" content="Machine Learning for Medicine and Science" />
<link rel="canonical" href="https://zitniklab.hms.harvard.edu/meetings/" />
<meta property="og:url" content="https://zitniklab.hms.harvard.edu/meetings/" />
<meta property="og:site_name" content="MARINKA ZITNIK" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="Organizing Conferences, Workshops, and Tutorials" />
<meta name="twitter:site" content="@marinkazitnik" />
<meta name="twitter:creator" content="@Marinka Zitnik" />
<script type="application/ld+json">
{"url":"https://zitniklab.hms.harvard.edu/meetings/","author":{"@type":"Person","name":"Marinka Zitnik"},"headline":"Organizing Conferences, Workshops, and Tutorials","description":"Machine Learning for Medicine and Science","@type":"WebPage","@context":"https://schema.org"}</script>
<!-- End Jekyll SEO tag -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-162129505-1"></script>
<script>
  window['ga-disable-UA-162129505-1'] = window.doNotTrack === "1" || navigator.doNotTrack === "1" || navigator.doNotTrack === "yes" || navigator.msDoNotTrack === "1";
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());
  gtag('config', 'UA-162129505-1');
</script><!-- head scripts --></head>

  <body>
    
<nav class="navbar is-primary" >
    <div class="container">
        <div class="navbar-brand">
            <a href="/" class="navbar-item"><b>
                MARINKA ZITNIK
            </b></a>
            <a role="button" class="navbar-burger burger" aria-label="menu" aria-expanded="false" data-target="navMenu">
                <span aria-hidden="true"></span>
                <span aria-hidden="true"></span>
                <span aria-hidden="true"></span>
            </a>
        </div>
        <div class="navbar-menu" id="navMenu">
            <div class="navbar-start">
<!--                <a href="/" class="navbar-item "><b>Home</b></a>-->
                
                
                    
                    <div class="navbar-item has-dropdown is-hoverable">
                        <a href="/#" class="navbar-link "><b>About</b></a>
                        <div class="navbar-dropdown">
                            
                            <a href="/bio/" class="navbar-item "><b>Bio</b></a>
                            
                            <a href="/contact/" class="navbar-item "><b>Contact</b></a>
                            
                            <a href="/talks/" class="navbar-item "><b>Recent Talks</b></a>
                            
                        </div>
                    </div>
                    
                
                    
                <a href="/research/" class="navbar-item "><b>Research</b></a>
                    
                
                    
                <a href="/publications/" class="navbar-item "><b>Publications</b></a>
                    
                
                    
                <a href="/people/" class="navbar-item "><b>Members</b></a>
                    
                
                    
                <a href="/meetings/" class="navbar-item is-active"><b>Conferences</b></a>
                    
                
                    
                <a href="/DMAI/" class="navbar-item "><b>DMAI</b></a>
                    
                
                    
                <a href="/data/" class="navbar-item "><b>Datasets</b></a>
                    
                
                    
                <a href="/software/" class="navbar-item "><b>ML Tools</b></a>
                    
                
                    
                <a href="https://zitniklab.hms.harvard.edu/TDC/" class="navbar-item "><b>TDC</b></a>
                    
                
                    
                <a href="/news/" class="navbar-item "><b>News</b></a>
                    
                
                    
                <a href="/jobs/" class="navbar-item "><b>Join Us</b></a>
                    
                
                
            </div>
        </div>
    </div>
</nav>

    
        <section class="hero  is-medium  is-bold is-primary"  style="background: url('/hero.jpg') no-repeat center center; background-size: cover;" >
    <div class="hero-body">
        <div class="container">
            <p class="title is-2">Organizing Conferences, Workshops, and Tutorials</p>
            <p class="subtitle is-3"></p>
            
        </div>
    </div>
</section>
    
    


    <section class="section">
        <div class="container">
            <div class="columns">
                
                <div class="column is-8">
                    
                    
                    
    
    

<div class="contents">
    <div class="menu">
        <p class="menu-label">Contents</p>
        <ul class="menu-list">
  <li><a href="#research-tutorials">Research Tutorials</a>
    <ul>
      <li><a href="#machine-learning-for-drug-development-ijcai">Machine Learning for Drug Development (IJCAI)</a></li>
      <li><a href="#deep-learning-for-network-biology-ismb">Deep Learning for Network Biology (ISMB)</a></li>
      <li><a href="#biomedical-data-fusion-embc-and-bc2">Biomedical Data Fusion (EMBC and [BC]^2)</a></li>
    </ul>
  </li>
  <li><a href="#international-workshops-and-conferences">International Workshops and Conferences</a>
    <ul>
      <li><a href="#ai-for-science-neurips">AI for Science (NeurIPS)</a></li>
      <li><a href="#national-symposium-on-drug-repurposing-for-future-pandemics">National Symposium on Drug Repurposing for Future Pandemics</a></li>
      <li><a href="#workshop-on-graph-learning-benchmarks-the-web-conference">Workshop on Graph Learning Benchmarks (The Web Conference)</a></li>
      <li><a href="#representation-learning-on-graphs-and-manifolds-iclr">Representation Learning on Graphs and Manifolds (ICLR)</a></li>
      <li><a href="#graph-representation-learning-and-beyond-icml">Graph Representation Learning and Beyond (ICML)</a></li>
      <li><a href="#trustworthy-ai-for-healthcare-aaai-2021">Trustworthy AI for Healthcare (AAAI 2021)</a></li>
      <li><a href="#trustworthy-ai-for-healthcare-aaai-2022">Trustworthy AI for Healthcare (AAAI 2022)</a></li>
      <li><a href="#ai-in-health-transferring-and-integrating-knowledge-for-better-health-the-web-conference">AI in Health: Transferring and Integrating Knowledge for Better Health (The Web Conference)</a></li>
    </ul>
  </li>
  <li><a href="#research-and-scholarship-meetings">Research and Scholarship Meetings</a>
    <ul>
      <li><a href="#phd-forum-ecmlpkdd">PhD Forum (ECML/PKDD)</a></li>
    </ul>
  </li>
</ul>
    </div>
</div>




<div class="content">
    <h2 id="research-tutorials">Research Tutorials</h2>

<h3 id="machine-learning-for-drug-development-ijcai">Machine Learning for Drug Development (IJCAI)</h3>

<div class="box has-background-info has-text-white">
Machine learning methods leverage big datasets to support decision-making in all stages of drug development, predict how drugs affect the human body and how they interact with each other, and seek ways to boost clinical trials and detect unwanted side effects. This tutorial covers generative modeling, reinforcement learning, and representation learning with a focus on theoretical foundations of methods and their use for key drug-related problems.
</div>

<p>A variety of machine learning methods are demonstrating their utility at all stages of drug development. These methods use big datasets created from high-throughput screening data and allow prediction of bioactivities for targets and molecular properties, identification of new molecules and repurposing of old drugs with increased levels of accuracy.</p>

<p>We have only just begun to realize the potential of these techniques. If methods were available for all aspects of drug development, they could be used seamlessly to predict whether a chemical compound is likely to ultimately become a drug used in patients. Much research needs to be done before this vision can be realized, modern machine learning may have a fundamental impact on the way drug development is done.</p>

<p>The general process of drug development involves five steps. In short, molecular compounds are filtered through a progressive series of tests, which determine their properties, toxicity, and effectiveness for later stages. Machine learning is increasingly being used to accelerate each of the steps, creating opportunities for reducing resources and time needed to develop new drugs. In this tutorial, we cover key problems in drug development that are amenable to machine learning. In doing so, we present a toolbox of AI algorithms for end-to-end drug development.</p>

<p>This tutorial was presented at the <a href="https://ijcai20.org/">International Joint Conference on Artificial Intelligence (IJCAI)</a>.</p>

<p><img src="/img/drugml-ijcai-sm.png" alt="drugml-ijcai" /></p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="/drugml">Tutorial Website</a>
</div>

<h3 id="deep-learning-for-network-biology-ismb">Deep Learning for Network Biology (ISMB)</h3>

<div class="box has-background-info has-text-white">
Networks are ubiquitous in biology where they encode connectivity patterns at all scales of organization, from molecular to the biome. This tutorial investigates key advancements in representation learning for networks over the last few years, with an emphasis on fundamentally new opportunities in network biology enabled by these advancements.
</div>

<p>Biological networks are powerful resources for the discovery of interactions and emergent properties in biological systems, ranging from single-cell to population level. Network approaches have been used many times to combine and amplify signals from individual genes, and have led to remarkable discoveries in biology, including drug discovery, protein function prediction, disease diagnosis, and precision medicine. Furthermore, these approaches have shown broad utility in uncovering new biology and have contributed to new discoveries in wet laboratory experiments.</p>

<p>Mathematical machinery that is central to these approaches is machine learning on networks. The main challenge in machine learning on networks is to find a way to extract information about interactions between nodes and to incorporate that information into a machine learning model. To extract this information from networks, classic machine learning approaches often rely on summary statistics (e.g., degrees or clustering coefficients) or carefully engineered features to measure local neighborhood structures (e.g., network motifs). These classic approaches can be limited because these hand-engineered features are inflexible, they often do not generalize to networks derived from other organisms, tissues and experimental technologies, and can fail on datasets with low experimental coverage.</p>

<p>Recent years have seen a surge in graph neural network (GNN) approaches that automatically learn to encode network structure into low-dimensional representations, using transformation techniques based on deep learning and nonlinear dimensionality reduction. The idea behind these <em>representation learning</em> approaches is to learn a data transformation function that maps nodes to points in a low-dimensional vector space, also termed <em>embeddings</em>. Representation learning methods have revolutionized the state-of-the-art in network science and the goal of this tutorial is to open the door for these methods to computational biology and bioinformatics.</p>

<p>This tutorial was presented at the <a href="https://www.iscb.org/ismb2018">International Conference on Intelligent Systems for Molecular Biology (ISMB)</a>.</p>

<p><img src="/img/deepnetbio-ismb.png" alt="deepnetbio-ismb" /></p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="http://snap.stanford.edu/deepnetbio-ismb/">Tutorial Slides</a>
    <a class="button is-primary" href="http://snap.stanford.edu/deepnetbio-ismb/ipynb/Human+Disease+Network.html">Demo Diseases</a>
    <a class="button is-primary" href="http://snap.stanford.edu/deepnetbio-ismb/ipynb/Graph+Convolutional+Prediction+of+Protein+Interactions+in+Yeast.html">Demo Yeast</a>
    <a class="button is-primary" href="https://github.com/mims-harvard/gnn-intro">GNN Tutorial</a>
</div>

<h3 id="biomedical-data-fusion-embc-and-bc2">Biomedical Data Fusion (EMBC and [BC]^2)</h3>

<div class="box has-background-info has-text-white">
Because of the complex and interconnected nature of biomedical systems, any single model trained on any single dataset can touch only a small part of the entire biomedical knowledge. It is thus critical to integrate diverse sources of information to gain a comprehensive understanding of the system.
</div>

<p>New technologies have enabled the investigation of biology and human health at an unprecedented scale and in multiple dimensions. These dimensions include a myriad of properties describing genome, epigenome, transcriptome, microbiome, phenotype, and lifestyle. No single data type, however, can capture the complexity of all the factors relevant to understanding a phenomenon such as a disease. Integrative methods that combine data from multiple technologies have thus emerged as critical statistical and computational approaches.</p>

<p>The key challenge in developing such approaches is the identification of effective models to provide a comprehensive and relevant systems view. An ideal method can answer a biological or medical question, identifying important features and predicting outcomes, by harnessing heterogeneous data across several dimensions of biological variation.</p>

<p>This tutorial was presented at the <a href="https://embc.embs.org">International Engineering in Medicine and Biology Conference (EMBC)</a> and at the <a href="https://www.bc2.ch/">Basel Compuational Biology Conference ([BC]^2)</a>.</p>

<p><img src="/img/biomedical-data-fusion.png" alt="biomedical-data-fusion" /></p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="http://helikoid.si/embc15/embc-slides.pdf">Tutorial Slides Part 1</a>
    <a class="button is-primary" href="http://helikoid.si/embc15/embc15-fusion.pdf">Tutorial Slides Part 2</a>
    <a class="button is-primary" href="http://helikoid.si/embc15/embc-handouts.pdf">Tutorial Handouts</a>
</div>

<hr />

<h2 id="international-workshops-and-conferences">International Workshops and Conferences</h2>

<h3 id="ai-for-science-neurips">AI for Science (NeurIPS)</h3>

<div class="box has-background-info has-text-white">
Machine learning has advanced a wide array of scientific disciplines and addressed many problems that previously could not be tackled computationally. Despite this promise, several key challenges remain open, and this workshop brings those gaps to the foreground of AI research. 
</div>

<ul>
  <li>
    <p><strong>Gap 1: Unrealistic methodological assumptions.</strong> While ML researchers strive for methodology advances, they often make unrealistic assumptions that limit real-world adoption. For example, most state-of-the-art molecule generation ML models generate molecules that have low synthesizability.</p>
  </li>
  <li>
    <p><strong>Gap 2: Overlooked scientific questions.</strong> Scientific communities contend with crucial and unsolved problems, but they are not yet formulated as solvable ML tasks and are thus overlooked by the ML community.</p>
  </li>
  <li>
    <p><strong>Gap 3: Limited exploration at the intersection of multiple disciplines.</strong> Solutions to grand challenges often stretch across multiple disciplines. For example, protein structure prediction requires collaboration across physics, chemistry and biology.</p>
  </li>
  <li>
    <p><strong>Gap 4: Science of science.</strong> Core principles of the scientific method have not changed since the 17th century. Can AI reason about the organizing principles of our world in a way that is complementary to the hypothesis-experiment cycle to understand a phenomenon?</p>
  </li>
  <li>
    <p><strong>Gap 5: Responsible use and development of AI for science.</strong> Interest in ML across scientific disciplines has surged, but few ML models have transitioned into practical scientific applications. We plan to present a roadmap and ultimately guidelines for accelerating the translation of ML in science. Translation requires a team of engaged stakeholders and a systematic process from the beginning (problem formulation) to the end (widespread deployment) of ML-based research lifecycle.</p>
  </li>
</ul>

<p>This workshop was presented at the <a href="https://nips.cc/Conferences/2021">International Conference on Neural Information Processing Systems (NeurIPS)</a>.</p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://ai4sciencecommunity.github.io/">Workshop Website</a>
</div>

<h3 id="national-symposium-on-drug-repurposing-for-future-pandemics">National Symposium on Drug Repurposing for Future Pandemics</h3>

<div class="box has-background-info has-text-white">
Pandemics demand safe and effective therapies developed and deployed at an unprecedented speed. This symposium, organized on behalf of the National Science Foundation (NSF), provides a forum for scientists and researchers from a variety of fields relevant to therapeutics. Participants discuss ways to expedite the development of therapies by compressing years of work into months or even weeks through automation, artificial intelligence and machine learning, novel data sources, and most recent biotechnology advancements. 
</div>

<p><img src="/img/futuretx20-symposium.png" alt="futuretx20-symposium" /></p>

<p>The symposium brings together leading experts in computer science, biology, statistics, medicine, automation, and regulation. While these areas of expertise are necessary for rapid therapeutic innovation, there is seldom an opportunity for these experts to interact with each other.</p>

<p>Bearing in mind new opportunities and pressing challenges, the symposium provides a roadmap and put forward recommendations on transforming today’s tools into ready-to-use solutions to fight future pathogens.</p>

<p>We announce a new initiative, <a href="https://zitniklab.hms.harvard.edu/TDC/">Therapeutics Data Commons (TDC)</a>, at the symposium <a href="/publications/slides/TDC-futuretx20.pdf">[Slides]</a>.</p>

<p><img src="/img/futuretx20-TDC.png" alt="futuretx20-TDC" /></p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://zitniklab.hms.harvard.edu/TDC/">TDC Website</a>
    <a class="button is-primary" href="https://www.drugsymposium.org/">Symposium Website</a>
</div>

<h3 id="workshop-on-graph-learning-benchmarks-the-web-conference">Workshop on Graph Learning Benchmarks (The Web Conference)</h3>

<div class="box has-background-info has-text-white">
While graph neural networks achieve promising performance on node classification, graph classification, and link prediction, reported performance gains can only be verified and compared within a limited set of publicly available benchmark datasets. The lack of diversity in benchmark datasets may have biased the development of graph representation learning techniques towards narrow directions.
</div>

<p>By crowdsourcing tasks and datasets, the workshop has increased the diversity of graph learning benchmarks, identified open questions in graph representation learning, and deepened the synergy between graph ML algorithms and benchmark datasets.</p>

<p>The workshop was organized at the <a href="https://www2022.thewebconf.org/">Web Conference (WWW)</a>.</p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://graph-learning-benchmarks.github.io/glb2022">Workshop Website</a>
</div>

<h3 id="representation-learning-on-graphs-and-manifolds-iclr">Representation Learning on Graphs and Manifolds (ICLR)</h3>

<div class="box has-background-info has-text-white">
Many scientific fields study data with an underlying graph or manifold structure—such as social networks, sensor networks, biomedical knowledge graphs, and meshed surfaces in computer graphics. Recent years have seen a surge in research on these problems—often under the umbrella terms of graph representation learning and geometric deep learning. 
</div>

<p>The need for new optimization methods and neural network architectures that can accommodate these relational and non-Euclidean structures is becoming increasingly clear. In parallel, there is a growing interest in how we can leverage insights from these domains to incorporate new kinds of relational and non-Euclidean inductive biases into deep learning.</p>

<p>This workshop was presented at the <a href="https://iclr.cc/Conferences/2019">International Conference on Learning Representations (ICLR)</a>.</p>

<p><img src="/img/decagon-architecture.png" alt="decagon-architecture" /></p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://rlgm.github.io/">Workshop Website</a>
</div>

<h3 id="graph-representation-learning-and-beyond-icml">Graph Representation Learning and Beyond (ICML)</h3>

<div class="box has-background-info has-text-white">
Recent years have seen a surge in research on graph representation learning, including techniques for deep graph embeddings, generalizations of CNNs to graph-structured data, and neural message-passing approaches. These advances in graph neural networks and related techniques have led to new state-of-the-art results in numerous domains: chemical synthesis, 3D-vision, recommender systems, question answering, continuous control, self-driving, and social network analysis. 
</div>

<p>This workshop was presented at the <a href="https://icml.cc/Conferences/2020">International Conference on Machine Learning (ICML)</a>.</p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://grlplus.github.io">Workshop Website</a>
</div>

<h3 id="trustworthy-ai-for-healthcare-aaai-2021">Trustworthy AI for Healthcare (AAAI 2021)</h3>

<div class="box has-background-info has-text-white">
Artificial intelligence for healthcare has emerged as an active research area that has made considerable progress, including achieving human-level performance for skin cancer classification, diabetic eye disease detection, chest radiograph diagnosis, and sepsis treatment. While the trends are encouraging, many open challenges prevent us from directly deploying AI solutions in hospitals and clinical environments. A major open problem is the lack of trust of biomedical practitioners in AI methods. Many AI methods make predictions in a black-box way, making decisions challenging to understand and interpret. Further, today's methods are sensitive to small perturbations and adversarial attacks, raising numerous security and privacy concerns. Finally, AI methods learn to make decisions based on training data, which can include biased human decisions or reflect historical or social inequities. These challenges raise numerous trustworthy issues that we need to address to realize the potential of AI in healthcare.  
</div>

<p>This workshop was presented at the <a href="https://aaai.org/Conferences/AAAI-21/">AAAI Conference on Artificial Intelligence (AAAI 2021)</a>.</p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://taih20.github.io/">Workshop Website</a>
</div>

<h3 id="trustworthy-ai-for-healthcare-aaai-2022">Trustworthy AI for Healthcare (AAAI 2022)</h3>

<div class="box has-background-info has-text-white">
AI for healthcare has emerged into a very active research area in the past few years and has made significant progress. For example, AI methods have achieved human-level performance in skin cancer classification, diabetic eye disease detection, chest radiograph diagnosis, sepsis treatment.  
</div>

<p>While current results are encouraging, few clinical AI solutions are deployed in hospitals or routinely used in the clinic. The major problem is that existing clinical AI methods are not sufficiently trustworthy. Black-box methods generate decisions that are difficult to understand and interpret. Furthermore, existing solutions are sensitive to small perturbations and adversarial attacks, which raises security and privacy concerns. Further, methods can produce results biased against certain populations. This workshop addresses these challenges and puts forward recommendations on how to make clinical AI solutions more trustworthy.</p>

<p>This workshop was presented at the <a href="https://aaai.org/Conferences/AAAI-22/">AAAI Conference on Artificial Intelligence (AAAI 2022)</a>.</p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://taih21.github.io/">Workshop Website</a>
</div>

<h3 id="ai-in-health-transferring-and-integrating-knowledge-for-better-health-the-web-conference">AI in Health: Transferring and Integrating Knowledge for Better Health (The Web Conference)</h3>

<div class="box has-background-info has-text-white">
Rich healthcare data connected by semantic relationships and integrated into knowledge graphs can drive biomedical discovery. Biomedical knowledge graphs can support better cohort identification for clinical trials, risk prediction, precision diagnosis, and can inform new and better decision support workflows. Dramatic increase of healthcare data offers unprecedented opportunities for evidence-based care, yet challenges related to interoperability, learning, and reasoning over healthcare data remain open.
</div>

<p>This workshop was presented at the <a href="https://www2021.thewebconf.org/">Web Conference (WWW)</a>.</p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://suitclub.ischool.utexas.edu/AIHealthWWW2021/index.html">Workshop Website</a>
    <a class="button is-primary" href="https://suitclub.ischool.utexas.edu/AIHealthWWW2021/challenge.html">Data Challenge Website</a>
</div>

<hr />

<h2 id="research-and-scholarship-meetings">Research and Scholarship Meetings</h2>

<h3 id="phd-forum-ecmlpkdd">PhD Forum (ECML/PKDD)</h3>

<div class="box has-background-info has-text-white">
PhD Forum provides an environment for junior PhD students to exchange ideas and experiences with peers in an interactive atmosphere and to get constructive feedback from senior researchers in data science, machine learning, and related areas.  
</div>

<p>This meeting took place at the <a href="https://ecmlpkdd2020.net/">European Conference on Machine Learning and Principles and Practice of Knowledge Discovery in Databases (ECML-PKDD)</a>.</p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://ecmlpkdd2020.net/programme/phdforum/">PhD Forum Website</a>
</div>

</div>
                </div>
                
                <div class="column is-4-desktop is-12-tablet">
                    <p class="title is-4">Latest News</p>

<div class="columns is-multiline">
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/01/03/GLB-workshop/">Workshop on Graph Learning Benchmarks</a>-->
        <p class="card-header-title">Jan 2022: &nbsp; <span class="has-text-primary">Workshop on Graph Learning Benchmarks</span></p>
<!--        <p class="card-header-item">Jan 2022</p>-->
<!--        <p class="card-footer-item">Jan 3, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Submit your papers to our <a href="https://graph-learning-benchmarks.github.io/glb2022">Workshop on Graph Learning Benchmarks</a> organized at <a href="https://www2022.thewebconf.org/">The Web Conference (WWW)</a>. <a href="https://graph-learning-benchmarks.github.io/glb2022#call-for-papers">Call for papers!</a></p>
</p>-->
            <p>Submit your papers to our <a href="https://graph-learning-benchmarks.github.io/glb2022">Workshop on Graph Learning Benchmarks</a> organized at <a href="https://www2022.thewebconf.org/">The Web Conference (WWW)</a>. <a href="https://graph-learning-benchmarks.github.io/glb2022#call-for-papers">Call for papers!</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/01/03/GLB-workshop/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Jan 3, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2021/12/24/SpacePrecisionHealth/">NASA: Precision Space Health System</a>-->
        <p class="card-header-title">Dec 2021: &nbsp; <span class="has-text-primary">NASA: Precision Space Health System</span></p>
<!--        <p class="card-header-item">Dec 2021</p>-->
<!--        <p class="card-footer-item">Dec 24, 2021</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Human space exploration beyond low Earth orbit will involve missions of significant distance and duration. To effectively mitigate myriad space health hazards, paradigm shifts in data and space health systems are necessary to enable Earth independence. Delighted to be working with <a href="https://www.nasa.gov/">NASA</a> and can share <a href="https://arxiv.org/abs/2112.12554">our recommendations</a>!</p>
</p>-->
            <p>Human space exploration beyond low Earth orbit will involve missions of significant distance and duration. To effectively mitigate myriad space health hazards, paradigm shifts in data and space health systems are necessary to enable Earth independence. Delighted to be working with <a href="https://www.nasa.gov/">NASA</a> and can share <a href="https://arxiv.org/abs/2112.12554">our recommendations</a>!</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2021/12/24/SpacePrecisionHealth/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Dec 24, 2021</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2021/12/23/SpaceBiology/">Beyond Low Earth Orbit: Biological Research & AI</a>-->
        <p class="card-header-title">Dec 2021: &nbsp; <span class="has-text-primary">Beyond Low Earth Orbit: Biological Research & AI</span></p>
<!--        <p class="card-header-item">Dec 2021</p>-->
<!--        <p class="card-footer-item">Dec 23, 2021</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>As space exploration is extended beyond low Earth orbit, experiments must be maximally autonomous and intelligent to guide biological research. Excited to be working with <a href="https://www.nasa.gov/">NASA</a> and can share <a href="https://arxiv.org/abs/2112.12582">our recommendations</a>!</p>
</p>-->
            <p>As space exploration is extended beyond low Earth orbit, experiments must be maximally autonomous and intelligent to guide biological research. Excited to be working with <a href="https://www.nasa.gov/">NASA</a> and can share <a href="https://arxiv.org/abs/2112.12582">our recommendations</a>!</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2021/12/23/SpaceBiology/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Dec 23, 2021</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2021/12/13/NeurIPSAI4Science/">Attend Our AI4Science Workshop at NeurIPS 2021</a>-->
        <p class="card-header-title">Dec 2021: &nbsp; <span class="has-text-primary">Attend Our AI4Science Workshop at NeurIPS 2021</span></p>
<!--        <p class="card-header-item">Dec 2021</p>-->
<!--        <p class="card-footer-item">Dec 13, 2021</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Join us at <a href="https://nips.cc/Conferences/2021">NeurIPS 2021</a> for the <a href="https://ai4sciencecommunity.github.io/">AI for Science Workshop</a> on Monday, Dec 13, 8am-6pm ET. This will be a great day to celebrate AI achievements in scientific discovery and highlight open challenges that need to be addressed to move the field forward.</p>
</p>-->
            <p>Join us at <a href="https://nips.cc/Conferences/2021">NeurIPS 2021</a> for the <a href="https://ai4sciencecommunity.github.io/">AI for Science Workshop</a> on Monday, Dec 13, 8am-6pm ET. This will be a great day to celebrate AI achievements in scientific discovery and highlight open challenges that need to be addressed to move the field forward.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2021/12/13/NeurIPSAI4Science/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Dec 13, 2021</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2021/11/11/TrustworthyAIWorkshop/">Submit to Our AAAI 2022 Workshop</a>-->
        <p class="card-header-title">Nov 2021: &nbsp; <span class="has-text-primary">Submit to Our AAAI 2022 Workshop</span></p>
<!--        <p class="card-header-item">Nov 2021</p>-->
<!--        <p class="card-footer-item">Nov 11, 2021</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Submit your finest work at the nexus between trustworthy AI and healthcare to <a href="https://aaai.org/Conferences/AAAI-22/">AAAI 2022</a>. <a href="https://taih21.github.io/">Call for papers</a>.</p>
</p>-->
            <p>Submit your finest work at the nexus between trustworthy AI and healthcare to <a href="https://aaai.org/Conferences/AAAI-22/">AAAI 2022</a>. <a href="https://taih21.github.io/">Call for papers</a>.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2021/11/11/TrustworthyAIWorkshop/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Nov 11, 2021</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2021/11/09/PhylogeneticProfilingCollab/">Co-Evolution for Functional Interactions</a>-->
        <p class="card-header-title">Nov 2021: &nbsp; <span class="has-text-primary">Co-Evolution for Functional Interactions</span></p>
<!--        <p class="card-header-item">Nov 2021</p>-->
<!--        <p class="card-footer-item">Nov 9, 2021</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Hot off the press in Nature Communications. Excited to <a href="https://www.nature.com/articles/s41467-021-26792-w">share an ML approach for predicting functional interactions between human genes</a> using the phylogenetic profiles across 1,154 eukaryotic species.</p>
</p>-->
            <p>Hot off the press in Nature Communications. Excited to <a href="https://www.nature.com/articles/s41467-021-26792-w">share an ML approach for predicting functional interactions between human genes</a> using the phylogenetic profiles across 1,154 eukaryotic species.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2021/11/09/PhylogeneticProfilingCollab/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Nov 9, 2021</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2021/10/18/HMSNews/">Adverse Drug Effects During the Pandemic</a>-->
        <p class="card-header-title">Oct 2021: &nbsp; <span class="has-text-primary">Adverse Drug Effects During the Pandemic</span></p>
<!--        <p class="card-header-item">Oct 2021</p>-->
<!--        <p class="card-footer-item">Oct 18, 2021</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>The COVID-19 pandemic has reshaped health and medicine in ways both dramatic and subtle. Some of the less obvious shifts can only emerge from analysis of millions of pieces of data—patient records, medical notes, clinical encounter reports. Read the <a href="https://hms.harvard.edu/news/adverse-drug-effects-during-covid-19-pandemic">story in Harvard Medicine News</a> highlighting our research.</p>
</p>-->
            <p>The COVID-19 pandemic has reshaped health and medicine in ways both dramatic and subtle. Some of the less obvious shifts can only emerge from analysis of millions of pieces of data—patient records, medical notes, clinical encounter reports. Read the <a href="https://hms.harvard.edu/news/adverse-drug-effects-during-covid-19-pandemic">story in Harvard Medicine News</a> highlighting our research.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2021/10/18/HMSNews/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Oct 18, 2021</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2021/10/11/RaindropPreprint/">Graph-Guided Networks for Time Series</a>-->
        <p class="card-header-title">Oct 2021: &nbsp; <span class="has-text-primary">Graph-Guided Networks for Time Series</span></p>
<!--        <p class="card-header-item">Oct 2021</p>-->
<!--        <p class="card-footer-item">Oct 11, 2021</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>New preprint! We introduce <a href="https://arxiv.org/abs/2110.05357">Raindrop</a>, a graph-guided network for learning representations of irregularly sampled multivariate time series.</p>
</p>-->
            <p>New preprint! We introduce <a href="https://arxiv.org/abs/2110.05357">Raindrop</a>, a graph-guided network for learning representations of irregularly sampled multivariate time series.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2021/10/11/RaindropPreprint/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Oct 11, 2021</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2021/10/06/DiffAENatComputScience/">Massive Analysis of Differential Adverse Events</a>-->
        <p class="card-header-title">Oct 2021: &nbsp; <span class="has-text-primary">Massive Analysis of Differential Adverse Events</span></p>
<!--        <p class="card-header-item">Oct 2021</p>-->
<!--        <p class="card-footer-item">Oct 6, 2021</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Hot off the press in Nature Computational Science! We develop an <a href="https://www.nature.com/articles/s43588-021-00138-4">algorithmic approach for massive analysis of drug adverse events</a>. Our analyses of 10,443,476 adverse event reports have implications for safe medication use and public health policy, and can enable comparison of COVID-19 pandemic to other health emergencies.</p>
</p>-->
            <p>Hot off the press in Nature Computational Science! We develop an <a href="https://www.nature.com/articles/s43588-021-00138-4">algorithmic approach for massive analysis of drug adverse events</a>. Our analyses of 10,443,476 adverse event reports have implications for safe medication use and public health policy, and can enable comparison of COVID-19 pandemic to other health emergencies.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2021/10/06/DiffAENatComputScience/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Oct 6, 2021</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2021/09/04/OnClassNatCommun/">Leveraging Cell Ontology to Classify Cell Types</a>-->
        <p class="card-header-title">Sep 2021: &nbsp; <span class="has-text-primary">Leveraging Cell Ontology to Classify Cell Types</span></p>
<!--        <p class="card-header-item">Sep 2021</p>-->
<!--        <p class="card-footer-item">Sep 4, 2021</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Hot off the press in Nature Communications! We developed <a href="https://www.nature.com/articles/s41467-021-25725-x">OnClass, an algorithm and accompanying software for automatically classifying cells into cell types</a> that are part of the controlled vocabulary that forms the Cell Ontology.</p>
</p>-->
            <p>Hot off the press in Nature Communications! We developed <a href="https://www.nature.com/articles/s41467-021-25725-x">OnClass, an algorithm and accompanying software for automatically classifying cells into cell types</a> that are part of the controlled vocabulary that forms the Cell Ontology.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2021/09/04/OnClassNatCommun/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Sep 4, 2021</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2021/09/04/TDCMajorRelease/">Major New Release of TDC</a>-->
        <p class="card-header-title">Sep 2021: &nbsp; <span class="has-text-primary">Major New Release of TDC</span></p>
<!--        <p class="card-header-item">Sep 2021</p>-->
<!--        <p class="card-footer-item">Sep 4, 2021</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>We are very excited to announce a major release of <a href="https://tdcommons.ai">Therapeutics Data Commons</a>! In the <a href="https://github.com/mims-harvard/TDC">0.3.0 release</a> we restructured the codebase, simplified the backend and kept user interfaces the same. We also provide detailed <a href="https://tdc.readthedocs.io/en/main">documentation for our TDC package</a>.</p>
</p>-->
            <p>We are very excited to announce a major release of <a href="https://tdcommons.ai">Therapeutics Data Commons</a>! In the <a href="https://github.com/mims-harvard/TDC">0.3.0 release</a> we restructured the codebase, simplified the backend and kept user interfaces the same. We also provide detailed <a href="https://tdc.readthedocs.io/en/main">documentation for our TDC package</a>.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2021/09/04/TDCMajorRelease/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Sep 4, 2021</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2021/08/30/AAAIWorkshop/">Trustworthy AI for Healthcare at AAAI</a>-->
        <p class="card-header-title">Aug 2021: &nbsp; <span class="has-text-primary">Trustworthy AI for Healthcare at AAAI</span></p>
<!--        <p class="card-header-item">Aug 2021</p>-->
<!--        <p class="card-footer-item">Aug 30, 2021</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>We will be organizing a meeting on Trustworthy AI for Healthcare at <a href="https://aaai.org/Conferences/AAAI-22/">AAAI 2022</a>. Stay tuned for details and call for papers.</p>
</p>-->
            <p>We will be organizing a meeting on Trustworthy AI for Healthcare at <a href="https://aaai.org/Conferences/AAAI-22/">AAAI 2022</a>. Stay tuned for details and call for papers.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2021/08/30/AAAIWorkshop/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Aug 30, 2021</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2021/08/08/NeurIPS-TDC/">New Paper on Therapeutics Data Commons</a>-->
        <p class="card-header-title">Aug 2021: &nbsp; <span class="has-text-primary">New Paper on Therapeutics Data Commons</span></p>
<!--        <p class="card-header-item">Aug 2021</p>-->
<!--        <p class="card-footer-item">Aug 8, 2021</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Our latest paper on <a href="https://arxiv.org/abs/2102.09548">Therapeutics Data Commons: Machine Learning Datasets and Tasks for Drug Discovery and Development</a> will appear at NeurIPS. We are excited to contribute novel datasets and benchmarks in the broad area of therapeutics.</p>
</p>-->
            <p>Our latest paper on <a href="https://arxiv.org/abs/2102.09548">Therapeutics Data Commons: Machine Learning Datasets and Tasks for Drug Discovery and Development</a> will appear at NeurIPS. We are excited to contribute novel datasets and benchmarks in the broad area of therapeutics.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2021/08/08/NeurIPS-TDC/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Aug 8, 2021</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2021/08/03/NeurIPS-AI-Science/">AI for Science at NeurIPS</a>-->
        <p class="card-header-title">Aug 2021: &nbsp; <span class="has-text-primary">AI for Science at NeurIPS</span></p>
<!--        <p class="card-header-item">Aug 2021</p>-->
<!--        <p class="card-footer-item">Aug 3, 2021</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>We are organizing the <a href="https://ai4sciencecommunity.github.io/">AI for Science</a> workshop at NeurIPS 2021 and have a stellar lineup of invited speakers.</p>
</p>-->
            <p>We are organizing the <a href="https://ai4sciencecommunity.github.io/">AI for Science</a> workshop at NeurIPS 2021 and have a stellar lineup of invited speakers.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2021/08/03/NeurIPS-AI-Science/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Aug 3, 2021</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2021/08/02/WCB-best-poster/">Best Poster Award at ICML Comp Biology</a>-->
        <p class="card-header-title">Aug 2021: &nbsp; <span class="has-text-primary">Best Poster Award at ICML Comp Biology</span></p>
<!--        <p class="card-header-item">Aug 2021</p>-->
<!--        <p class="card-footer-item">Aug 2, 2021</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Congratulations to Michelle for winning the Best Poster Award for her work on <a href="https://arxiv.org/abs/2106.02246">deep contextual learners for protein networks</a> at the <a href="https://icml-compbio.github.io/">ICML Workshop on Computational Biology.</a></p>
</p>-->
            <p>Congratulations to Michelle for winning the Best Poster Award for her work on <a href="https://arxiv.org/abs/2106.02246">deep contextual learners for protein networks</a> at the <a href="https://icml-compbio.github.io/">ICML Workshop on Computational Biology.</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2021/08/02/WCB-best-poster/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Aug 2, 2021</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2021/07/24/IMLH-best-paper/">Best Paper Award at ICML Interpretable ML</a>-->
        <p class="card-header-title">Jul 2021: &nbsp; <span class="has-text-primary">Best Paper Award at ICML Interpretable ML</span></p>
<!--        <p class="card-header-item">Jul 2021</p>-->
<!--        <p class="card-footer-item">Jul 24, 2021</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Our short paper on Interactive Visual Explanations for Deep Drug Repurposing received the Best Paper Award at the <a href="https://sites.google.com/view/imlh2021/program?authuser=0">ICML Interpretable ML in Healthcare Workshop</a>. Stay tuned for more news on this evolving project.</p>
</p>-->
            <p>Our short paper on Interactive Visual Explanations for Deep Drug Repurposing received the Best Paper Award at the <a href="https://sites.google.com/view/imlh2021/program?authuser=0">ICML Interpretable ML in Healthcare Workshop</a>. Stay tuned for more news on this evolving project.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2021/07/24/IMLH-best-paper/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Jul 24, 2021</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2021/07/20/ICML-papers/">Five presentations at ICML 2021</a>-->
        <p class="card-header-title">Jul 2021: &nbsp; <span class="has-text-primary">Five presentations at ICML 2021</span></p>
<!--        <p class="card-header-item">Jul 2021</p>-->
<!--        <p class="card-footer-item">Jul 20, 2021</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>We are excited to be at <a href="https://icml.cc/">ICML 2021</a> where we will present 1 paper at <a href="https://icmlsrml2021.github.io/">Workshop on Socially Responsible 
Machine Learning</a>, 1 paper at <a href="https://icml2021-xai.github.io/">Workshop on Theoretic Foundation, Criticism, and Application Trend of Explainable AI</a>, 
2 papers at <a href="https://sites.google.com/view/imlh2021/">Workshop on Interpretable Machine Learning in Healthcare</a>, and 1 paper at <a href="https://icml-compbio.github.io/">Workshop on Computational Biology</a>. Congratulations to fantastic students!</p>
</p>-->
            <p>We are excited to be at <a href="https://icml.cc/">ICML 2021</a> where we will present 1 paper at <a href="https://icmlsrml2021.github.io/">Workshop on Socially Responsible 
Machine Learning</a>, 1 paper at <a href="https://icml2021-xai.github.io/">Workshop on Theoretic Foundation, Criticism, and Application Trend of Explainable AI</a>, 
2 papers at <a href="https://sites.google.com/view/imlh2021/">Workshop on Interpretable Machine Learning in Healthcare</a>, and 1 paper at <a href="https://icml-compbio.github.io/">Workshop on Computational Biology</a>. Congratulations to fantastic students!</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2021/07/20/ICML-papers/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Jul 20, 2021</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2021/06/18/exGNN-preprint/">Theory and Evaluation for Explanations</a>-->
        <p class="card-header-title">Jun 2021: &nbsp; <span class="has-text-primary">Theory and Evaluation for Explanations</span></p>
<!--        <p class="card-header-item">Jun 2021</p>-->
<!--        <p class="card-footer-item">Jun 18, 2021</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>We introduce the <a href="https://arxiv.org/abs/2106.09078">first axiomatic framework</a> for theoretically analyzing, evaluating, and comparing GNN explanation methods. 
We formalize key properties that all methods should satisfy to generate reliable explanations: faithfulness, stability, and fairness.</p>
</p>-->
            <p>We introduce the <a href="https://arxiv.org/abs/2106.09078">first axiomatic framework</a> for theoretically analyzing, evaluating, and comparing GNN explanation methods. 
We formalize key properties that all methods should satisfy to generate reliable explanations: faithfulness, stability, and fairness.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2021/06/18/exGNN-preprint/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Jun 18, 2021</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2021/06/05/nifty-AWARE/">Deep Contextual Learners for Protein Networks</a>-->
        <p class="card-header-title">Jun 2021: &nbsp; <span class="has-text-primary">Deep Contextual Learners for Protein Networks</span></p>
<!--        <p class="card-header-item">Jun 2021</p>-->
<!--        <p class="card-footer-item">Jun 5, 2021</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p><a href="https://arxiv.org/abs/2106.02246">New preprint on contextualized protein embeddings</a> aims to characterize genes with disease-specific interactions and elucidate disease manifestation in specific cell types.</p>
</p>-->
            <p><a href="https://arxiv.org/abs/2106.02246">New preprint on contextualized protein embeddings</a> aims to characterize genes with disease-specific interactions and elucidate disease manifestation in specific cell types.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2021/06/05/nifty-AWARE/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Jun 5, 2021</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2021/05/15/nifty-UAI/">New Paper Accepted at UAI</a>-->
        <p class="card-header-title">May 2021: &nbsp; <span class="has-text-primary">New Paper Accepted at UAI</span></p>
<!--        <p class="card-header-item">May 2021</p>-->
<!--        <p class="card-footer-item">May 15, 2021</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Our <a href="https://arxiv.org/abs/2102.13186">unified framework for fair and stable graph representation learning</a> has just been accepted at UAI. We establish a 
theoretical connection between counterfactual fairness and stability and use it in a framework that can be used with any GNN to learn fair and stable embeddings.</p>
</p>-->
            <p>Our <a href="https://arxiv.org/abs/2102.13186">unified framework for fair and stable graph representation learning</a> has just been accepted at UAI. We establish a 
theoretical connection between counterfactual fairness and stability and use it in a framework that can be used with any GNN to learn fair and stable embeddings.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2021/05/15/nifty-UAI/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: May 15, 2021</p>-->
<!--    </footer>-->
</div>
    </div>
    

    <div class="column is-12">
        <div class="card">
            <header class="card-header">
        <!--        <a class="card-header-title" href=""></a>-->
            <p class="card-header-title"><span class="has-text-primary">Tweets</span></p>
        <!--        <p class="card-header-item"></p>-->
        <!--        <p class="card-footer-item"></p>-->
            </header>
        <div class="card-content">
        <!--        <div class="content">-->
        <!--            -->
        <!--            <p></p>-->
        <a class="twitter-timeline" data-width="500" data-height="500" data-theme="light" href="https://twitter.com/marinkazitnik?ref_src=twsrc%5Etfw" data-chrome="noheader transparent nofooter nofooter noborders">Tweets by marinkazitnik</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
        <!--        </div>-->
        <!--        <div class="has-text-centered">-->
        <!--            <a href="" class="button is-primary">Read more</a>-->
        <!--        </div>-->
            </div>
        </div>
    </div>
</div>




                </div>
                
            </div>
        </div>
    </section>
    
    <script src="/assets/js/app.js" type="text/javascript"></script><!-- footer scripts -->
<div style="background-color:#A41034">
        <div class="content is-normal has-text-centered">
            <p style="color:white;padding-top:20px;padding-bottom:20px;"><a href="https://scholar.harvard.edu/marinka" style="color:white"><b>Zitnik Lab</b></a>
                &nbsp;&middot;&nbsp; <a href="https://harvard.edu" style="color:white"><b>Harvard</b></a>
                &nbsp;&middot;&nbsp; <a href="https://dbmi.hms.harvard.edu/" style="color:white"><b>Department of Biomedical Informatics</b></a></p>
        </div>
</div></body>
</html>

