
<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Scientific Community, Conferences, Workshops, Courses and Tutorials - Zitnik Lab</title>
    <link rel="stylesheet" href="/assets/css/app.css">
    <link rel="shortcut icon" type="image/png"
           href="/favicon.png" 
    />
    <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>
    <!-- Begin Jekyll SEO tag v2.8.0 -->
<title>Scientific Community, Conferences, Workshops, Courses and Tutorials | Zitnik Lab</title>
<meta name="generator" content="Jekyll v4.3.4" />
<meta property="og:title" content="Scientific Community, Conferences, Workshops, Courses and Tutorials" />
<meta name="author" content="Marinka Zitnik" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Artificial Intelligence (AI), Medicine, Science, and Drug Discovery at Harvard" />
<meta property="og:description" content="Artificial Intelligence (AI), Medicine, Science, and Drug Discovery at Harvard" />
<link rel="canonical" href="https://zitniklab.hms.harvard.edu/meetings/" />
<meta property="og:url" content="https://zitniklab.hms.harvard.edu/meetings/" />
<meta property="og:site_name" content="Zitnik Lab" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="Scientific Community, Conferences, Workshops, Courses and Tutorials" />
<meta name="twitter:site" content="@marinkazitnik" />
<meta name="twitter:creator" content="@Marinka Zitnik" />
<script type="application/ld+json">
{"@context":"https://schema.org","@type":"WebPage","author":{"@type":"Person","name":"Marinka Zitnik"},"description":"Artificial Intelligence (AI), Medicine, Science, and Drug Discovery at Harvard","headline":"Scientific Community, Conferences, Workshops, Courses and Tutorials","url":"https://zitniklab.hms.harvard.edu/meetings/"}</script>
<!-- End Jekyll SEO tag -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-162129505-1"></script>
<script>
  window['ga-disable-UA-162129505-1'] = window.doNotTrack === "1" || navigator.doNotTrack === "1" || navigator.doNotTrack === "yes" || navigator.msDoNotTrack === "1";
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());
  gtag('config', 'UA-162129505-1');
</script><!-- head scripts --></head>

  <body>
    
<nav class="navbar is-primary" >
    <div class="container">
        <div class="navbar-brand">
            <a href="/" class="navbar-item"><b>
                Zitnik Lab
            </b></a>
            <a role="button" class="navbar-burger burger" aria-label="menu" aria-expanded="false" data-target="navMenu">
                <span aria-hidden="true"></span>
                <span aria-hidden="true"></span>
                <span aria-hidden="true"></span>
            </a>
        </div>
        <div class="navbar-menu" id="navMenu">
            <div class="navbar-start">
<!--                <a href="/" class="navbar-item "><b>Home</b></a>-->
                
                
                    
                    <div class="navbar-item has-dropdown is-hoverable">
                        <a href="/#" class="navbar-link "><b>About</b></a>
                        <div class="navbar-dropdown">
                            
                            <a href="/bio/" class="navbar-item "><b>Bio</b></a>
                            
                            <a href="/contact/" class="navbar-item "><b>Contact</b></a>
                            
                            <a href="/talks/" class="navbar-item "><b>Recent Talks</b></a>
                            
                        </div>
                    </div>
                    
                
                    
                <a href="/research/" class="navbar-item "><b>Research</b></a>
                    
                
                    
                <a href="/publications/" class="navbar-item "><b>Publications</b></a>
                    
                
                    
                <a href="/people/" class="navbar-item "><b>Members</b></a>
                    
                
                    
                <a href="/meetings/" class="navbar-item is-active"><b>Conferences & Courses</b></a>
                    
                
                    
                <a href="/DMAI/" class="navbar-item "><b>DMAI</b></a>
                    
                
                    
                <a href="/data/" class="navbar-item "><b>Datasets</b></a>
                    
                
                    
                <a href="/software/" class="navbar-item "><b>AI Models</b></a>
                    
                
                    
                <a href="https://zitniklab.hms.harvard.edu/TDC/" class="navbar-item "><b>TDC</b></a>
                    
                
                    
                <a href="/news/" class="navbar-item "><b>News</b></a>
                    
                
                    
                <a href="/jobs/" class="navbar-item "><b>Join Us</b></a>
                    
                
                
            </div>
        </div>
    </div>
</nav>

    
        <section class="hero  is-medium  is-bold is-primary"  style="background: url('/hero.jpg') no-repeat center center; background-size: cover;" >
    <div class="hero-body">
        <div class="container">
            <p class="title is-2">Scientific Community, Conferences, Workshops, Courses and Tutorials</p>
            <p class="subtitle is-3"></p>
            
        </div>
    </div>
</section>
    
    


    <section class="section">
        <div class="container">
            <div class="columns">
                
                <div class="column is-8">
                    
                    
                    
    
    

<div class="contents">
    <div class="menu">
        <p class="menu-label">Contents</p>
        <ul class="menu-list">
  <li><a href="#teaching">Teaching</a>
    <ul>
      <li><a href="#aim-2-artificial-intelligence-in-medicine-2">AIM 2: Artificial Intelligence in Medicine 2</a></li>
      <li><a href="#bmi-702-biomedical-artificial-intelligence">BMI 702: Biomedical Artificial Intelligence</a></li>
    </ul>
  </li>
  <li><a href="#international-competitions">International Competitions</a>
    <ul>
      <li><a href="#curebench-neurips-2025">CUREBench (NeurIPS 2025)</a></li>
    </ul>
  </li>
  <li><a href="#international-workshops-and-conferences">International Workshops and Conferences</a>
    <ul>
      <li><a href="#ai-virtual-cells-and-instruments-a-new-era-in-drug-discovery-and-development-neurips-2025">AI Virtual Cells and Instruments: A New Era in Drug Discovery and Development (NeurIPS 2025)</a></li>
      <li><a href="#ai-for-science-the-reach-and-limits-of-ai-for-scientific-discovery-neurips-2025">AI for Science: The Reach and Limits of AI for Scientific Discovery (NeurIPS 2025)</a></li>
      <li><a href="#harvard-ai-for-genomics-and-health-18th-annual-pqg-conference">Harvard AI for Genomics and Health (18th Annual PQG Conference)</a></li>
      <li><a href="#ai-for-science-scaling-in-ai-for-scientific-discovery-icml-2024">AI for Science: Scaling in AI for Scientific Discovery (ICML 2024)</a></li>
      <li><a href="#ai-for-scientific-discovery-from-theory-to-practice-neurips-2023">AI for Scientific Discovery: From Theory to Practice (NeurIPS 2023)</a></li>
      <li><a href="#graph-learning-benchmarks-kdd-2023">Graph Learning Benchmarks (KDD 2023)</a></li>
      <li><a href="#graph-representations-and-algorithms-in-biomedicine-psb-2023">Graph Representations and Algorithms in Biomedicine (PSB 2023)</a></li>
      <li><a href="#ai4science-progress-and-promises-neurips-2022">AI4Science: Progress and Promises (NeurIPS 2022)</a></li>
      <li><a href="#new-frontiers-in-graph-learning-neurips-2022">New Frontiers in Graph Learning (NeurIPS 2022)</a></li>
      <li><a href="#ai-for-science-icml-2022">AI for Science (ICML 2022)</a></li>
      <li><a href="#trustworthy-ai-for-healthcare-aaai-2022">Trustworthy AI for Healthcare (AAAI 2022)</a></li>
      <li><a href="#workshop-on-graph-learning-benchmarks-the-web-conference-2022">Workshop on Graph Learning Benchmarks (The Web Conference 2022)</a></li>
      <li><a href="#ai-for-science-neurips-2021">AI for Science (NeurIPS 2021)</a></li>
      <li><a href="#trustworthy-ai-for-healthcare-aaai-2021">Trustworthy AI for Healthcare (AAAI 2021)</a></li>
      <li><a href="#ai-in-health-transferring-and-integrating-knowledge-for-better-health-the-web-conference-2021">AI in Health: Transferring and Integrating Knowledge for Better Health (The Web Conference 2021)</a></li>
      <li><a href="#national-symposium-on-drug-repurposing-for-future-pandemics-2020">National Symposium on Drug Repurposing for Future Pandemics (2020)</a></li>
      <li><a href="#graph-representation-learning-and-beyond-icml-2020">Graph Representation Learning and Beyond (ICML 2020)</a></li>
      <li><a href="#representation-learning-on-graphs-and-manifolds-iclr-2019">Representation Learning on Graphs and Manifolds (ICLR 2019)</a></li>
    </ul>
  </li>
  <li><a href="#research-and-scholarship-meetings">Research and Scholarship Meetings</a>
    <ul>
      <li><a href="#therapeutics-data-commons-user-group-meeting-2022">Therapeutics Data Commons User Group Meeting (2022)</a></li>
      <li><a href="#phd-forum-ecmlpkdd-2020">PhD Forum (ECML/PKDD 2020)</a></li>
    </ul>
  </li>
  <li><a href="#seminar-series">Seminar Series</a>
    <ul>
      <li><a href="#harvard-university-efficient-ml-seminar-series">Harvard University Efficient ML Seminar Series</a></li>
    </ul>
  </li>
  <li><a href="#research-tutorials">Research Tutorials</a>
    <ul>
      <li><a href="#towards-precision-medicine-with-graph-representation-learning-ismb-2022">Towards Precision Medicine with Graph Representation Learning (ISMB 2022)</a></li>
      <li><a href="#machine-learning-for-drug-development-ijcai-2021">Machine Learning for Drug Development (IJCAI 2021)</a></li>
      <li><a href="#deep-learning-for-network-biology-ismb-2018">Deep Learning for Network Biology (ISMB 2018)</a></li>
      <li><a href="#biomedical-data-fusion-embc-and-bc2-2015">Biomedical Data Fusion (EMBC and [BC]^2 2015)</a></li>
    </ul>
  </li>
</ul>
    </div>
</div>




<div class="content">
    <h2 id="teaching">Teaching</h2>

<h3 id="aim-2-artificial-intelligence-in-medicine-2">AIM 2: Artificial Intelligence in Medicine 2</h3>

<div class="box has-background-info has-text-white">
AI continues to transform medicine, offering cutting-edge approaches to address challenges in medical research and practice. This course covers modern paradigms in AI, including self-supervised learning, generative models, and multimodal techniques with applications to natural language processing, medical image analysis, relational and structure understanding, and longitudinal sensor data.
</div>

<p><a href="https://arxiv.org/abs/2507.00032">Our study on the use of generative AI in education with prospective evaluation of knowledge tracing in the classroom.</a></p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://zitniklab.hms.harvard.edu/AIM2/">Course Website</a>
</div>

<h3 id="bmi-702-biomedical-artificial-intelligence">BMI 702: Biomedical Artificial Intelligence</h3>

<div class="box has-background-info has-text-white">
AI is poised to enable breakthroughs in science and reshape medicine. This introductory course provides a survey of artificial intelligence for biomedical informatics, covering methods for key data modalities: clinical data, networks, language, and images. It introduces machine learning problems from a practical perspective, focusing on tasks that drive the adoption of machine learning in biology and medicine.

It outlines foundational algorithms and emphasizes the subtleties of working with biomedical data and ways to evaluate and transition machine learning methods into biomedical and clinical implementation. An important consideration in this course is the broader impact of artificial intelligence, particularly topics of bias and fairness, interpretability, and ethical and legal considerations when dealing with artificial intelligence.
</div>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://zitniklab.hms.harvard.edu/BMI702/">Course Website</a>
</div>

<hr />

<h2 id="international-competitions">International Competitions</h2>

<h3 id="curebench-neurips-2025">CUREBench (NeurIPS 2025)</h3>

<div class="box has-background-info has-text-white">
Benchmarking AI reasoning for therapeutic decision-making at scale. This competition is colocated with <a href="https://neurips.cc/Conferences/2025">the International Conference on Neural Information Processing Systems (NeurIPS).</a>
</div>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://curebench.ai/">Competition Website</a>
</div>

<hr />

<h2 id="international-workshops-and-conferences">International Workshops and Conferences</h2>

<h3 id="ai-virtual-cells-and-instruments-a-new-era-in-drug-discovery-and-development-neurips-2025">AI Virtual Cells and Instruments: A New Era in Drug Discovery and Development (NeurIPS 2025)</h3>

<div class="box has-background-info has-text-white">
This workshop focuses on building virtual cells and instruments with AI to support large-scale simulation and analysis of molecules, cells, and tissues. We define this emerging approach and examine how it can be applied to improve drug discovery and development.
</div>

<p>This workshop is presented at <a href="https://neurips.cc/Conferences/2025">the International Conference on Neural Information Processing Systems (NeurIPS).</a></p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://ai4d3.github.io/2025/index.html">Workshop Website</a>
</div>

<h3 id="ai-for-science-the-reach-and-limits-of-ai-for-scientific-discovery-neurips-2025">AI for Science: The Reach and Limits of AI for Scientific Discovery (NeurIPS 2025)</h3>

<div class="box has-background-info has-text-white">
This workshop maps the current capabilities of AI models across scientific domains and explores how to push them further. We focus on three central challenges. First, we ask whether today’s large language models generate rigorously testable hypotheses and reason over experimental results spanning fields like physics, chemistry, and biology. Second, we examine the fidelity of generative and surrogate simulators. In biology, all-atom models gain power; in chemistry, machine learning force fields improve in accuracy and generalizability; and in climate science, models predict weather up to 15 days ahead. We consider how far these advances reach and which spatial or temporal scales remain intractable. Third, we address experimental data scarcity and bias. Consortium-driven efforts like the Protein Data Bank, Human Cell Atlas, Materials Project, and Open Molecules Dataset drive progress, but many fields still lack large-scale datasets. We ask where AI can benefit most from new data generation, how far models trained on limited data can take us, and when lab-in-the-loop strategies are essential. 
</div>

<p>This workshop is presented at <a href="https://neurips.cc/Conferences/2025">the International Conference on Neural Information Processing Systems (NeurIPS).</a></p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://ai4sciencecommunity.github.io/neurips25.html">Workshop Website</a>
</div>

<h3 id="harvard-ai-for-genomics-and-health-18th-annual-pqg-conference">Harvard AI for Genomics and Health (18th Annual PQG Conference)</h3>

<div class="box has-background-info has-text-white">
Recent breakthroughs at the nexus of AI, genomics, and healthcare have been remarkable. Cutting-edge technologies, including transformers, large language models, and protein structure models, are leveraging vast datasets to exhibit remarkable emergent capabilities. At the same time, biotechnological innovations such as high-throughput CRISPR screens, multiplex imaging, and spatial omics are producing rich data at unprecedented scales and resolutions.
</div>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://www.hsph.harvard.edu/pqg-conference/">Conference Website</a>
</div>

<h3 id="ai-for-science-scaling-in-ai-for-scientific-discovery-icml-2024">AI for Science: Scaling in AI for Scientific Discovery (ICML 2024)</h3>

<div class="box has-background-info has-text-white">
As AI extends its reach to a broader range of scientific questions, we explore the role of scalable AI in current scientific endeavors: what further contributions can we expect from AI in research? How can AI techniques be effectively harnessed? Moreover, how does AI influence the objectives and methods of science?
</div>

<p>This workshop is presented at the <a href="https://icml.cc/Conferences/2024/Dates">International Conference on Machine Learning (ICML).</a></p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://ai4sciencecommunity.github.io/icml24.html">Workshop Website</a>
</div>

<h3 id="ai-for-scientific-discovery-from-theory-to-practice-neurips-2023">AI for Scientific Discovery: From Theory to Practice (NeurIPS 2023)</h3>

<div class="box has-background-info has-text-white">
For centuries, the method of discovery—the fundamental practice of science that scientists use to systematically and logically explain the natural world—has remained largely unchanged. Artificial intelligence and machine learning hold tremendous promise for revolutionizing the way scientific discovery is conducted at a fundamental level. However, to realize this potential, it's imperative to identify priorities and address the pressing open questions that stand at the cutting edge of AI. We are especially keen on exploring topics such as solving grand challenges in structural biology, scaling dynamical system modeling to millions of particles, visualizing the previously unimaginable black hole, incorporating physical insights into AI methods, and accelerating drug discovery.
</div>

<p>This workshop is presented at the <a href="https://nips.cc/">International Conference on Neural Information Processing Systems (NeurIPS).</a></p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://ai4sciencecommunity.github.io/neurips23.html">Workshop Website</a>
</div>

<h3 id="graph-learning-benchmarks-kdd-2023">Graph Learning Benchmarks (KDD 2023)</h3>

<div class="box has-background-info has-text-white">
The meeting aims to address the limitations of current benchmark datasets in graph machine learning and the bias towards certain directions in the development of graph learning techniques. To increase the diversity of graph learning benchmarks, identify new research directions, and better understand how different techniques perform on these benchmarks, the meeting discusses novel tasks and datasets. Unlike most existing graph learning workshops that focus on developing new models and algorithms, the meeting focuss on specifying new datasets, tasks, and applications and formalizing them as new benchmarks. 
</div>

<p>The workshop is presented at the <a href="https://kdd.org/kdd2023/">KDD conference</a>.</p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://graph-learning-benchmarks.github.io/glb2023">Workshop Website</a>
</div>

<h3 id="graph-representations-and-algorithms-in-biomedicine-psb-2023">Graph Representations and Algorithms in Biomedicine (PSB 2023)</h3>

<div class="box has-background-info has-text-white">
Connectivity is a fundamental property of biological systems: on the cellular level, proteins interact with each other to form protein-protein interaction networks; on the organism level, neurons are arranged in a network; and on a community level, species can have complex relationships with one another that drive the development of an ecosystem. Graphs, mathematical representations modeling entities as vertices and their relationships as edges, have proved useful for understanding biological systems that naturally have such a network structure.  Graph representations and algorithms (often in combination with machine learning techniques) can be used to organize massive amounts of related (and sometimes heterogenous or unstructured) data, and to ultimately to identify patterns that represent novel biological insights.  
</div>

<p>This session will encompass modern developments in graph theory, computational topology, and graph machine learning applied to various fields of biomedicine. This workshop is presented at the <a href="https://psb.stanford.edu/">Pacific Symposium on Biocomputing (PSB)</a>.</p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://sites.google.com/view/psb-2023-graphs-in-biomedicine/home">Session Website</a>
</div>

<h3 id="ai4science-progress-and-promises-neurips-2022">AI4Science: Progress and Promises (NeurIPS 2022)</h3>

<div class="box has-background-info has-text-white">
For centuries, the method of discovery—the fundamental practice of science that scientists use to explain the natural world systematically and logically—has remained largely the same. Artificial intelligence (AI) and machine learning (ML) hold tremendous promise in having an impact on the way scientific discovery is performed today at the fundamental level. However, to realize this promise, we need to identify priorities and outstanding open questions for the cutting edge of AI going forward.
</div>

<p>This workshop is presented at the <a href="https://nips.cc/Conferences/2022">International Conference on Neural Information Processing Systems (NeurIPS).</a></p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://ai4sciencecommunity.github.io/index.html">Workshop Website</a>
</div>

<h3 id="new-frontiers-in-graph-learning-neurips-2022">New Frontiers in Graph Learning (NeurIPS 2022)</h3>

<div class="box has-background-info has-text-white">
Despite the success, existing graph learning paradigms have not captured the full spectrum of relationships in the physical and the virtual worlds. For example, in terms of applicability of graph learning algorithms, current graph learning paradigms are often restricted to datasets with explicit graph representations, whereas recent works have shown promise of graph learning methods for applications without explicit graph representations. Further, in terms of generalizability, unlike domains such as computer vision and natural language processing where large-scale pre-trained models generalize across downstream applications with little to no fine-tuning and demonstrate impressive performance, such a paradigm has yet to succeed in the graph learning domain. 
</div>

<p>The primary goal of this workshop is to expand the impact of graph learning beyond the current boundaries. Graph learning is poised to become a broadly useful toolbox for learning and understanding any type of (structured) data.</p>

<p>This workshop is presented at the <a href="https://nips.cc/Conferences/2022">International Conference on Neural Information Processing Systems (NeurIPS).</a></p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://glfrontiers.github.io/">Workshop Website</a>
</div>

<h3 id="ai-for-science-icml-2022">AI for Science (ICML 2022)</h3>

<div class="box has-background-info has-text-white">
Machine learning is poised to transform scientific discovery. Despite this promise, critical challenges stifle algorithmic and scientific innovation across scientific disciplines. 
</div>

<p>This workshop brings those challenges to the forefront and discusses which are likely/unlikely to have a broad impact on scientific discovery. With hundreds of AI/ML scientists beginning projects in this area, the workshop brings them together to facilitate community building and consolidate the fast growing area of AI4Science into a vibrant research field.</p>

<p>This workshop is presented at the <a href="https://icml.cc/Conferences/2022/">International Conference on Machine Learning (ICML).</a></p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://ai4sciencecommunity.github.io/icml22.html">Workshop Website</a>
</div>

<h3 id="trustworthy-ai-for-healthcare-aaai-2022">Trustworthy AI for Healthcare (AAAI 2022)</h3>

<div class="box has-background-info has-text-white">
AI for healthcare has emerged into a very active research area in the past few years and has made significant progress. For example, AI methods have achieved human-level performance in skin cancer classification, diabetic eye disease detection, chest radiograph diagnosis, sepsis treatment.  
</div>

<p>While current results are encouraging, few clinical AI solutions are deployed in hospitals or routinely used in the clinic. The major problem is that existing clinical AI methods are not sufficiently trustworthy. Black-box methods generate decisions that are difficult to understand and interpret. Furthermore, existing solutions are sensitive to small perturbations and adversarial attacks, which raises security and privacy concerns. Further, methods can produce results biased against certain populations. This workshop addresses these challenges and puts forward recommendations on how to make clinical AI solutions more trustworthy.</p>

<p>This workshop was presented at the <a href="https://aaai.org/Conferences/AAAI-22/">AAAI Conference on Artificial Intelligence (AAAI 2022)</a>.</p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://taih21.github.io/">Workshop Website</a>
</div>

<h3 id="workshop-on-graph-learning-benchmarks-the-web-conference-2022">Workshop on Graph Learning Benchmarks (The Web Conference 2022)</h3>

<div class="box has-background-info has-text-white">
While graph neural networks achieve promising performance on node classification, graph classification, and link prediction, reported performance gains can only be verified and compared within a limited set of publicly available benchmark datasets. The lack of diversity in benchmark datasets may have biased the development of graph representation learning techniques towards narrow directions.
</div>

<p>By crowdsourcing tasks and datasets, the workshop has increased the diversity of graph learning benchmarks, identified open questions in graph representation learning, and deepened the synergy between graph ML algorithms and benchmark datasets.</p>

<p>The workshop was organized at the <a href="https://www2022.thewebconf.org/">Web Conference (WWW)</a>.</p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://graph-learning-benchmarks.github.io/glb2022">Workshop Website</a>
</div>

<h3 id="ai-for-science-neurips-2021">AI for Science (NeurIPS 2021)</h3>

<div class="box has-background-info has-text-white">
Machine learning has advanced a wide array of scientific disciplines and addressed many problems that previously could not be tackled computationally. Despite this promise, several key challenges remain open, and this workshop brings those gaps to the foreground of AI research. 
</div>

<ul>
  <li>
    <p><strong>Gap 1: Unrealistic methodological assumptions.</strong> While ML researchers strive for methodology advances, they often make unrealistic assumptions that limit real-world adoption. For example, most state-of-the-art molecule generation ML models generate molecules that have low synthesizability.</p>
  </li>
  <li>
    <p><strong>Gap 2: Overlooked scientific questions.</strong> Scientific communities contend with crucial and unsolved problems, but they are not yet formulated as solvable ML tasks and are thus overlooked by the ML community.</p>
  </li>
  <li>
    <p><strong>Gap 3: Limited exploration at the intersection of multiple disciplines.</strong> Solutions to grand challenges often stretch across multiple disciplines. For example, protein structure prediction requires collaboration across physics, chemistry and biology.</p>
  </li>
  <li>
    <p><strong>Gap 4: Science of science.</strong> Core principles of the scientific method have not changed since the 17th century. Can AI reason about the organizing principles of our world in a way that is complementary to the hypothesis-experiment cycle to understand a phenomenon?</p>
  </li>
  <li>
    <p><strong>Gap 5: Responsible use and development of AI for science.</strong> Interest in ML across scientific disciplines has surged, but few ML models have transitioned into practical scientific applications. We plan to present a roadmap and ultimately guidelines for accelerating the translation of ML in science. Translation requires a team of engaged stakeholders and a systematic process from the beginning (problem formulation) to the end (widespread deployment) of ML-based research lifecycle.</p>
  </li>
</ul>

<p>This workshop was presented at the <a href="https://nips.cc/Conferences/2021">International Conference on Neural Information Processing Systems (NeurIPS)</a>.</p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://ai4sciencecommunity.github.io/neurips21.html">Workshop Website</a>
</div>

<h3 id="trustworthy-ai-for-healthcare-aaai-2021">Trustworthy AI for Healthcare (AAAI 2021)</h3>

<div class="box has-background-info has-text-white">
Artificial intelligence for healthcare has emerged as an active research area that has made considerable progress, including achieving human-level performance for skin cancer classification, diabetic eye disease detection, chest radiograph diagnosis, and sepsis treatment. While the trends are encouraging, many open challenges prevent us from directly deploying AI solutions in hospitals and clinical environments. A major open problem is the lack of trust of biomedical practitioners in AI methods. Many AI methods make predictions in a black-box way, making decisions challenging to understand and interpret. Further, today's methods are sensitive to small perturbations and adversarial attacks, raising numerous security and privacy concerns. Finally, AI methods learn to make decisions based on training data, which can include biased human decisions or reflect historical or social inequities. These challenges raise numerous trustworthy issues that we need to address to realize the potential of AI in healthcare.  
</div>

<p>This workshop was presented at the <a href="https://aaai.org/Conferences/AAAI-21/">AAAI Conference on Artificial Intelligence (AAAI 2021)</a>.</p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://taih20.github.io/">Workshop Website</a>
</div>

<h3 id="ai-in-health-transferring-and-integrating-knowledge-for-better-health-the-web-conference-2021">AI in Health: Transferring and Integrating Knowledge for Better Health (The Web Conference 2021)</h3>

<div class="box has-background-info has-text-white">
Rich healthcare data connected by semantic relationships and integrated into knowledge graphs can drive biomedical discovery. Biomedical knowledge graphs can support better cohort identification for clinical trials, risk prediction, precision diagnosis, and can inform new and better decision support workflows. Dramatic increase of healthcare data offers unprecedented opportunities for evidence-based care, yet challenges related to interoperability, learning, and reasoning over healthcare data remain open.
</div>

<p>This workshop was presented at the <a href="https://www2021.thewebconf.org/">Web Conference (WWW)</a>.</p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://suitclub.ischool.utexas.edu/AIHealthWWW2021/index.html">Workshop Website</a>
    <a class="button is-primary" href="https://suitclub.ischool.utexas.edu/AIHealthWWW2021/challenge.html">Data Challenge Website</a>
</div>

<h3 id="national-symposium-on-drug-repurposing-for-future-pandemics-2020">National Symposium on Drug Repurposing for Future Pandemics (2020)</h3>

<div class="box has-background-info has-text-white">
Pandemics demand safe and effective therapies developed and deployed at an unprecedented speed. This symposium, organized on behalf of the National Science Foundation (NSF), provides a forum for scientists and researchers from a variety of fields relevant to therapeutics. Participants discuss ways to expedite the development of therapies by compressing years of work into months or even weeks through automation, artificial intelligence and machine learning, novel data sources, and most recent biotechnology advancements. 
</div>

<p><img src="/img/futuretx20-symposium.png" alt="futuretx20-symposium" /></p>

<p>The symposium brings together leading experts in computer science, biology, statistics, medicine, automation, and regulation. While these areas of expertise are necessary for rapid therapeutic innovation, there is seldom an opportunity for these experts to interact with each other.</p>

<p>Bearing in mind new opportunities and pressing challenges, the symposium provides a roadmap and put forward recommendations on transforming today’s tools into ready-to-use solutions to fight future pathogens.</p>

<p>We announce a new initiative, <a href="https://zitniklab.hms.harvard.edu/TDC/">Therapeutics Data Commons (TDC)</a>, at the symposium <a href="/publications/slides/TDC-futuretx20.pdf">[Slides]</a>.</p>

<p><img src="/img/futuretx20-TDC.png" alt="futuretx20-TDC" /></p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://zitniklab.hms.harvard.edu/TDC/">TDC Website</a>
    <a class="button is-primary" href="https://www.drugsymposium.org/">Symposium Website</a>
</div>

<h3 id="graph-representation-learning-and-beyond-icml-2020">Graph Representation Learning and Beyond (ICML 2020)</h3>

<div class="box has-background-info has-text-white">
Recent years have seen a surge in research on graph representation learning, including techniques for deep graph embeddings, generalizations of CNNs to graph-structured data, and neural message-passing approaches. These advances in graph neural networks and related techniques have led to new state-of-the-art results in numerous domains: chemical synthesis, 3D-vision, recommender systems, question answering, continuous control, self-driving, and social network analysis. 
</div>

<p>This workshop was presented at the <a href="https://icml.cc/Conferences/2020">International Conference on Machine Learning (ICML)</a>.</p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://grlplus.github.io">Workshop Website</a>
</div>

<h3 id="representation-learning-on-graphs-and-manifolds-iclr-2019">Representation Learning on Graphs and Manifolds (ICLR 2019)</h3>

<div class="box has-background-info has-text-white">
Many scientific fields study data with an underlying graph or manifold structure—such as social networks, sensor networks, biomedical knowledge graphs, and meshed surfaces in computer graphics. Recent years have seen a surge in research on these problems—often under the umbrella terms of graph representation learning and geometric deep learning. 
</div>

<p>The need for new optimization methods and neural network architectures that can accommodate these relational and non-Euclidean structures is becoming increasingly clear. In parallel, there is a growing interest in how we can leverage insights from these domains to incorporate new kinds of relational and non-Euclidean inductive biases into deep learning.</p>

<p>This workshop was presented at the <a href="https://iclr.cc/Conferences/2019">International Conference on Learning Representations (ICLR)</a>.</p>

<p><img src="/img/decagon-architecture.png" alt="decagon-architecture" /></p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://rlgm.github.io/">Workshop Website</a>
</div>

<hr />

<h2 id="research-and-scholarship-meetings">Research and Scholarship Meetings</h2>

<h3 id="therapeutics-data-commons-user-group-meeting-2022">Therapeutics Data Commons User Group Meeting (2022)</h3>

<div class="box has-background-info has-text-white">
<a href="https://tdcommons.ai/">Therapeutics Data Commons</a> is an open-science initiative started at Harvard with AI/ML-ready datasets and ML tasks for therapeutics. TDC provides an ecosystem of tools, leaderboards, and community resources, including data functions, strategies for model benchmarking and comparison, meaningful data splits, data processors, public leaderboards, and molecule generation oracles. All resources are integrated via an [open Python library.](https://github.com/mims-harvard/TDC) 
</div>

<p>The lack of high-quality benchmarks impedes the advancement of ML tools for drug discovery. To this end, TDC supports the development of novel ML theory and methods, with a strong bent towards developing the mathematical foundations of which ML algorithms are most suitable for drug discovery applications and why. TDC contains benchmarks for therapeutics ML tasks, including molecular property prediction, molecular interaction prediction, and molecular optimization, all accompanied by extensive programmatic support and leaderboards.</p>

<p>The <a href="https://shoutout.wix.com/so/d1Nv1pC2d#/main">first live user meeting</a> outlined opportunities for how to engage with the TDC community, provided technical background and demos on how TDC supports ML for therapeutics and molecules.</p>

<p align="center">
  <img src="/img/tdc_user_meetup22.jpeg" alt="TDC User Meetup" />
</p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://tdcommons.ai/">Therapeutics Data Commons Website</a>
    <a class="button is-primary" href="https://shoutout.wix.com/so/d1Nv1pC2d#/main">User Group Meetup</a>
</div>

<h3 id="phd-forum-ecmlpkdd-2020">PhD Forum (ECML/PKDD 2020)</h3>

<div class="box has-background-info has-text-white">
PhD Forum provides an environment for junior PhD students to exchange ideas and experiences with peers in an interactive atmosphere and to get constructive feedback from senior researchers in data science, machine learning, and related areas.  
</div>

<p>This meeting took place at the <a href="https://ecmlpkdd2020.net/">European Conference on Machine Learning and Principles and Practice of Knowledge Discovery in Databases (ECML-PKDD)</a>.</p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://ecmlpkdd2020.net/programme/phdforum/">PhD Forum Website</a>
</div>

<hr />

<h2 id="seminar-series">Seminar Series</h2>

<h3 id="harvard-university-efficient-ml-seminar-series">Harvard University Efficient ML Seminar Series</h3>

<div class="box has-background-info has-text-white">
With the advent of foundation models and the emergence of capabilities at scale, algorithms designed to improve the training and inference efficiency of machine learning systems have become critical research topics. Despite their importance, expertise in training the most efficient systems is often in short supply, limiting the ability of teams to scale experiments, work at maximum productivity, develop foundation models for underexplored domains, and make the best use of our computing resources. 
</div>

<p>This seminar series seeks to educate, provide a platform for, and facilitate collaborative research opportunities for all algorithmic and technological questions arising in the quest for more efficient machine learning at scale:</p>

<ul>
  <li>
    <p>A proliferation of knowledge on state-of-the-art approaches directly employable by diverse research teams hoping to scale training or make use of large-scale models.</p>
  </li>
  <li>
    <p>The provision of a research discussion forum on emerging topics to help catalyze further research and form connections among researchers invested in researching these questions.</p>
  </li>
</ul>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://efficientml.org/">Seminar Series Website</a>
</div>

<hr />

<h2 id="research-tutorials">Research Tutorials</h2>

<h3 id="towards-precision-medicine-with-graph-representation-learning-ismb-2022">Towards Precision Medicine with Graph Representation Learning (ISMB 2022)</h3>

<div class="box has-background-info has-text-white">
Graph representation learning has matured immensely as a field within the last few years. Graph machine learning approaches, also known as geometric deep learning, or graph neural networks has become widely used in biomedical applications. This tutorial surveys impact areas in precision medicine (e.g., modeling disease progression, candidate biomarker discovery for targeted therapies, rapid disease diagnostics, treatment regimen recommendations) and highlights new opportunities enabled by these approaches.
</div>

<p>This tutorial was presented at the <a href="https://www.iscb.org/ismb2022">International Conference on Intelligent Systems for Molecular Biology (ISMB)</a>.</p>

<p><img src="/img/biomedgraphml-ismb.png" alt="biomedgraphml-ismb" /></p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="/biomedgraphml">Tutorial Website</a>
</div>

<h3 id="machine-learning-for-drug-development-ijcai-2021">Machine Learning for Drug Development (IJCAI 2021)</h3>

<div class="box has-background-info has-text-white">
Machine learning methods leverage big datasets to support decision-making in all stages of drug development, predict how drugs affect the human body and how they interact with each other, and seek ways to boost clinical trials and detect unwanted side effects. This tutorial covers generative modeling, reinforcement learning, and representation learning with a focus on theoretical foundations of methods and their use for key drug-related problems.
</div>

<p>A variety of machine learning methods are demonstrating their utility at all stages of drug development. These methods use big datasets created from high-throughput screening data and allow prediction of bioactivities for targets and molecular properties, identification of new molecules and repurposing of old drugs with increased levels of accuracy.</p>

<p>We have only just begun to realize the potential of these techniques. If methods were available for all aspects of drug development, they could be used seamlessly to predict whether a chemical compound is likely to ultimately become a drug used in patients. Much research needs to be done before this vision can be realized, modern machine learning may have a fundamental impact on the way drug development is done.</p>

<p>The general process of drug development involves five steps. In short, molecular compounds are filtered through a progressive series of tests, which determine their properties, toxicity, and effectiveness for later stages. Machine learning is increasingly being used to accelerate each of the steps, creating opportunities for reducing resources and time needed to develop new drugs. In this tutorial, we cover key problems in drug development that are amenable to machine learning. In doing so, we present a toolbox of AI algorithms for end-to-end drug development.</p>

<p>This tutorial was presented at the <a href="https://ijcai20.org/">International Joint Conference on Artificial Intelligence (IJCAI)</a>.</p>

<p><img src="/img/drugml-ijcai-sm.png" alt="drugml-ijcai" /></p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="/drugml">Tutorial Website</a>
</div>

<h3 id="deep-learning-for-network-biology-ismb-2018">Deep Learning for Network Biology (ISMB 2018)</h3>

<div class="box has-background-info has-text-white">
Networks are ubiquitous in biology where they encode connectivity patterns at all scales of organization, from molecular to the biome. This tutorial investigates key advancements in representation learning for networks over the last few years, with an emphasis on fundamentally new opportunities in network biology enabled by these advancements.
</div>

<p>Biological networks are powerful resources for the discovery of interactions and emergent properties in biological systems, ranging from single-cell to population level. Network approaches have been used many times to combine and amplify signals from individual genes, and have led to remarkable discoveries in biology, including drug discovery, protein function prediction, disease diagnosis, and precision medicine. Furthermore, these approaches have shown broad utility in uncovering new biology and have contributed to new discoveries in wet laboratory experiments.</p>

<p>Mathematical machinery that is central to these approaches is machine learning on networks. The main challenge in machine learning on networks is to find a way to extract information about interactions between nodes and to incorporate that information into a machine learning model. To extract this information from networks, classic machine learning approaches often rely on summary statistics (e.g., degrees or clustering coefficients) or carefully engineered features to measure local neighborhood structures (e.g., network motifs). These classic approaches can be limited because these hand-engineered features are inflexible, they often do not generalize to networks derived from other organisms, tissues and experimental technologies, and can fail on datasets with low experimental coverage.</p>

<p>Recent years have seen a surge in graph neural network (GNN) approaches that automatically learn to encode network structure into low-dimensional representations, using transformation techniques based on deep learning and nonlinear dimensionality reduction. The idea behind these <em>representation learning</em> approaches is to learn a data transformation function that maps nodes to points in a low-dimensional vector space, also termed <em>embeddings</em>. Representation learning methods have revolutionized the state-of-the-art in network science and the goal of this tutorial is to open the door for these methods to computational biology and bioinformatics.</p>

<p>This tutorial was presented at the <a href="https://www.iscb.org/ismb2018">International Conference on Intelligent Systems for Molecular Biology (ISMB)</a>.</p>

<p><img src="/img/deepnetbio-ismb.png" alt="deepnetbio-ismb" /></p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="http://snap.stanford.edu/deepnetbio-ismb/">Tutorial Slides</a>
    <a class="button is-primary" href="http://snap.stanford.edu/deepnetbio-ismb/ipynb/Human+Disease+Network.html">Demo Diseases</a>
    <a class="button is-primary" href="http://snap.stanford.edu/deepnetbio-ismb/ipynb/Graph+Convolutional+Prediction+of+Protein+Interactions+in+Yeast.html">Demo Yeast</a>
    <a class="button is-primary" href="https://github.com/mims-harvard/gnn-intro">GNN Tutorial</a>
</div>

<h3 id="biomedical-data-fusion-embc-and-bc2-2015">Biomedical Data Fusion (EMBC and [BC]^2 2015)</h3>

<div class="box has-background-info has-text-white">
Because of the complex and interconnected nature of biomedical systems, any single model trained on any single dataset can touch only a small part of the entire biomedical knowledge. It is thus critical to integrate diverse sources of information to gain a comprehensive understanding of the system.
</div>

<p>New technologies have enabled the investigation of biology and human health at an unprecedented scale and in multiple dimensions. These dimensions include a myriad of properties describing genome, epigenome, transcriptome, microbiome, phenotype, and lifestyle. No single data type, however, can capture the complexity of all the factors relevant to understanding a phenomenon such as a disease. Integrative methods that combine data from multiple technologies have thus emerged as critical statistical and computational approaches.</p>

<p>The key challenge in developing such approaches is the identification of effective models to provide a comprehensive and relevant systems view. An ideal method can answer a biological or medical question, identifying important features and predicting outcomes, by harnessing heterogeneous data across several dimensions of biological variation.</p>

<p>This tutorial was presented at the <a href="https://embc.embs.org">International Engineering in Medicine and Biology Conference (EMBC)</a> and at the <a href="https://www.bc2.ch/">Basel Compuational Biology Conference ([BC]^2)</a>.</p>

<p><img src="/img/biomedical-data-fusion.png" alt="biomedical-data-fusion" /></p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="http://helikoid.si/embc15/embc-slides.pdf">Tutorial Slides Part 1</a>
    <a class="button is-primary" href="http://helikoid.si/embc15/embc15-fusion.pdf">Tutorial Slides Part 2</a>
    <a class="button is-primary" href="http://helikoid.si/embc15/embc-handouts.pdf">Tutorial Handouts</a>
</div>


</div>
                </div>
                
                <div class="column is-4-desktop is-12-tablet">
                    <p class="title is-4">Latest News</p>

<div class="columns is-multiline">
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2026/01/22/ARK/">Zoom-Out and Zoom-In Retrieval for LLMs</a>-->
        <p class="card-header-title">Jan 2026: &nbsp; <span class="has-text-primary">Zoom-Out and Zoom-In Retrieval for LLMs</span></p>
<!--        <p class="card-header-item">Jan 2026</p>-->
<!--        <p class="card-footer-item">Jan 22, 2026</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Much of the world’s knowledge lies outside public web text accessible to LLMs, including internal ontologies, curated catalogs, drug safety tables, patient health data, and lab knowledge bases. <a href="https://arxiv.org/abs/2601.13969">ARK helps an LLM to choose, one step at a time, whether to look broadly for relevant information or to dig deeper by following specific links in the data</a>.</p>
</p>-->
            <p>Much of the world’s knowledge lies outside public web text accessible to LLMs, including internal ontologies, curated catalogs, drug safety tables, patient health data, and lab knowledge bases. <a href="https://arxiv.org/abs/2601.13969">ARK helps an LLM to choose, one step at a time, whether to look broadly for relevant information or to dig deeper by following specific links in the data</a>.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2026/01/22/ARK/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Jan 22, 2026</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2026/01/20/Medea/">AI Scientist for Therapeutic Discovery</a>-->
        <p class="card-header-title">Jan 2026: &nbsp; <span class="has-text-primary">AI Scientist for Therapeutic Discovery</span></p>
<!--        <p class="card-header-item">Jan 2026</p>-->
<!--        <p class="card-footer-item">Jan 20, 2026</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>New preprint on <a href="https://www.biorxiv.org/content/10.64898/2026.01.16.696667">Medea, an omics AI agent for therapeutic discovery.</a> <a href="https://medea.openscientist.ai/">[Project website]</a></p>
</p>-->
            <p>New preprint on <a href="https://www.biorxiv.org/content/10.64898/2026.01.16.696667">Medea, an omics AI agent for therapeutic discovery.</a> <a href="https://medea.openscientist.ai/">[Project website]</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2026/01/20/Medea/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Jan 20, 2026</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2026/01/12/ToolUniverse/">AI Scientists - LLMs Using Scientific Tools</a>-->
        <p class="card-header-title">Jan 2026: &nbsp; <span class="has-text-primary">AI Scientists - LLMs Using Scientific Tools</span></p>
<!--        <p class="card-header-item">Jan 2026</p>-->
<!--        <p class="card-footer-item">Jan 12, 2026</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Excited about this <a href="https://anthropic.com/news/healthcare-life-sciences">academic collaboration with Anthropic</a> on adding connectors to <a href="https://aiscientist.tools/">ToolUniverse</a> to make Claude even more powerful for scientific discovery.</p>
</p>-->
            <p>Excited about this <a href="https://anthropic.com/news/healthcare-life-sciences">academic collaboration with Anthropic</a> on adding connectors to <a href="https://aiscientist.tools/">ToolUniverse</a> to make Claude even more powerful for scientific discovery.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2026/01/12/ToolUniverse/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Jan 12, 2026</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2025/12/17/PROTON/">AI + Validation in Molecular, Organoid, and Clinical Systems</a>-->
        <p class="card-header-title">Dec 2025: &nbsp; <span class="has-text-primary">AI + Validation in Molecular, Organoid, and Clinical Systems</span></p>
<!--        <p class="card-header-item">Dec 2025</p>-->
<!--        <p class="card-footer-item">Dec 17, 2025</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Pairing AI with experiments: Our latest PROTON AI <a href="https://arxiv.org/abs/2512.13724">generates neurological hypotheses that we validated across molecular, organoid, and clinical systems.</a> <a href="https://www.protonmodel.ai/">[Project website]</a></p>

</p>-->
            <p>Pairing AI with experiments: Our latest PROTON AI <a href="https://arxiv.org/abs/2512.13724">generates neurological hypotheses that we validated across molecular, organoid, and clinical systems.</a> <a href="https://www.protonmodel.ai/">[Project website]</a></p>


<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2025/12/17/PROTON/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Dec 17, 2025</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2025/12/16/Magneton/">Greater than the Sum of Its Parts</a>-->
        <p class="card-header-title">Dec 2025: &nbsp; <span class="has-text-primary">Greater than the Sum of Its Parts</span></p>
<!--        <p class="card-header-item">Dec 2025</p>-->
<!--        <p class="card-footer-item">Dec 16, 2025</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>New preprint on <a href="https://arxiv.org/abs/2512.18114">Greater than the Sum of Its Parts: Building Substructure into Protein Encoding Models</a>. <a href="https://zitniklab.hms.harvard.edu/projects/Magneton/">[Project website]</a></p>
</p>-->
            <p>New preprint on <a href="https://arxiv.org/abs/2512.18114">Greater than the Sum of Its Parts: Building Substructure into Protein Encoding Models</a>. <a href="https://zitniklab.hms.harvard.edu/projects/Magneton/">[Project website]</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2025/12/16/Magneton/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Dec 16, 2025</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2025/12/09/DigitalTwins/">Digital Twinning</a>-->
        <p class="card-header-title">Dec 2025: &nbsp; <span class="has-text-primary">Digital Twinning</span></p>
<!--        <p class="card-header-item">Dec 2025</p>-->
<!--        <p class="card-footer-item">Dec 9, 2025</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>A piece in <a href="https://news.harvard.edu/gazette/story/2025/12/your-digital-twin-might-save-your-life/">Harvard Gazette</a> on digital twins, cellular chatbots, and building digital twins at a cellular scale.</p>

</p>-->
            <p>A piece in <a href="https://news.harvard.edu/gazette/story/2025/12/your-digital-twin-might-save-your-life/">Harvard Gazette</a> on digital twins, cellular chatbots, and building digital twins at a cellular scale.</p>


<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2025/12/09/DigitalTwins/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Dec 9, 2025</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2025/12/09/NeurIPS-DrugDiscovery/">Virtual Cells and Instruments</a>-->
        <p class="card-header-title">Dec 2025: &nbsp; <span class="has-text-primary">Virtual Cells and Instruments</span></p>
<!--        <p class="card-header-item">Dec 2025</p>-->
<!--        <p class="card-footer-item">Dec 9, 2025</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>We are excited to meet hundreds of researchers attending our <a href="https://ai4d3.github.io/2025/index.html">AI Virtual Cells and Instruments: A New Era in Drug Discovery and Development</a> workshop at NeurIPS 2025.</p>

</p>-->
            <p>We are excited to meet hundreds of researchers attending our <a href="https://ai4d3.github.io/2025/index.html">AI Virtual Cells and Instruments: A New Era in Drug Discovery and Development</a> workshop at NeurIPS 2025.</p>


<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2025/12/09/NeurIPS-DrugDiscovery/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Dec 9, 2025</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2025/12/07/NeurIPS-CUREBench/">CUREBench</a>-->
        <p class="card-header-title">Dec 2025: &nbsp; <span class="has-text-primary">CUREBench</span></p>
<!--        <p class="card-header-item">Dec 2025</p>-->
<!--        <p class="card-footer-item">Dec 7, 2025</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Excited to see 1,622 researchers from around the world entering our <a href="https://curebench.ai/">CUREBench Challenge</a> with 398 participating teams that made 3,383 submissions to the competition and submitted 8,457,500+ AI reasoning traces for therapeutics. Join us at the Award Ceremony at NeurIPS.</p>
</p>-->
            <p>Excited to see 1,622 researchers from around the world entering our <a href="https://curebench.ai/">CUREBench Challenge</a> with 398 participating teams that made 3,383 submissions to the competition and submitted 8,457,500+ AI reasoning traces for therapeutics. Join us at the Award Ceremony at NeurIPS.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2025/12/07/NeurIPS-CUREBench/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Dec 7, 2025</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2025/12/07/NeurIPS-AI4Science/">AI For Science at NeurIPS</a>-->
        <p class="card-header-title">Dec 2025: &nbsp; <span class="has-text-primary">AI For Science at NeurIPS</span></p>
<!--        <p class="card-header-item">Dec 2025</p>-->
<!--        <p class="card-footer-item">Dec 7, 2025</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Join us and hundreds of other scientists at the <a href="https://ai4sciencecommunity.github.io/neurips25.html">6th AI for Science workshop at NeurIPS.</a></p>

</p>-->
            <p>Join us and hundreds of other scientists at the <a href="https://ai4sciencecommunity.github.io/neurips25.html">6th AI for Science workshop at NeurIPS.</a></p>


<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2025/12/07/NeurIPS-AI4Science/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Dec 7, 2025</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2025/11/18/GeoBPE/">Protein Structure Tokenization</a>-->
        <p class="card-header-title">Nov 2025: &nbsp; <span class="has-text-primary">Protein Structure Tokenization</span></p>
<!--        <p class="card-header-item">Nov 2025</p>-->
<!--        <p class="card-footer-item">Nov 18, 2025</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>New preprint introducing <a href="https://arxiv.org/abs/2511.11758">GeoBPE - Protein structure tokenization via geometric byte pair encoding.</a></p>

</p>-->
            <p>New preprint introducing <a href="https://arxiv.org/abs/2511.11758">GeoBPE - Protein structure tokenization via geometric byte pair encoding.</a></p>


<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2025/11/18/GeoBPE/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Nov 18, 2025</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2025/11/11/CONCERT/">Generative AI Model for Spatial Biology</a>-->
        <p class="card-header-title">Nov 2025: &nbsp; <span class="has-text-primary">Generative AI Model for Spatial Biology</span></p>
<!--        <p class="card-header-item">Nov 2025</p>-->
<!--        <p class="card-footer-item">Nov 11, 2025</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>New preprint introducing <a href="https://www.biorxiv.org/content/10.1101/2025.11.08.686890">CONCERT, a niche-aware generative model that predicts perturbation effects across spatial tissue contexts.</a></p>

</p>-->
            <p>New preprint introducing <a href="https://www.biorxiv.org/content/10.1101/2025.11.08.686890">CONCERT, a niche-aware generative model that predicts perturbation effects across spatial tissue contexts.</a></p>


<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2025/11/11/CONCERT/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Nov 11, 2025</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2025/11/02/ScienceFeature/">AI Cell Models</a>-->
        <p class="card-header-title">Nov 2025: &nbsp; <span class="has-text-primary">AI Cell Models</span></p>
<!--        <p class="card-header-item">Nov 2025</p>-->
<!--        <p class="card-footer-item">Nov 2, 2025</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>A piece in <a href="https://www.science.org/content/article/can-ai-capture-mind-boggling-complexity-human-cell">Science</a> explores how AI cell models could transform biomedicine (if they work as promised) and highlights <a href="https://aiscientist.tools/">ToolUniverse</a>. ToolUniverse lets AI co-scientists test, analyze, and build on AI cell models.</p>

</p>-->
            <p>A piece in <a href="https://www.science.org/content/article/can-ai-capture-mind-boggling-complexity-human-cell">Science</a> explores how AI cell models could transform biomedicine (if they work as promised) and highlights <a href="https://aiscientist.tools/">ToolUniverse</a>. ToolUniverse lets AI co-scientists test, analyze, and build on AI cell models.</p>


<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2025/11/02/ScienceFeature/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Nov 2, 2025</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2025/10/25/NatureNewsFeature/">Is AI sycophancy holding science back?</a>-->
        <p class="card-header-title">Oct 2025: &nbsp; <span class="has-text-primary">Is AI sycophancy holding science back?</span></p>
<!--        <p class="card-header-item">Oct 2025</p>-->
<!--        <p class="card-footer-item">Oct 25, 2025</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>A piece in <a href="https://www.nature.com/articles/d41586-025-03390-0">Nature</a> explores how AI sycophancy, in which models agree too much with users instead of reasoning on its own, could affect the use of AI in medical research.</p>

</p>-->
            <p>A piece in <a href="https://www.nature.com/articles/d41586-025-03390-0">Nature</a> explores how AI sycophancy, in which models agree too much with users instead of reasoning on its own, could affect the use of AI in medical research.</p>


<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2025/10/25/NatureNewsFeature/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Oct 25, 2025</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2025/10/12/NewsFeature/">Our research featured by Kempner and Crimson</a>-->
        <p class="card-header-title">Oct 2025: &nbsp; <span class="has-text-primary">Our research featured by Kempner and Crimson</span></p>
<!--        <p class="card-header-item">Oct 2025</p>-->
<!--        <p class="card-footer-item">Oct 12, 2025</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>A news story about <a href="https://www.nature.com/articles/s41551-025-01481-x">PDGrapher</a> in <a href="https://www.thecrimson.com/article/2025/10/6/hms-tool-drug-discovery/">Harvard Crimson.</a> <a href="https://aiscientist.tools/">ToolUniverse</a> featured on the <a href="https://kempnerinstitute.harvard.edu/research/deeper-learning/from-models-to-scientists-building-ai-agents-for-scientific-discovery/">Kempner Institute blog.</a></p>
</p>-->
            <p>A news story about <a href="https://www.nature.com/articles/s41551-025-01481-x">PDGrapher</a> in <a href="https://www.thecrimson.com/article/2025/10/6/hms-tool-drug-discovery/">Harvard Crimson.</a> <a href="https://aiscientist.tools/">ToolUniverse</a> featured on the <a href="https://kempnerinstitute.harvard.edu/research/deeper-learning/from-models-to-scientists-building-ai-agents-for-scientific-discovery/">Kempner Institute blog.</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2025/10/12/NewsFeature/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Oct 12, 2025</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2025/10/05/AIAgentsForScience/">A Scientist's Guide to AI Agents in Nature</a>-->
        <p class="card-header-title">Oct 2025: &nbsp; <span class="has-text-primary">A Scientist's Guide to AI Agents in Nature</span></p>
<!--        <p class="card-header-item">Oct 2025</p>-->
<!--        <p class="card-footer-item">Oct 5, 2025</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p><a href="https://www.nature.com/articles/d41586-025-03246-7">A piece on AI agents in Nature</a> highlights ongoing projects in our group, including methods for evaluating scientific hypotheses, challenges in benchmarking AI agents, and the open ToolUniverse ecosystem.</p>
</p>-->
            <p><a href="https://www.nature.com/articles/d41586-025-03246-7">A piece on AI agents in Nature</a> highlights ongoing projects in our group, including methods for evaluating scientific hypotheses, challenges in benchmarking AI agents, and the open ToolUniverse ecosystem.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2025/10/05/AIAgentsForScience/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Oct 5, 2025</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2025/09/29/ToolUniversePaper/">ToolUniverse: AI Agents for Science and Medicine</a>-->
        <p class="card-header-title">Sep 2025: &nbsp; <span class="has-text-primary">ToolUniverse: AI Agents for Science and Medicine</span></p>
<!--        <p class="card-header-item">Sep 2025</p>-->
<!--        <p class="card-footer-item">Sep 29, 2025</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>New paper: <a href="https://arxiv.org/abs/2509.23426">ToolUniverse</a> introduces an open ecosystem for building AI scientists with 600+ scientific and biomedical tools. Build your AI co-scientists at <a href="https://aiscientist.tools/">https://aiscientist.tools</a>.</p>
</p>-->
            <p>New paper: <a href="https://arxiv.org/abs/2509.23426">ToolUniverse</a> introduces an open ecosystem for building AI scientists with 600+ scientific and biomedical tools. Build your AI co-scientists at <a href="https://aiscientist.tools/">https://aiscientist.tools</a>.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2025/09/29/ToolUniversePaper/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Sep 29, 2025</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2025/09/27/ToolUniverse/">Democratizing "AI Scientists" with ToolUniverse</a>-->
        <p class="card-header-title">Sep 2025: &nbsp; <span class="has-text-primary">Democratizing "AI Scientists" with ToolUniverse</span></p>
<!--        <p class="card-header-item">Sep 2025</p>-->
<!--        <p class="card-footer-item">Sep 27, 2025</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Our new initiative: <strong>Use Tool Universe to build an AI scientist for yourself from any language or reasoning model, whether open or closed. <a href="https://aiscientist.tools/">https://aiscientist.tools</a></strong></p>
</p>-->
            <p>Our new initiative: <strong>Use Tool Universe to build an AI scientist for yourself from any language or reasoning model, whether open or closed. <a href="https://aiscientist.tools/">https://aiscientist.tools</a></strong></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2025/09/27/ToolUniverse/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Sep 27, 2025</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2025/09/15/InfEHR/">InfEHR in Nature Communications</a>-->
        <p class="card-header-title">Sep 2025: &nbsp; <span class="has-text-primary">InfEHR in Nature Communications</span></p>
<!--        <p class="card-header-item">Sep 2025</p>-->
<!--        <p class="card-footer-item">Sep 15, 2025</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Collaboration with Ben and Girish on <a href="https://www.nature.com/articles/s41467-025-63366-6">clinical phenotype resolution through deep geometric learning on electronic health records</a> published in Nature Communications.</p>
</p>-->
            <p>Collaboration with Ben and Girish on <a href="https://www.nature.com/articles/s41467-025-63366-6">clinical phenotype resolution through deep geometric learning on electronic health records</a> published in Nature Communications.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2025/09/15/InfEHR/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Sep 15, 2025</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2025/09/10/PDGrapher/">PDGrapher in Nature Biomedical Engineering</a>-->
        <p class="card-header-title">Sep 2025: &nbsp; <span class="has-text-primary">PDGrapher in Nature Biomedical Engineering</span></p>
<!--        <p class="card-header-item">Sep 2025</p>-->
<!--        <p class="card-footer-item">Sep 10, 2025</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>New paper in Nature Biomedical Engineering introducing <a href="https://www.nature.com/articles/s41551-025-01481-x">PDGrapher, a model for phenotype-based target discovery</a>. <a href="https://hms.harvard.edu/news/new-ai-tool-pinpoints-genes-drug-combos-restore-health-diseased-cells">[Harvard Medicine News]</a></p>
</p>-->
            <p>New paper in Nature Biomedical Engineering introducing <a href="https://www.nature.com/articles/s41551-025-01481-x">PDGrapher, a model for phenotype-based target discovery</a>. <a href="https://hms.harvard.edu/news/new-ai-tool-pinpoints-genes-drug-combos-restore-health-diseased-cells">[Harvard Medicine News]</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2025/09/10/PDGrapher/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Sep 10, 2025</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2025/09/01/PrecisionMedicineNEJMAI/">AI and Net Medicine: Path to Precision Medicine</a>-->
        <p class="card-header-title">Sep 2025: &nbsp; <span class="has-text-primary">AI and Net Medicine: Path to Precision Medicine</span></p>
<!--        <p class="card-header-item">Sep 2025</p>-->
<!--        <p class="card-footer-item">Sep 1, 2025</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>New perspective published in NEJM AI on <a href="https://ai.nejm.org/stoken/default+domain/AGEYPVA9IEY6DAHJ2INC/full?redirectUri=doi/full/10.1056/AIra2401229">advancing AI and network medicine on a path to precision medicine.</a></p>
</p>-->
            <p>New perspective published in NEJM AI on <a href="https://ai.nejm.org/stoken/default+domain/AGEYPVA9IEY6DAHJ2INC/full?redirectUri=doi/full/10.1056/AIra2401229">advancing AI and network medicine on a path to precision medicine.</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2025/09/01/PrecisionMedicineNEJMAI/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Sep 1, 2025</p>-->
<!--    </footer>-->
</div>
    </div>
    

    <div class="column is-12">
        <div class="card">
            <header class="card-header">
        <!--        <a class="card-header-title" href=""></a>-->
            <p class="card-header-title"><span class="has-text-primary">Tweets</span></p>
        <!--        <p class="card-header-item"></p>-->
        <!--        <p class="card-footer-item"></p>-->
            </header>
        <div class="card-content">
        <!--        <div class="content">-->
        <!--            -->
        <!--            <p></p>-->
        <a class="twitter-timeline" data-width="500" data-height="500" data-theme="light" href="https://twitter.com/marinkazitnik?ref_src=twsrc%5Etfw" data-chrome="noheader transparent nofooter nofooter noborders">Tweets by marinkazitnik</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
        <!--        </div>-->
        <!--        <div class="has-text-centered">-->
        <!--            <a href="" class="button is-primary">Read more</a>-->
        <!--        </div>-->
            </div>
        </div>
    </div>
</div>




                </div>
                
            </div>
        </div>
    </section>
    
    <script src="/assets/js/app.js" type="text/javascript"></script><!-- footer scripts -->
<div style="background-color:#A41034">
        <div class="content is-normal has-text-centered">
            <p style="color:white;padding-top:20px;padding-bottom:20px;"><a href="https://scholar.harvard.edu/marinka" style="color:white"><b>Zitnik Lab</b></a>
                &nbsp;&middot;&nbsp; <a href="#" style="color:white"><b>Artificial Intelligence in Medicine and Science</b></a>
                &nbsp;&middot;&nbsp; <a href="https://harvard.edu" style="color:white"><b>Harvard</b></a>
                &nbsp;&middot;&nbsp; <a href="https://dbmi.hms.harvard.edu/" style="color:white"><b>Department of Biomedical Informatics</b></a></p>
        </div>
</div></body>
</html>

