
<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Scientific Community, Conferences, Workshops, Courses and Tutorials - Zitnik Lab</title>
    <link rel="stylesheet" href="/assets/css/app.css">
    <link rel="shortcut icon" type="image/png"
           href="/favicon.png" 
    />
    <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>
    <!-- Begin Jekyll SEO tag v2.6.1 -->
<title>Scientific Community, Conferences, Workshops, Courses and Tutorials | Zitnik Lab</title>
<meta name="generator" content="Jekyll v3.8.6" />
<meta property="og:title" content="Scientific Community, Conferences, Workshops, Courses and Tutorials" />
<meta name="author" content="Marinka Zitnik" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Artificial Intelligence (AI), Medicine, Science, and Drug Discovery at Harvard" />
<meta property="og:description" content="Artificial Intelligence (AI), Medicine, Science, and Drug Discovery at Harvard" />
<link rel="canonical" href="https://zitniklab.hms.harvard.edu/meetings/" />
<meta property="og:url" content="https://zitniklab.hms.harvard.edu/meetings/" />
<meta property="og:site_name" content="Zitnik Lab" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="Scientific Community, Conferences, Workshops, Courses and Tutorials" />
<meta name="twitter:site" content="@marinkazitnik" />
<meta name="twitter:creator" content="@Marinka Zitnik" />
<script type="application/ld+json">
{"url":"https://zitniklab.hms.harvard.edu/meetings/","author":{"@type":"Person","name":"Marinka Zitnik"},"headline":"Scientific Community, Conferences, Workshops, Courses and Tutorials","description":"Artificial Intelligence (AI), Medicine, Science, and Drug Discovery at Harvard","@type":"WebPage","@context":"https://schema.org"}</script>
<!-- End Jekyll SEO tag -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-162129505-1"></script>
<script>
  window['ga-disable-UA-162129505-1'] = window.doNotTrack === "1" || navigator.doNotTrack === "1" || navigator.doNotTrack === "yes" || navigator.msDoNotTrack === "1";
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());
  gtag('config', 'UA-162129505-1');
</script><!-- head scripts --></head>

  <body>
    
<nav class="navbar is-primary" >
    <div class="container">
        <div class="navbar-brand">
            <a href="/" class="navbar-item"><b>
                Zitnik Lab
            </b></a>
            <a role="button" class="navbar-burger burger" aria-label="menu" aria-expanded="false" data-target="navMenu">
                <span aria-hidden="true"></span>
                <span aria-hidden="true"></span>
                <span aria-hidden="true"></span>
            </a>
        </div>
        <div class="navbar-menu" id="navMenu">
            <div class="navbar-start">
<!--                <a href="/" class="navbar-item "><b>Home</b></a>-->
                
                
                    
                    <div class="navbar-item has-dropdown is-hoverable">
                        <a href="/#" class="navbar-link "><b>About</b></a>
                        <div class="navbar-dropdown">
                            
                            <a href="/bio/" class="navbar-item "><b>Bio</b></a>
                            
                            <a href="/contact/" class="navbar-item "><b>Contact</b></a>
                            
                            <a href="/talks/" class="navbar-item "><b>Recent Talks</b></a>
                            
                        </div>
                    </div>
                    
                
                    
                <a href="/research/" class="navbar-item "><b>Research</b></a>
                    
                
                    
                <a href="/publications/" class="navbar-item "><b>Publications</b></a>
                    
                
                    
                <a href="/people/" class="navbar-item "><b>Members</b></a>
                    
                
                    
                <a href="/meetings/" class="navbar-item is-active"><b>Education</b></a>
                    
                
                    
                <a href="/DMAI/" class="navbar-item "><b>DMAI</b></a>
                    
                
                    
                <a href="/data/" class="navbar-item "><b>Datasets</b></a>
                    
                
                    
                <a href="/software/" class="navbar-item "><b>ML Tools</b></a>
                    
                
                    
                <a href="https://zitniklab.hms.harvard.edu/TDC/" class="navbar-item "><b>TDC</b></a>
                    
                
                    
                <a href="/news/" class="navbar-item "><b>News</b></a>
                    
                
                    
                <a href="/jobs/" class="navbar-item "><b>Join Us</b></a>
                    
                
                
            </div>
        </div>
    </div>
</nav>

    
        <section class="hero  is-medium  is-bold is-primary"  style="background: url('/hero.jpg') no-repeat center center; background-size: cover;" >
    <div class="hero-body">
        <div class="container">
            <p class="title is-2">Scientific Community, Conferences, Workshops, Courses and Tutorials</p>
            <p class="subtitle is-3"></p>
            
        </div>
    </div>
</section>
    
    


    <section class="section">
        <div class="container">
            <div class="columns">
                
                <div class="column is-8">
                    
                    
                    
    
    

<div class="contents">
    <div class="menu">
        <p class="menu-label">Contents</p>
        <ul class="menu-list">
  <li><a href="#teaching">Teaching</a>
    <ul>
      <li><a href="#bmi-702-biomedical-artificial-intelligence">BMI 702: Biomedical Artificial Intelligence</a></li>
    </ul>
  </li>
  <li><a href="#research-tutorials">Research Tutorials</a>
    <ul>
      <li><a href="#towards-precision-medicine-with-graph-representation-learning-ismb-2022">Towards Precision Medicine with Graph Representation Learning (ISMB 2022)</a></li>
      <li><a href="#machine-learning-for-drug-development-ijcai-2021">Machine Learning for Drug Development (IJCAI 2021)</a></li>
      <li><a href="#deep-learning-for-network-biology-ismb-2018">Deep Learning for Network Biology (ISMB 2018)</a></li>
      <li><a href="#biomedical-data-fusion-embc-and-bc2-2015">Biomedical Data Fusion (EMBC and [BC]^2 2015)</a></li>
    </ul>
  </li>
  <li><a href="#international-workshops-and-conferences">International Workshops and Conferences</a>
    <ul>
      <li><a href="#ai4science-progress-and-promises-neurips-2022">AI4Science: Progress and Promises (NeurIPS 2022)</a></li>
      <li><a href="#new-frontiers-in-graph-learning-neurips-2022">New Frontiers in Graph Learning (NeurIPS 2022)</a></li>
      <li><a href="#graph-representations-and-algorithms-in-biomedicine-psb-2023">Graph Representations and Algorithms in Biomedicine (PSB 2023)</a></li>
      <li><a href="#ai-for-science-icml-2022">AI for Science (ICML 2022)</a></li>
      <li><a href="#trustworthy-ai-for-healthcare-aaai-2022">Trustworthy AI for Healthcare (AAAI 2022)</a></li>
      <li><a href="#workshop-on-graph-learning-benchmarks-the-web-conference-2022">Workshop on Graph Learning Benchmarks (The Web Conference 2022)</a></li>
      <li><a href="#ai-for-science-neurips-2021">AI for Science (NeurIPS 2021)</a></li>
      <li><a href="#trustworthy-ai-for-healthcare-aaai-2021">Trustworthy AI for Healthcare (AAAI 2021)</a></li>
      <li><a href="#ai-in-health-transferring-and-integrating-knowledge-for-better-health-the-web-conference-2021">AI in Health: Transferring and Integrating Knowledge for Better Health (The Web Conference 2021)</a></li>
      <li><a href="#national-symposium-on-drug-repurposing-for-future-pandemics-2020">National Symposium on Drug Repurposing for Future Pandemics (2020)</a></li>
      <li><a href="#graph-representation-learning-and-beyond-icml-2020">Graph Representation Learning and Beyond (ICML 2020)</a></li>
      <li><a href="#representation-learning-on-graphs-and-manifolds-iclr-2019">Representation Learning on Graphs and Manifolds (ICLR 2019)</a></li>
    </ul>
  </li>
  <li><a href="#research-and-scholarship-meetings">Research and Scholarship Meetings</a>
    <ul>
      <li><a href="#therapeutics-data-commons-user-group-meeting-2022">Therapeutics Data Commons User Group Meeting (2022)</a></li>
      <li><a href="#phd-forum-ecmlpkdd-2020">PhD Forum (ECML/PKDD 2020)</a></li>
    </ul>
  </li>
</ul>
    </div>
</div>




<div class="content">
    <h2 id="teaching">Teaching</h2>

<h3 id="bmi-702-biomedical-artificial-intelligence">BMI 702: Biomedical Artificial Intelligence</h3>

<div class="box has-background-info has-text-white">
Artificial intelligence is poised to enable breakthroughs in science and reshape medicine. This introductory course provides a survey of artificial intelligence for biomedical informatics, covering methods for key data modalities: clinical data, networks, language, and images. It introduces machine learning problems from a practical perspective, focusing on tasks that drive the adoption of machine learning in biology and medicine.

It outlines foundational algorithms and emphasizes the subtleties of working with biomedical data and ways to evaluate and transition machine learning methods into biomedical and clinical implementation. An important consideration in this course is the broader impact of artificial intelligence, particularly topics of bias and fairness, interpretability, and ethical and legal considerations when dealing with artificial intelligence.
</div>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://zitniklab.hms.harvard.edu/BMI702/">Course Website</a>
</div>

<h2 id="research-tutorials">Research Tutorials</h2>

<h3 id="towards-precision-medicine-with-graph-representation-learning-ismb-2022">Towards Precision Medicine with Graph Representation Learning (ISMB 2022)</h3>

<div class="box has-background-info has-text-white">
Graph representation learning has matured immensely as a field within the last few years. Graph machine learning approaches, also known as geometric deep learning, or graph neural networks has become widely used in biomedical applications. This tutorial surveys impact areas in precision medicine (e.g., modeling disease progression, candidate biomarker discovery for targeted therapies, rapid disease diagnostics, treatment regimen recommendations) and highlights new opportunities enabled by these approaches.
</div>

<p>This tutorial was presented at the <a href="https://www.iscb.org/ismb2022">International Conference on Intelligent Systems for Molecular Biology (ISMB)</a>.</p>

<p><img src="/img/biomedgraphml-ismb.png" alt="biomedgraphml-ismb" /></p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="/biomedgraphml">Tutorial Website</a>
</div>

<h3 id="machine-learning-for-drug-development-ijcai-2021">Machine Learning for Drug Development (IJCAI 2021)</h3>

<div class="box has-background-info has-text-white">
Machine learning methods leverage big datasets to support decision-making in all stages of drug development, predict how drugs affect the human body and how they interact with each other, and seek ways to boost clinical trials and detect unwanted side effects. This tutorial covers generative modeling, reinforcement learning, and representation learning with a focus on theoretical foundations of methods and their use for key drug-related problems.
</div>

<p>A variety of machine learning methods are demonstrating their utility at all stages of drug development. These methods use big datasets created from high-throughput screening data and allow prediction of bioactivities for targets and molecular properties, identification of new molecules and repurposing of old drugs with increased levels of accuracy.</p>

<p>We have only just begun to realize the potential of these techniques. If methods were available for all aspects of drug development, they could be used seamlessly to predict whether a chemical compound is likely to ultimately become a drug used in patients. Much research needs to be done before this vision can be realized, modern machine learning may have a fundamental impact on the way drug development is done.</p>

<p>The general process of drug development involves five steps. In short, molecular compounds are filtered through a progressive series of tests, which determine their properties, toxicity, and effectiveness for later stages. Machine learning is increasingly being used to accelerate each of the steps, creating opportunities for reducing resources and time needed to develop new drugs. In this tutorial, we cover key problems in drug development that are amenable to machine learning. In doing so, we present a toolbox of AI algorithms for end-to-end drug development.</p>

<p>This tutorial was presented at the <a href="https://ijcai20.org/">International Joint Conference on Artificial Intelligence (IJCAI)</a>.</p>

<p><img src="/img/drugml-ijcai-sm.png" alt="drugml-ijcai" /></p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="/drugml">Tutorial Website</a>
</div>

<h3 id="deep-learning-for-network-biology-ismb-2018">Deep Learning for Network Biology (ISMB 2018)</h3>

<div class="box has-background-info has-text-white">
Networks are ubiquitous in biology where they encode connectivity patterns at all scales of organization, from molecular to the biome. This tutorial investigates key advancements in representation learning for networks over the last few years, with an emphasis on fundamentally new opportunities in network biology enabled by these advancements.
</div>

<p>Biological networks are powerful resources for the discovery of interactions and emergent properties in biological systems, ranging from single-cell to population level. Network approaches have been used many times to combine and amplify signals from individual genes, and have led to remarkable discoveries in biology, including drug discovery, protein function prediction, disease diagnosis, and precision medicine. Furthermore, these approaches have shown broad utility in uncovering new biology and have contributed to new discoveries in wet laboratory experiments.</p>

<p>Mathematical machinery that is central to these approaches is machine learning on networks. The main challenge in machine learning on networks is to find a way to extract information about interactions between nodes and to incorporate that information into a machine learning model. To extract this information from networks, classic machine learning approaches often rely on summary statistics (e.g., degrees or clustering coefficients) or carefully engineered features to measure local neighborhood structures (e.g., network motifs). These classic approaches can be limited because these hand-engineered features are inflexible, they often do not generalize to networks derived from other organisms, tissues and experimental technologies, and can fail on datasets with low experimental coverage.</p>

<p>Recent years have seen a surge in graph neural network (GNN) approaches that automatically learn to encode network structure into low-dimensional representations, using transformation techniques based on deep learning and nonlinear dimensionality reduction. The idea behind these <em>representation learning</em> approaches is to learn a data transformation function that maps nodes to points in a low-dimensional vector space, also termed <em>embeddings</em>. Representation learning methods have revolutionized the state-of-the-art in network science and the goal of this tutorial is to open the door for these methods to computational biology and bioinformatics.</p>

<p>This tutorial was presented at the <a href="https://www.iscb.org/ismb2018">International Conference on Intelligent Systems for Molecular Biology (ISMB)</a>.</p>

<p><img src="/img/deepnetbio-ismb.png" alt="deepnetbio-ismb" /></p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="http://snap.stanford.edu/deepnetbio-ismb/">Tutorial Slides</a>
    <a class="button is-primary" href="http://snap.stanford.edu/deepnetbio-ismb/ipynb/Human+Disease+Network.html">Demo Diseases</a>
    <a class="button is-primary" href="http://snap.stanford.edu/deepnetbio-ismb/ipynb/Graph+Convolutional+Prediction+of+Protein+Interactions+in+Yeast.html">Demo Yeast</a>
    <a class="button is-primary" href="https://github.com/mims-harvard/gnn-intro">GNN Tutorial</a>
</div>

<h3 id="biomedical-data-fusion-embc-and-bc2-2015">Biomedical Data Fusion (EMBC and [BC]^2 2015)</h3>

<div class="box has-background-info has-text-white">
Because of the complex and interconnected nature of biomedical systems, any single model trained on any single dataset can touch only a small part of the entire biomedical knowledge. It is thus critical to integrate diverse sources of information to gain a comprehensive understanding of the system.
</div>

<p>New technologies have enabled the investigation of biology and human health at an unprecedented scale and in multiple dimensions. These dimensions include a myriad of properties describing genome, epigenome, transcriptome, microbiome, phenotype, and lifestyle. No single data type, however, can capture the complexity of all the factors relevant to understanding a phenomenon such as a disease. Integrative methods that combine data from multiple technologies have thus emerged as critical statistical and computational approaches.</p>

<p>The key challenge in developing such approaches is the identification of effective models to provide a comprehensive and relevant systems view. An ideal method can answer a biological or medical question, identifying important features and predicting outcomes, by harnessing heterogeneous data across several dimensions of biological variation.</p>

<p>This tutorial was presented at the <a href="https://embc.embs.org">International Engineering in Medicine and Biology Conference (EMBC)</a> and at the <a href="https://www.bc2.ch/">Basel Compuational Biology Conference ([BC]^2)</a>.</p>

<p><img src="/img/biomedical-data-fusion.png" alt="biomedical-data-fusion" /></p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="http://helikoid.si/embc15/embc-slides.pdf">Tutorial Slides Part 1</a>
    <a class="button is-primary" href="http://helikoid.si/embc15/embc15-fusion.pdf">Tutorial Slides Part 2</a>
    <a class="button is-primary" href="http://helikoid.si/embc15/embc-handouts.pdf">Tutorial Handouts</a>
</div>

<hr />

<h2 id="international-workshops-and-conferences">International Workshops and Conferences</h2>

<h3 id="ai4science-progress-and-promises-neurips-2022">AI4Science: Progress and Promises (NeurIPS 2022)</h3>

<div class="box has-background-info has-text-white">
For centuries, the method of discovery—the fundamental practice of science that scientists use to explain the natural world systematically and logically—has remained largely the same. Artificial intelligence (AI) and machine learning (ML) hold tremendous promise in having an impact on the way scientific discovery is performed today at the fundamental level. However, to realize this promise, we need to identify priorities and outstanding open questions for the cutting edge of AI going forward.
</div>

<p>This workshop is presented at the the <a href="https://nips.cc/Conferences/2022">International Conference on Neural Information Processing Systems (NeurIPS).</a></p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://ai4sciencecommunity.github.io/index.html">Workshop Website</a>
</div>

<h3 id="new-frontiers-in-graph-learning-neurips-2022">New Frontiers in Graph Learning (NeurIPS 2022)</h3>

<div class="box has-background-info has-text-white">
Despite the success, existing graph learning paradigms have not captured the full spectrum of relationships in the physical and the virtual worlds. For example, in terms of applicability of graph learning algorithms, current graph learning paradigms are often restricted to datasets with explicit graph representations, whereas recent works have shown promise of graph learning methods for applications without explicit graph representations. Further, in terms of generalizability, unlike domains such as computer vision and natural language processing where large-scale pre-trained models generalize across downstream applications with little to no fine-tuning and demonstrate impressive performance, such a paradigm has yet to succeed in the graph learning domain. 
</div>

<p>The primary goal of this workshop is to expand the impact of graph learning beyond the current boundaries. Graph learning is poised to become a broadly useful toolbox for learning and understanding any type of (structured) data. This workshop is presented at the the <a href="https://nips.cc/Conferences/2022">International Conference on Neural Information Processing Systems (NeurIPS).</a></p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://glfrontiers.github.io/">Workshop Website</a>
</div>

<h3 id="graph-representations-and-algorithms-in-biomedicine-psb-2023">Graph Representations and Algorithms in Biomedicine (PSB 2023)</h3>

<div class="box has-background-info has-text-white">
Connectivity is a fundamental property of biological systems: on the cellular level, proteins interact with each other to form protein-protein interaction networks; on the organism level, neurons are arranged in a network; and on a community level, species can have complex relationships with one another that drive the development of an ecosystem. Graphs, mathematical representations modeling entities as vertices and their relationships as edges, have proved useful for understanding biological systems that naturally have such a network structure.  Graph representations and algorithms (often in combination with machine learning techniques) can be used to organize massive amounts of related (and sometimes heterogenous or unstructured) data, and to ultimately to identify patterns that represent novel biological insights.  
</div>

<p>This session will encompass modern developments in graph theory, computational topology, and graph machine learning applied to various fields of biomedicine. This workshop is presented at the <a href="https://psb.stanford.edu/">Pacific Symposium on Biocomputing (PSB)</a>.</p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://sites.google.com/view/psb-2023-graphs-in-biomedicine/home">Session Website</a>
</div>

<h3 id="ai-for-science-icml-2022">AI for Science (ICML 2022)</h3>

<div class="box has-background-info has-text-white">
Machine learning is poised to transform scientific discovery. Despite this promise, critical challenges stifle algorithmic and scientific innovation across scientific disciplines. 
</div>

<p>This workshop brings those challenges to the forefront and discusses which are likely/unlikely to have a broad impact on scientific discovery. With hundreds of AI/ML scientists beginning projects in this area, the workshop brings them together to facilitate community building and consolidate the fast growing area of AI4Science into a vibrant research field.</p>

<p>This workshop is presented at the <a href="https://icml.cc/Conferences/2022/">International Conference on Machine Learning (ICML).</a></p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://ai4sciencecommunity.github.io/icml22.html">Workshop Website</a>
</div>

<h3 id="trustworthy-ai-for-healthcare-aaai-2022">Trustworthy AI for Healthcare (AAAI 2022)</h3>

<div class="box has-background-info has-text-white">
AI for healthcare has emerged into a very active research area in the past few years and has made significant progress. For example, AI methods have achieved human-level performance in skin cancer classification, diabetic eye disease detection, chest radiograph diagnosis, sepsis treatment.  
</div>

<p>While current results are encouraging, few clinical AI solutions are deployed in hospitals or routinely used in the clinic. The major problem is that existing clinical AI methods are not sufficiently trustworthy. Black-box methods generate decisions that are difficult to understand and interpret. Furthermore, existing solutions are sensitive to small perturbations and adversarial attacks, which raises security and privacy concerns. Further, methods can produce results biased against certain populations. This workshop addresses these challenges and puts forward recommendations on how to make clinical AI solutions more trustworthy.</p>

<p>This workshop was presented at the <a href="https://aaai.org/Conferences/AAAI-22/">AAAI Conference on Artificial Intelligence (AAAI 2022)</a>.</p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://taih21.github.io/">Workshop Website</a>
</div>

<h3 id="workshop-on-graph-learning-benchmarks-the-web-conference-2022">Workshop on Graph Learning Benchmarks (The Web Conference 2022)</h3>

<div class="box has-background-info has-text-white">
While graph neural networks achieve promising performance on node classification, graph classification, and link prediction, reported performance gains can only be verified and compared within a limited set of publicly available benchmark datasets. The lack of diversity in benchmark datasets may have biased the development of graph representation learning techniques towards narrow directions.
</div>

<p>By crowdsourcing tasks and datasets, the workshop has increased the diversity of graph learning benchmarks, identified open questions in graph representation learning, and deepened the synergy between graph ML algorithms and benchmark datasets.</p>

<p>The workshop was organized at the <a href="https://www2022.thewebconf.org/">Web Conference (WWW)</a>.</p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://graph-learning-benchmarks.github.io/glb2022">Workshop Website</a>
</div>

<h3 id="ai-for-science-neurips-2021">AI for Science (NeurIPS 2021)</h3>

<div class="box has-background-info has-text-white">
Machine learning has advanced a wide array of scientific disciplines and addressed many problems that previously could not be tackled computationally. Despite this promise, several key challenges remain open, and this workshop brings those gaps to the foreground of AI research. 
</div>

<ul>
  <li>
    <p><strong>Gap 1: Unrealistic methodological assumptions.</strong> While ML researchers strive for methodology advances, they often make unrealistic assumptions that limit real-world adoption. For example, most state-of-the-art molecule generation ML models generate molecules that have low synthesizability.</p>
  </li>
  <li>
    <p><strong>Gap 2: Overlooked scientific questions.</strong> Scientific communities contend with crucial and unsolved problems, but they are not yet formulated as solvable ML tasks and are thus overlooked by the ML community.</p>
  </li>
  <li>
    <p><strong>Gap 3: Limited exploration at the intersection of multiple disciplines.</strong> Solutions to grand challenges often stretch across multiple disciplines. For example, protein structure prediction requires collaboration across physics, chemistry and biology.</p>
  </li>
  <li>
    <p><strong>Gap 4: Science of science.</strong> Core principles of the scientific method have not changed since the 17th century. Can AI reason about the organizing principles of our world in a way that is complementary to the hypothesis-experiment cycle to understand a phenomenon?</p>
  </li>
  <li>
    <p><strong>Gap 5: Responsible use and development of AI for science.</strong> Interest in ML across scientific disciplines has surged, but few ML models have transitioned into practical scientific applications. We plan to present a roadmap and ultimately guidelines for accelerating the translation of ML in science. Translation requires a team of engaged stakeholders and a systematic process from the beginning (problem formulation) to the end (widespread deployment) of ML-based research lifecycle.</p>
  </li>
</ul>

<p>This workshop was presented at the <a href="https://nips.cc/Conferences/2021">International Conference on Neural Information Processing Systems (NeurIPS)</a>.</p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://ai4sciencecommunity.github.io/neurips21.html">Workshop Website</a>
</div>

<h3 id="trustworthy-ai-for-healthcare-aaai-2021">Trustworthy AI for Healthcare (AAAI 2021)</h3>

<div class="box has-background-info has-text-white">
Artificial intelligence for healthcare has emerged as an active research area that has made considerable progress, including achieving human-level performance for skin cancer classification, diabetic eye disease detection, chest radiograph diagnosis, and sepsis treatment. While the trends are encouraging, many open challenges prevent us from directly deploying AI solutions in hospitals and clinical environments. A major open problem is the lack of trust of biomedical practitioners in AI methods. Many AI methods make predictions in a black-box way, making decisions challenging to understand and interpret. Further, today's methods are sensitive to small perturbations and adversarial attacks, raising numerous security and privacy concerns. Finally, AI methods learn to make decisions based on training data, which can include biased human decisions or reflect historical or social inequities. These challenges raise numerous trustworthy issues that we need to address to realize the potential of AI in healthcare.  
</div>

<p>This workshop was presented at the <a href="https://aaai.org/Conferences/AAAI-21/">AAAI Conference on Artificial Intelligence (AAAI 2021)</a>.</p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://taih20.github.io/">Workshop Website</a>
</div>

<h3 id="ai-in-health-transferring-and-integrating-knowledge-for-better-health-the-web-conference-2021">AI in Health: Transferring and Integrating Knowledge for Better Health (The Web Conference 2021)</h3>

<div class="box has-background-info has-text-white">
Rich healthcare data connected by semantic relationships and integrated into knowledge graphs can drive biomedical discovery. Biomedical knowledge graphs can support better cohort identification for clinical trials, risk prediction, precision diagnosis, and can inform new and better decision support workflows. Dramatic increase of healthcare data offers unprecedented opportunities for evidence-based care, yet challenges related to interoperability, learning, and reasoning over healthcare data remain open.
</div>

<p>This workshop was presented at the <a href="https://www2021.thewebconf.org/">Web Conference (WWW)</a>.</p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://suitclub.ischool.utexas.edu/AIHealthWWW2021/index.html">Workshop Website</a>
    <a class="button is-primary" href="https://suitclub.ischool.utexas.edu/AIHealthWWW2021/challenge.html">Data Challenge Website</a>
</div>

<h3 id="national-symposium-on-drug-repurposing-for-future-pandemics-2020">National Symposium on Drug Repurposing for Future Pandemics (2020)</h3>

<div class="box has-background-info has-text-white">
Pandemics demand safe and effective therapies developed and deployed at an unprecedented speed. This symposium, organized on behalf of the National Science Foundation (NSF), provides a forum for scientists and researchers from a variety of fields relevant to therapeutics. Participants discuss ways to expedite the development of therapies by compressing years of work into months or even weeks through automation, artificial intelligence and machine learning, novel data sources, and most recent biotechnology advancements. 
</div>

<p><img src="/img/futuretx20-symposium.png" alt="futuretx20-symposium" /></p>

<p>The symposium brings together leading experts in computer science, biology, statistics, medicine, automation, and regulation. While these areas of expertise are necessary for rapid therapeutic innovation, there is seldom an opportunity for these experts to interact with each other.</p>

<p>Bearing in mind new opportunities and pressing challenges, the symposium provides a roadmap and put forward recommendations on transforming today’s tools into ready-to-use solutions to fight future pathogens.</p>

<p>We announce a new initiative, <a href="https://zitniklab.hms.harvard.edu/TDC/">Therapeutics Data Commons (TDC)</a>, at the symposium <a href="/publications/slides/TDC-futuretx20.pdf">[Slides]</a>.</p>

<p><img src="/img/futuretx20-TDC.png" alt="futuretx20-TDC" /></p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://zitniklab.hms.harvard.edu/TDC/">TDC Website</a>
    <a class="button is-primary" href="https://www.drugsymposium.org/">Symposium Website</a>
</div>

<h3 id="graph-representation-learning-and-beyond-icml-2020">Graph Representation Learning and Beyond (ICML 2020)</h3>

<div class="box has-background-info has-text-white">
Recent years have seen a surge in research on graph representation learning, including techniques for deep graph embeddings, generalizations of CNNs to graph-structured data, and neural message-passing approaches. These advances in graph neural networks and related techniques have led to new state-of-the-art results in numerous domains: chemical synthesis, 3D-vision, recommender systems, question answering, continuous control, self-driving, and social network analysis. 
</div>

<p>This workshop was presented at the <a href="https://icml.cc/Conferences/2020">International Conference on Machine Learning (ICML)</a>.</p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://grlplus.github.io">Workshop Website</a>
</div>

<h3 id="representation-learning-on-graphs-and-manifolds-iclr-2019">Representation Learning on Graphs and Manifolds (ICLR 2019)</h3>

<div class="box has-background-info has-text-white">
Many scientific fields study data with an underlying graph or manifold structure—such as social networks, sensor networks, biomedical knowledge graphs, and meshed surfaces in computer graphics. Recent years have seen a surge in research on these problems—often under the umbrella terms of graph representation learning and geometric deep learning. 
</div>

<p>The need for new optimization methods and neural network architectures that can accommodate these relational and non-Euclidean structures is becoming increasingly clear. In parallel, there is a growing interest in how we can leverage insights from these domains to incorporate new kinds of relational and non-Euclidean inductive biases into deep learning.</p>

<p>This workshop was presented at the <a href="https://iclr.cc/Conferences/2019">International Conference on Learning Representations (ICLR)</a>.</p>

<p><img src="/img/decagon-architecture.png" alt="decagon-architecture" /></p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://rlgm.github.io/">Workshop Website</a>
</div>

<hr />

<h2 id="research-and-scholarship-meetings">Research and Scholarship Meetings</h2>

<h3 id="therapeutics-data-commons-user-group-meeting-2022">Therapeutics Data Commons User Group Meeting (2022)</h3>

<div class="box has-background-info has-text-white">
<a href="https://tdcommons.ai/">Therapeutics Data Commons</a> is an open-science initiative started at Harvard with AI/ML-ready datasets and ML tasks for therapeutics. TDC provides an ecosystem of tools, leaderboards, and community resources, including data functions, strategies for model benchmarking and comparison, meaningful data splits, data processors, public leaderboards, and molecule generation oracles. All resources are integrated via an [open Python library.](https://github.com/mims-harvard/TDC) 
</div>

<p>The lack of high-quality benchmarks impedes the advancement of ML tools for drug discovery. To this end, TDC supports the development of novel ML theory and methods, with a strong bent towards developing the mathematical foundations of which ML algorithms are most suitable for drug discovery applications and why. TDC contains benchmarks for therapeutics ML tasks, including molecular property prediction, molecular interaction prediction, and molecular optimization, all accompanied by extensive programmatic support and leaderboards.</p>

<p>The <a href="https://shoutout.wix.com/so/d1Nv1pC2d#/main">first live user meeting</a> outlined opportunities for how to engage with the TDC community, provided technical background and demos on how TDC supports ML for therapeutics and molecules.</p>

<p align="center">
  <img src="/img/tdc_user_meetup22.jpeg" alt="TDC User Meetup" />
</p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://tdcommons.ai/">Therapeutics Data Commons Website</a>
    <a class="button is-primary" href="https://shoutout.wix.com/so/d1Nv1pC2d#/main">User Group Meetup</a>
</div>

<h3 id="phd-forum-ecmlpkdd-2020">PhD Forum (ECML/PKDD 2020)</h3>

<div class="box has-background-info has-text-white">
PhD Forum provides an environment for junior PhD students to exchange ideas and experiences with peers in an interactive atmosphere and to get constructive feedback from senior researchers in data science, machine learning, and related areas.  
</div>

<p>This meeting took place at the <a href="https://ecmlpkdd2020.net/">European Conference on Machine Learning and Principles and Practice of Knowledge Discovery in Databases (ECML-PKDD)</a>.</p>

<div class="box has-text-centered is-shadowless">
    <a class="button is-primary" href="https://ecmlpkdd2020.net/programme/phdforum/">PhD Forum Website</a>
</div>


</div>
                </div>
                
                <div class="column is-4-desktop is-12-tablet">
                    <p class="title is-4">Latest News</p>

<div class="columns is-multiline">
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2023/02/01/PrimeKG/">PrimeKG published in Scientific Data</a>-->
        <p class="card-header-title">Feb 2023: &nbsp; <span class="has-text-primary">PrimeKG published in Scientific Data</span></p>
<!--        <p class="card-header-item">Feb 2023</p>-->
<!--        <p class="card-footer-item">Feb 1, 2023</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Paper introducing <a href="https://www.nature.com/articles/s41597-023-01960-3">PrimeKG, a multimodal knowledge graph for precision medicine,</a> is published in Scientific Data. <a href="/projects/PrimeKG/">Project website.</a></p>

</p>-->
            <p>Paper introducing <a href="https://www.nature.com/articles/s41597-023-01960-3">PrimeKG, a multimodal knowledge graph for precision medicine,</a> is published in Scientific Data. <a href="/projects/PrimeKG/">Project website.</a></p>


<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2023/02/01/PrimeKG/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Feb 1, 2023</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2023/01/20/GNNDelete/">GNNDelete published at ICLR 2023</a>-->
        <p class="card-header-title">Jan 2023: &nbsp; <span class="has-text-primary">GNNDelete published at ICLR 2023</span></p>
<!--        <p class="card-header-item">Jan 2023</p>-->
<!--        <p class="card-footer-item">Jan 20, 2023</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>New paper on <a href="https://openreview.net/pdf?id=X9yCkmT5Qrl">machine unlearning for graph neural networks</a> accepted at <a href="https://iclr.cc/">ICLR 2023.</a> <a href="/projects/GNNDelete/">Project website.</a></p>
</p>-->
            <p>New paper on <a href="https://openreview.net/pdf?id=X9yCkmT5Qrl">machine unlearning for graph neural networks</a> accepted at <a href="https://iclr.cc/">ICLR 2023.</a> <a href="/projects/GNNDelete/">Project website.</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2023/01/20/GNNDelete/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Jan 20, 2023</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2023/01/02/MutualInteractors/">New Network Principle for Molecular Phenotypes</a>-->
        <p class="card-header-title">Jan 2023: &nbsp; <span class="has-text-primary">New Network Principle for Molecular Phenotypes</span></p>
<!--        <p class="card-header-item">Jan 2023</p>-->
<!--        <p class="card-footer-item">Jan 2, 2023</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>New paper introducing <a href="https://psb.stanford.edu/psb-online/proceedings/psb23/eyuboglu.pdf">mutual interactor-based GNN for molecular phenotype prediction</a> at <a href="https://psb.stanford.edu/">PSB</a>. <a href="/projects/Milieu/">Project website.</a></p>
</p>-->
            <p>New paper introducing <a href="https://psb.stanford.edu/psb-online/proceedings/psb23/eyuboglu.pdf">mutual interactor-based GNN for molecular phenotype prediction</a> at <a href="https://psb.stanford.edu/">PSB</a>. <a href="/projects/Milieu/">Project website.</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2023/01/02/MutualInteractors/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Jan 2, 2023</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/12/14/SHEPHERD/">Can we shorten rare disease diagnostic odyssey?</a>-->
        <p class="card-header-title">Dec 2022: &nbsp; <span class="has-text-primary">Can we shorten rare disease diagnostic odyssey?</span></p>
<!--        <p class="card-header-item">Dec 2022</p>-->
<!--        <p class="card-footer-item">Dec 14, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>New preprint! <a href="https://www.medrxiv.org/content/10.1101/2022.12.07.22283238">Geometric deep learning for diagnosing patients with rare genetic diseases.</a> Implications for using deep learning on sparsely-labeled medical datasets. Thankful for this collaboration with <a href="https://zaklab.org/">Zak Lab.</a> <a href="/projects/SHEPHERD/">Project website.</a></p>
</p>-->
            <p>New preprint! <a href="https://www.medrxiv.org/content/10.1101/2022.12.07.22283238">Geometric deep learning for diagnosing patients with rare genetic diseases.</a> Implications for using deep learning on sparsely-labeled medical datasets. Thankful for this collaboration with <a href="https://zaklab.org/">Zak Lab.</a> <a href="/projects/SHEPHERD/">Project website.</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/12/14/SHEPHERD/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Dec 14, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/11/15/HarvardMedicineTDC/">Can AI transform the way we discover new drugs?</a>-->
        <p class="card-header-title">Nov 2022: &nbsp; <span class="has-text-primary">Can AI transform the way we discover new drugs?</span></p>
<!--        <p class="card-header-item">Nov 2022</p>-->
<!--        <p class="card-footer-item">Nov 15, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Our <a href="https://hms.harvard.edu/news/can-ai-transform-way-we-discover-new-drugs">conversation with Harvard Medicine News</a> highlights recent developments and new features in <a href="https://tdcommons.ai/">Therapeutics Data Commons.</a></p>
</p>-->
            <p>Our <a href="https://hms.harvard.edu/news/can-ai-transform-way-we-discover-new-drugs">conversation with Harvard Medicine News</a> highlights recent developments and new features in <a href="https://tdcommons.ai/">Therapeutics Data Commons.</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/11/15/HarvardMedicineTDC/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Nov 15, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/10/31/GRL-NatureBiomedEng/">New Paper in Nature Biomedical Engineering</a>-->
        <p class="card-header-title">Oct 2022: &nbsp; <span class="has-text-primary">New Paper in Nature Biomedical Engineering</span></p>
<!--        <p class="card-header-item">Oct 2022</p>-->
<!--        <p class="card-footer-item">Oct 31, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>New paper on <a href="https://www.nature.com/articles/s41551-022-00942-x">graph representation learning in biomedicine and healthcare</a> published in Nature Biomedical Engineering.</p>
</p>-->
            <p>New paper on <a href="https://www.nature.com/articles/s41551-022-00942-x">graph representation learning in biomedicine and healthcare</a> published in Nature Biomedical Engineering.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/10/31/GRL-NatureBiomedEng/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Oct 31, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/09/21/TDC-NatureChemBio/">New Paper in Nature Chemical Biology</a>-->
        <p class="card-header-title">Sep 2022: &nbsp; <span class="has-text-primary">New Paper in Nature Chemical Biology</span></p>
<!--        <p class="card-header-item">Sep 2022</p>-->
<!--        <p class="card-footer-item">Sep 21, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Our paper on <a href="https://www.nature.com/articles/s41589-022-01131-2">artificial intelligence foundation for therapeutic science</a> is published in Nature Chemical Biology.</p>
</p>-->
            <p>Our paper on <a href="https://www.nature.com/articles/s41589-022-01131-2">artificial intelligence foundation for therapeutic science</a> is published in Nature Chemical Biology.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/09/21/TDC-NatureChemBio/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Sep 21, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/09/14/TF-C-NeurIPS/">Self-Supervised Pre-Training at NeurIPS 2022</a>-->
        <p class="card-header-title">Sep 2022: &nbsp; <span class="has-text-primary">Self-Supervised Pre-Training at NeurIPS 2022</span></p>
<!--        <p class="card-header-item">Sep 2022</p>-->
<!--        <p class="card-footer-item">Sep 14, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>New paper on self-supervised contrastive pre-training accepted at <a href="https://nips.cc/Conferences/2022">NeurIPS 2022.</a> <a href="https://zitniklab.hms.harvard.edu/projects/TF-C/">Project page.</a> Thankful for this collaboration with the <a href="https://www.ll.mit.edu/">Lincoln National Laboratory</a>.</p>
</p>-->
            <p>New paper on self-supervised contrastive pre-training accepted at <a href="https://nips.cc/Conferences/2022">NeurIPS 2022.</a> <a href="https://zitniklab.hms.harvard.edu/projects/TF-C/">Project page.</a> Thankful for this collaboration with the <a href="https://www.ll.mit.edu/">Lincoln National Laboratory</a>.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/09/14/TF-C-NeurIPS/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Sep 14, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/09/06/BestPaperAwardIEEEVIS/">Best Paper Honorable Mention Award at IEEE VIS</a>-->
        <p class="card-header-title">Sep 2022: &nbsp; <span class="has-text-primary">Best Paper Honorable Mention Award at IEEE VIS</span></p>
<!--        <p class="card-header-item">Sep 2022</p>-->
<!--        <p class="card-footer-item">Sep 6, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Our paper on <a href="https://osf.io/yhdpv">user-centric AI of drug repurposing</a> received the <a href="http://ieeevis.org/year/2022/info/awards/best-paper-awards">Best Paper Honorable Mention Award</a> at IEEE VIS 2022. Thankful for this collaboration with <a href="http://gehlenborglab.org/">Gehlenborg Lab.</a></p>
</p>-->
            <p>Our paper on <a href="https://osf.io/yhdpv">user-centric AI of drug repurposing</a> received the <a href="http://ieeevis.org/year/2022/info/awards/best-paper-awards">Best Paper Honorable Mention Award</a> at IEEE VIS 2022. Thankful for this collaboration with <a href="http://gehlenborglab.org/">Gehlenborg Lab.</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/09/06/BestPaperAwardIEEEVIS/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Sep 6, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/09/05/MGLPreprint/">Multimodal Representation Learning with Graphs</a>-->
        <p class="card-header-title">Sep 2022: &nbsp; <span class="has-text-primary">Multimodal Representation Learning with Graphs</span></p>
<!--        <p class="card-header-item">Sep 2022</p>-->
<!--        <p class="card-footer-item">Sep 5, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>New preprint! We present the <a href="https://arxiv.org/abs/2209.03299">blueprint for graph-centric multimodal learning.</a></p>
</p>-->
            <p>New preprint! We present the <a href="https://arxiv.org/abs/2209.03299">blueprint for graph-centric multimodal learning.</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/09/05/MGLPreprint/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Sep 5, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/08/24/ISMB22-Tutorial-Recording/">On Graph AI for Precision Medicine</a>-->
        <p class="card-header-title">Aug 2022: &nbsp; <span class="has-text-primary">On Graph AI for Precision Medicine</span></p>
<!--        <p class="card-header-item">Aug 2022</p>-->
<!--        <p class="card-footer-item">Aug 24, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>The recording of our <a href="https://zitniklab.hms.harvard.edu/biomedgraphml/">tutorial on using graph AI to advance precision medicine</a> is available. <a href="https://iscb.junolive.co/ismb2022/library/search/OnDemand_ismb2022_tutorial_vt4">Tune into four hours of interactive lectures</a> about state-of-the-art graph AI methods and applications in precision medicine.</p>
</p>-->
            <p>The recording of our <a href="https://zitniklab.hms.harvard.edu/biomedgraphml/">tutorial on using graph AI to advance precision medicine</a> is available. <a href="https://iscb.junolive.co/ismb2022/library/search/OnDemand_ismb2022_tutorial_vt4">Tune into four hours of interactive lectures</a> about state-of-the-art graph AI methods and applications in precision medicine.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/08/24/ISMB22-Tutorial-Recording/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Aug 24, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/08/22/GraphXAI/">Evaluating Explainability for GNNs</a>-->
        <p class="card-header-title">Aug 2022: &nbsp; <span class="has-text-primary">Evaluating Explainability for GNNs</span></p>
<!--        <p class="card-header-item">Aug 2022</p>-->
<!--        <p class="card-footer-item">Aug 22, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>New preprint! We introduce a resource for <a href="https://arxiv.org/abs/2208.09339">broad evaluation of the quality and reliability of GNN explanations</a>, addressing challenges and providing solutions for GNN explainability. <a href="https://zitniklab.hms.harvard.edu/projects/GraphXAI/">Project website.</a></p>
</p>-->
            <p>New preprint! We introduce a resource for <a href="https://arxiv.org/abs/2208.09339">broad evaluation of the quality and reliability of GNN explanations</a>, addressing challenges and providing solutions for GNN explainability. <a href="https://zitniklab.hms.harvard.edu/projects/GraphXAI/">Project website.</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/08/22/GraphXAI/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Aug 22, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/07/16/NeurIPS-NewFrontiersGraphs/">New Frontiers in Graph Learning at NeurIPS</a>-->
        <p class="card-header-title">Jul 2022: &nbsp; <span class="has-text-primary">New Frontiers in Graph Learning at NeurIPS</span></p>
<!--        <p class="card-header-item">Jul 2022</p>-->
<!--        <p class="card-footer-item">Jul 16, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Excited to organize the <a href="https://glfrontiers.github.io/">New Frontiers in Graph Learning</a> workshop at NeurIPS.</p>
</p>-->
            <p>Excited to organize the <a href="https://glfrontiers.github.io/">New Frontiers in Graph Learning</a> workshop at NeurIPS.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/07/16/NeurIPS-NewFrontiersGraphs/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Jul 16, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/07/16/NeurIPS-AI4Science/">AI4Science at NeurIPS</a>-->
        <p class="card-header-title">Jul 2022: &nbsp; <span class="has-text-primary">AI4Science at NeurIPS</span></p>
<!--        <p class="card-header-item">Jul 2022</p>-->
<!--        <p class="card-footer-item">Jul 16, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>We are excited to host the <a href="https://ai4sciencecommunity.github.io/index.html">AI4Science meeting</a> at NeurIPS discussing AI-driven scientific discovery, implementation and verification of AI in science, the influence AI has on the conduct of science, and more.</p>
</p>-->
            <p>We are excited to host the <a href="https://ai4sciencecommunity.github.io/index.html">AI4Science meeting</a> at NeurIPS discussing AI-driven scientific discovery, implementation and verification of AI in science, the influence AI has on the conduct of science, and more.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/07/16/NeurIPS-AI4Science/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Jul 16, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/07/09/ISMB-Tutorial-Materials/">Graph AI for Precision Medicine at ISMB</a>-->
        <p class="card-header-title">Jul 2022: &nbsp; <span class="has-text-primary">Graph AI for Precision Medicine at ISMB</span></p>
<!--        <p class="card-header-item">Jul 2022</p>-->
<!--        <p class="card-footer-item">Jul 9, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Check out our half-day tutorial with resources on <a href="/biomedgraphml/">methods and applications in graph representation learning for precision medicine.</a></p>
</p>-->
            <p>Check out our half-day tutorial with resources on <a href="/biomedgraphml/">methods and applications in graph representation learning for precision medicine.</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/07/09/ISMB-Tutorial-Materials/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Jul 9, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/07/04/SummerInterns/">Welcoming Fellows and Summer Students</a>-->
        <p class="card-header-title">Jul 2022: &nbsp; <span class="has-text-primary">Welcoming Fellows and Summer Students</span></p>
<!--        <p class="card-header-item">Jul 2022</p>-->
<!--        <p class="card-footer-item">Jul 4, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Welcoming a research fellow Julia Balla and three Summer students, Nicholas Ho, Satvik Tripathi, and Isuru Herath.</p>
</p>-->
            <p>Welcoming a research fellow Julia Balla and three Summer students, Nicholas Ho, Satvik Tripathi, and Isuru Herath.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/07/04/SummerInterns/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Jul 4, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/06/20/ContrastivePreTrainingTimeSeries/">Broadly Generalizable Pre-Training Approach</a>-->
        <p class="card-header-title">Jun 2022: &nbsp; <span class="has-text-primary">Broadly Generalizable Pre-Training Approach</span></p>
<!--        <p class="card-header-item">Jun 2022</p>-->
<!--        <p class="card-footer-item">Jun 20, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Excited to share a <a href="https://arxiv.org/abs/2206.08496">preprint on self-supervised method for pre-training</a>. <a href="/projects/TF-C">Project website</a> with evaluation on eight datasets, including electrodiagnostic testing, human daily activity recognition, and health state monitoring.</p>
</p>-->
            <p>Excited to share a <a href="https://arxiv.org/abs/2206.08496">preprint on self-supervised method for pre-training</a>. <a href="/projects/TF-C">Project website</a> with evaluation on eight datasets, including electrodiagnostic testing, human daily activity recognition, and health state monitoring.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/06/20/ContrastivePreTrainingTimeSeries/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Jun 20, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/06/07/NewPostdocs/">Welcoming New Postdocs</a>-->
        <p class="card-header-title">Jun 2022: &nbsp; <span class="has-text-primary">Welcoming New Postdocs</span></p>
<!--        <p class="card-header-item">Jun 2022</p>-->
<!--        <p class="card-footer-item">Jun 7, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Excited to welcome George Dasoulas and Huan He, new postdocs joining us this Summer.</p>
</p>-->
            <p>Excited to welcome George Dasoulas and Huan He, new postdocs joining us this Summer.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/06/07/NewPostdocs/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Jun 7, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/05/20/DasoulasHDSIFellow/">George Named the 2022 Wojcicki Troper Fellow</a>-->
        <p class="card-header-title">May 2022: &nbsp; <span class="has-text-primary">George Named the 2022 Wojcicki Troper Fellow</span></p>
<!--        <p class="card-header-item">May 2022</p>-->
<!--        <p class="card-footer-item">May 20, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Congratulations to <a href="https://www.hdsi.news/post/the-2022-harvard-data-science-initiative-postdoctoral-fellows">George Dasoulas, our incoming postdoctoral fellow, on being named the 2022 Wojcicki Troper HDSI Postdoctoral Fellow</a>. We are delighted to welcome George in our group.</p>
</p>-->
            <p>Congratulations to <a href="https://www.hdsi.news/post/the-2022-harvard-data-science-initiative-postdoctoral-fellows">George Dasoulas, our incoming postdoctoral fellow, on being named the 2022 Wojcicki Troper HDSI Postdoctoral Fellow</a>. We are delighted to welcome George in our group.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/05/20/DasoulasHDSIFellow/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: May 20, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/05/01/PrimeKGPreprint/">New preprint on PrimeKG</a>-->
        <p class="card-header-title">May 2022: &nbsp; <span class="has-text-primary">New preprint on PrimeKG</span></p>
<!--        <p class="card-header-item">May 2022</p>-->
<!--        <p class="card-footer-item">May 1, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p><a href="https://www.biorxiv.org/content/10.1101/2022.05.01.489928">New preprint on building knowledge graphs</a> to enable precision medicine applications.</p>
</p>-->
            <p><a href="https://www.biorxiv.org/content/10.1101/2022.05.01.489928">New preprint on building knowledge graphs</a> to enable precision medicine applications.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/05/01/PrimeKGPreprint/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: May 1, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    

    <div class="column is-12">
        <div class="card">
            <header class="card-header">
        <!--        <a class="card-header-title" href=""></a>-->
            <p class="card-header-title"><span class="has-text-primary">Tweets</span></p>
        <!--        <p class="card-header-item"></p>-->
        <!--        <p class="card-footer-item"></p>-->
            </header>
        <div class="card-content">
        <!--        <div class="content">-->
        <!--            -->
        <!--            <p></p>-->
        <a class="twitter-timeline" data-width="500" data-height="500" data-theme="light" href="https://twitter.com/marinkazitnik?ref_src=twsrc%5Etfw" data-chrome="noheader transparent nofooter nofooter noborders">Tweets by marinkazitnik</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
        <!--        </div>-->
        <!--        <div class="has-text-centered">-->
        <!--            <a href="" class="button is-primary">Read more</a>-->
        <!--        </div>-->
            </div>
        </div>
    </div>
</div>




                </div>
                
            </div>
        </div>
    </section>
    
    <script src="/assets/js/app.js" type="text/javascript"></script><!-- footer scripts -->
<div style="background-color:#A41034">
        <div class="content is-normal has-text-centered">
            <p style="color:white;padding-top:20px;padding-bottom:20px;"><a href="https://scholar.harvard.edu/marinka" style="color:white"><b>Zitnik Lab</b></a>
                &nbsp;&middot;&nbsp; <a href="#" style="color:white"><b>Artificial Intelligence in Medicine and Science</b></a>
                &nbsp;&middot;&nbsp; <a href="https://harvard.edu" style="color:white"><b>Harvard</b></a>
                &nbsp;&middot;&nbsp; <a href="https://dbmi.hms.harvard.edu/" style="color:white"><b>Department of Biomedical Informatics</b></a></p>
        </div>
</div></body>
</html>

