
<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Combinatorial Prediction of Therapeutic Perturbations Using Causally-Inspired Neural Networks - Zitnik Lab</title>
    <link rel="stylesheet" href="/assets/css/app.css">
    <link rel="shortcut icon" type="image/png"
           href="/favicon.png" 
    />
    <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>
    <!-- Begin Jekyll SEO tag v2.6.1 -->
<title>Combinatorial Prediction of Therapeutic Perturbations Using Causally-Inspired Neural Networks | Zitnik Lab</title>
<meta name="generator" content="Jekyll v3.8.6" />
<meta property="og:title" content="Combinatorial Prediction of Therapeutic Perturbations Using Causally-Inspired Neural Networks" />
<meta name="author" content="Marinka Zitnik" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="phenotype-driven drug discovery, target identification, drug discovery, graph neural network, geometric deep learning" />
<meta property="og:description" content="phenotype-driven drug discovery, target identification, drug discovery, graph neural network, geometric deep learning" />
<link rel="canonical" href="https://zitniklab.hms.harvard.edu/projects/PDGrapher/" />
<meta property="og:url" content="https://zitniklab.hms.harvard.edu/projects/PDGrapher/" />
<meta property="og:site_name" content="Zitnik Lab" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="Combinatorial Prediction of Therapeutic Perturbations Using Causally-Inspired Neural Networks" />
<meta name="twitter:site" content="@marinkazitnik" />
<meta name="twitter:creator" content="@Marinka Zitnik" />
<script type="application/ld+json">
{"url":"https://zitniklab.hms.harvard.edu/projects/PDGrapher/","author":{"@type":"Person","name":"Marinka Zitnik"},"headline":"Combinatorial Prediction of Therapeutic Perturbations Using Causally-Inspired Neural Networks","description":"phenotype-driven drug discovery, target identification, drug discovery, graph neural network, geometric deep learning","@type":"WebPage","@context":"https://schema.org"}</script>
<!-- End Jekyll SEO tag -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-162129505-1"></script>
<script>
  window['ga-disable-UA-162129505-1'] = window.doNotTrack === "1" || navigator.doNotTrack === "1" || navigator.doNotTrack === "yes" || navigator.msDoNotTrack === "1";
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());
  gtag('config', 'UA-162129505-1');
</script><!-- head scripts --></head>

  <body>
    
<nav class="navbar is-primary" >
    <div class="container">
        <div class="navbar-brand">
            <a href="/" class="navbar-item"><b>
                Zitnik Lab
            </b></a>
            <a role="button" class="navbar-burger burger" aria-label="menu" aria-expanded="false" data-target="navMenu">
                <span aria-hidden="true"></span>
                <span aria-hidden="true"></span>
                <span aria-hidden="true"></span>
            </a>
        </div>
        <div class="navbar-menu" id="navMenu">
            <div class="navbar-start">
<!--                <a href="/" class="navbar-item "><b>Home</b></a>-->
                
                
                    
                    <div class="navbar-item has-dropdown is-hoverable">
                        <a href="/#" class="navbar-link "><b>About</b></a>
                        <div class="navbar-dropdown">
                            
                            <a href="/bio/" class="navbar-item "><b>Bio</b></a>
                            
                            <a href="/contact/" class="navbar-item "><b>Contact</b></a>
                            
                            <a href="/talks/" class="navbar-item "><b>Recent Talks</b></a>
                            
                        </div>
                    </div>
                    
                
                    
                <a href="/research/" class="navbar-item "><b>Research</b></a>
                    
                
                    
                <a href="/publications/" class="navbar-item "><b>Publications</b></a>
                    
                
                    
                <a href="/people/" class="navbar-item "><b>Members</b></a>
                    
                
                    
                <a href="/meetings/" class="navbar-item "><b>Education</b></a>
                    
                
                    
                <a href="/DMAI/" class="navbar-item "><b>DMAI</b></a>
                    
                
                    
                <a href="/data/" class="navbar-item "><b>Datasets</b></a>
                    
                
                    
                <a href="/software/" class="navbar-item "><b>AI Models</b></a>
                    
                
                    
                <a href="https://zitniklab.hms.harvard.edu/TDC/" class="navbar-item "><b>TDC</b></a>
                    
                
                    
                <a href="/news/" class="navbar-item "><b>News</b></a>
                    
                
                    
                <a href="/jobs/" class="navbar-item "><b>Join Us</b></a>
                    
                
                
            </div>
        </div>
    </div>
</nav>

    
        <section class="hero  is-medium  is-bold is-primary"  style="background: url('/hero.jpg') no-repeat center center; background-size: cover;" >
    <div class="hero-body">
        <div class="container">
            <p class="title is-2">Combinatorial Prediction of Therapeutic Perturbations Using Causally-Inspired Neural Networks</p>
            <p class="subtitle is-3"></p>
            
        </div>
    </div>
</section>
    
    


    <section class="section">
        <div class="container">
            <div class="columns">
                
                <div class="column is-8">
                    
                    
                    
<div class="content">
    <div class="box has-background-info has-text-white">

<p>
As an alternative to target-driven drug discovery, phenotype-driven approaches identify compounds that counteract the overall disease effects by analyzing phenotypic signatures. Our study introduces a novel approach to this field, aiming to expand the search space for new therapeutic agents. We introduce PDGrapher, a causally-inspired graph neural network model designed to predict arbitrary perturbagens – sets of therapeutic targets – capable of reversing disease effects. Unlike existing methods that learn responses to perturbations, PDGrapher solves the inverse problem, which is to infer the perturbagens necessary to achieve a specific response – i.e., directly predicting perturbagens by learning which perturbations elicit a desired response. 
</p>

<p>
Experiments across eight datasets of genetic and chemical perturbations show that PDGrapher successfully predicted effective perturbagens in up to 10% additional test samples and ranked therapeutic targets up to 35% higher than competing methods. A key innovation of PDGrapher is its direct prediction capability, which contrasts with the indirect, computationally intensive models traditionally used in phenotype-driven drug discovery that only predict changes in phenotypes due to perturbations. The direct approach enables PDGrapher to train up to 30 times faster, representing a significant leap in efficiency. Our results suggest that PDGrapher can advance phenotype-driven drug discovery, offering a fast and comprehensive approach to identifying therapeutically useful perturbations.
</p>
</div>

<p>Target-driven drug discovery, predominant since the 1990s, focuses on the design of highly specific compounds against disease-associated targets such as proteins or enzymes.</p>

<ul>
  <li>A prime example of target-driven drug discovery is the development of small molecule kinase inhibitors like Imatinib. Imatinib halts the progression of chronic myeloid leukemia (CML) by inhibiting the BCR-ABL tyrosine kinase, a mutated protein that drives uncontrolled proliferation of leukemic cells in CML patients.</li>
  <li>A second notable example are monoclonal antibodies like Trastuzumab which specifically targets the HER2 receptor, a protein overexpressed in certain types of breast cancer. Trastuzumab inhibits cell proliferation while engaging the body’s immune system to initiate an attack against the cancer.</li>
</ul>

<p>These examples illustrate the success of target-driven approaches, yet the past decade has seen a revival of phenotype-driven approaches. This shift has been fueled by the observation that the majority of the first-in-class drugs approved by the US FDA between 1999 and 2008 were discovered empirically without a drug target hypothesis.</p>

<p>Instead of the <strong>one drug, one gene, one disease</strong> model of target-driven approaches, phenotype-driven approaches aim to identify compounds, or perturbagens, that reverse phenotypic disease effects as measured by high-throughput screening (HTS) assays. Notable recent successes include Ivacaftor, the first available therapy treating the underlying cause of cystic fibrosis, and Risdiplam, the first oral medicine approved to treat spinal muscular atrophy.</p>

<h2 id="overview-of-pdgrapher">Overview of PDGrapher</h2>

<p>Current phenotype-driven lead discovery methods are limited as they identify potential leads by predicting and matching gene expression response signatures with known disease signatures. We propose an alternative approach where the objective is to determine which genes a new perturbagen should target to reverse disease effects as represented by a disease signature. We term this approach <strong>combinatorial prediction of therapeutic targets</strong> due to its focus on predicting gene combinations.</p>

<p>The problem of finding which elements of a system should be perturbed to achieve a desired outcome is typically referred to as <em>optimal intervention design</em> within the causal discovery literature. Here, we build on this concept to introduce PDGrapher, an approach for the combinatorial prediction of therapeutic targets to shift cell gene expression from an initial to a treated state in genetic and chemical intervention datasets. Our predictive task is formulated using a causal model, where genes are the nodes in a causal graph and structural equations encode their causal relationships. The goal is to identify a set of genes a perturbagen should target to transition nodes states from diseased to treated.</p>

<center>
<img src="/img/PDGrapher-overview.png" />
</center>

<p>We consider protein-protein interaction networks (PPI) or gene regulatory networks (GRN) as approximations of the causal graph, assuming no unobserved confounders. We tackle our objective through a representation learning approach, using graph neural network (GNN)-based models to implicitly represent the structural equations. For a given pair of diseased and treated samples, PDGrapher outputs a ranking of genes, with the top-predicted genes identified as the primary therapeutic targets that shift gene expression phenotypes from a diseased to a treated state.</p>

<p>We evaluate PDGrapher across four datasets of genetic and chemical interventions in held out folds containing novel samples, and held out folds containing novel samples and cell lines:</p>

<ul>
  <li>PDGrapher ranks ground-truth therapeutic targets up to 35% higher in chemical intervention datasets and 16% higher in genetic intervention datasets in held out folds containing novel samples, compared with existing approaches.</li>
  <li>In held out folds containing both novel samples and cell lines, PDGrapher maintained its robust performance.</li>
  <li>A key innovation of PDGrapher is its ability to directly predict perturbagens that can shift gene expression from diseased to treated states in contrast with prevailing approaches that indirectly predict perturbagens through extensive computational modeling of cell responses. This enables up to 30 times faster training than existing methods.</li>
  <li>Therapeutic targets predicted by PDGrapher in chemical intervention datasets agree well with ground-truth therapeutic targets, indicating that PDGrapher can elucidate the mode of action for chemical perturbagens and potentially  enhance phenotype-driven lead discovery.</li>
</ul>

<center>
<img src="/img/PDGrapher-example.png" />
</center>

<h2 id="publication">Publication</h2>

<p><a href="https://www.biorxiv.org/content/10.1101/2024.01.03.573985">Combinatorial Prediction of Therapeutic Perturbations Using Causally-Inspired Neural Networks</a><br />
Guadalupe Gonzalez, Isuru Herath, Kirill Veselkov, Michael Bronstein and Marinka Zitnik<br />
<em>In Review</em> 2024 <a href="https://www.biorxiv.org/content/10.1101/2024.01.03.573985">[bioRxiv]</a></p>

<div class="highlighter-rouge"><div class="highlight"><pre class="highlight"><code>@article{gonzalez2024combinatorial,
  title={Combinatorial Prediction of Therapeutic Perturbations Using Causally-Inspired Neural Networks},
  author={Gonzalez, Guadalupe and Herath, Isuru and Veselkov, Kirill and Bronstein, Michael and Zitnik, Marinka},
  journal={bioRxiv},
  url={},
  year={2024}
}
</code></pre></div></div>

<h2 id="code-availability">Code Availability</h2>

<p>Pytorch implementation of PDGrapher is available in the <a href="https://github.com/mims-harvard/PDGrapher">GitHub repository</a>.</p>

<h2 id="data-availability">Data Availability</h2>

<p>Processed data, including the cell line gene expression dataset, protein-protein interaction network, drug targets, and disease-associated genes are shared with the research community:</p>

<ul>
  <li>Protein-protein interaction network data are from <a href="https://downloads.thebiogrid.org/File/BioGRID/Release-Archive/BIOGRID-3.5.186/BIOGRID-MV-Physical-3.5.186.tab3.zip">BioGrid</a>, <a href="https://www.science.org/doi/suppl/10.1126/science.1257601/suppl_file/datasets_s1-s4.zip">Interactome</a> and <a href="http://www.interactome-atlas.org/data/HuRI.tsv">Interactome Atlas</a>.</li>
  <li>Gene expression datasets are from <a href="https://clue.io/releases/data-dashboard">CLUE</a>.</li>
  <li>Disease-associated genes are from <a href="https://cancer.sanger.ac.uk/cell_lines/archive-download#:~:text=Complete%20mutation%20data">COSMIC</a> and <a href="https://cancer.sanger.ac.uk/cosmic/curation">COSMIC Curation</a>.</li>
  <li>Drug target information is extracted from <a href="https://go.drugbank.com/releases/5-1-9">DrugBank</a>.</li>
  <li>List of cancer drugs is obtained from <a href="https://www.cancer.gov/about-cancer/treatment/types/targeted-therapies/approved-drug-list">NCI</a>.</li>
</ul>

<p>Processed dataset is available through <a href="https://figshare.com/articles/dataset/Combinatorial_prediction_of_therapeutic_targets_using_a_causally-inspired_neural_network/24798855">Figshare</a>.</p>

<h2 id="authors">Authors</h2>

<ul>
  <li><a href="https://www.guadalupegonzalez.io/">Guadalupe Gonzalez</a></li>
  <li><a href="#">Isuru Herath</a></li>
  <li><a href="https://www.imperial.ac.uk/people/kirill.veselkov04">Kirill Veselkov</a></li>
  <li><a href="https://www.cs.ox.ac.uk/people/michael.bronstein/">Michael Bronstein</a></li>
  <li><a href="https://zitniklab.hms.harvard.edu/">Marinka Zitnik</a></li>
</ul>

</div>
                </div>
                
                <div class="column is-4-desktop is-12-tablet">
                    <p class="title is-4">Latest News</p>

<div class="columns is-multiline">
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2024/07/27/PINNACLENatureMethods/">PINNACLE in Nature Methods</a>-->
        <p class="card-header-title">Jul 2024: &nbsp; <span class="has-text-primary">PINNACLE in Nature Methods</span></p>
<!--        <p class="card-header-item">Jul 2024</p>-->
<!--        <p class="card-footer-item">Jul 27, 2024</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>PINNACLE contextual AI model is published in Nature Methods. <a href="https://www.nature.com/articles/s41592-024-02341-3">Paper.</a> <a href="https://www.nature.com/articles/s41592-024-02342-2">Research Briefing.</a> <a href="https://zitniklab.hms.harvard.edu/projects/PINNACLE/">Project website.</a></p>
</p>-->
            <p>PINNACLE contextual AI model is published in Nature Methods. <a href="https://www.nature.com/articles/s41592-024-02341-3">Paper.</a> <a href="https://www.nature.com/articles/s41592-024-02342-2">Research Briefing.</a> <a href="https://zitniklab.hms.harvard.edu/projects/PINNACLE/">Project website.</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2024/07/27/PINNACLENatureMethods/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Jul 27, 2024</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2024/07/15/DigitalTwins/">Digital Twins as Global Health and Disease Models of Individuals</a>-->
        <p class="card-header-title">Jul 2024: &nbsp; <span class="has-text-primary">Digital Twins as Global Health and Disease Models of Individuals</span></p>
<!--        <p class="card-header-item">Jul 2024</p>-->
<!--        <p class="card-footer-item">Jul 15, 2024</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p><a href="https://www.preprints.org/manuscript/202406.0357/">Paper on digitial twins</a> outlining strategies to leverage molecular and computational techniques to construct dynamic digital twins on the scale of populations to individuals.</p>
</p>-->
            <p><a href="https://www.preprints.org/manuscript/202406.0357/">Paper on digitial twins</a> outlining strategies to leverage molecular and computational techniques to construct dynamic digital twins on the scale of populations to individuals.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2024/07/15/DigitalTwins/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Jul 15, 2024</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2024/07/14/GraphAdversarialDiffusion/">Graph Diffusion Convolutions at ICML</a>-->
        <p class="card-header-title">Jul 2024: &nbsp; <span class="has-text-primary">Graph Diffusion Convolutions at ICML</span></p>
<!--        <p class="card-header-item">Jul 2024</p>-->
<!--        <p class="card-footer-item">Jul 14, 2024</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p><a href="https://arxiv.org/abs/2406.02059">Graph diffusion convolution is a geometric deep learning architecture that aggregates information from higher-order network neighbors through a generalized graph diffusion</a> to enhance model robustness to noisy and incomplete datasets. <a href="https://arxiv.org/abs/2406.02059">Paper at ICML.</a></p>
</p>-->
            <p><a href="https://arxiv.org/abs/2406.02059">Graph diffusion convolution is a geometric deep learning architecture that aggregates information from higher-order network neighbors through a generalized graph diffusion</a> to enhance model robustness to noisy and incomplete datasets. <a href="https://arxiv.org/abs/2406.02059">Paper at ICML.</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2024/07/14/GraphAdversarialDiffusion/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Jul 14, 2024</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2024/07/13/TrialBenchLLM3DStructure/">Three Papers: TrialBench, 3D Structure Design, LLM Editing</a>-->
        <p class="card-header-title">Jul 2024: &nbsp; <span class="has-text-primary">Three Papers: TrialBench, 3D Structure Design, LLM Editing</span></p>
<!--        <p class="card-header-item">Jul 2024</p>-->
<!--        <p class="card-footer-item">Jul 13, 2024</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>New papers on <a href="https://arxiv.org/abs/2407.00631">TrialBench with AI-ready clinical trial datasets</a>, <a href="https://arxiv.org/abs/2406.03403">structure-based drug design benchmark</a>, and <a href="https://arxiv.org/abs/2407.06483">composable interventions for language models</a>.</p>
</p>-->
            <p>New papers on <a href="https://arxiv.org/abs/2407.00631">TrialBench with AI-ready clinical trial datasets</a>, <a href="https://arxiv.org/abs/2406.03403">structure-based drug design benchmark</a>, and <a href="https://arxiv.org/abs/2407.06483">composable interventions for language models</a>.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2024/07/13/TrialBenchLLM3DStructure/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Jul 13, 2024</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2024/06/23/TDC2/">TDC-2: Multimodal Foundation for Therapeutics</a>-->
        <p class="card-header-title">Jun 2024: &nbsp; <span class="has-text-primary">TDC-2: Multimodal Foundation for Therapeutics</span></p>
<!--        <p class="card-header-item">Jun 2024</p>-->
<!--        <p class="card-footer-item">Jun 23, 2024</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p><a href="https://tdcommons.ai/">The Commons 2.0 (TDC-2)</a> is an overhaul of Therapeutic Data Commons to catalyze research in multimodal models for drug discovery by unifying single-cell biology of diseases, biochemistry of molecules, and effects of drugs through multimodal datasets, AI-powered API endpoints, new tasks and benchmarks. <a href="https://www.biorxiv.org/content/10.1101/2024.06.12.598655v2">Our paper.</a></p>
</p>-->
            <p><a href="https://tdcommons.ai/">The Commons 2.0 (TDC-2)</a> is an overhaul of Therapeutic Data Commons to catalyze research in multimodal models for drug discovery by unifying single-cell biology of diseases, biochemistry of molecules, and effects of drugs through multimodal datasets, AI-powered API endpoints, new tasks and benchmarks. <a href="https://www.biorxiv.org/content/10.1101/2024.06.12.598655v2">Our paper.</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2024/06/23/TDC2/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Jun 23, 2024</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2024/05/29/BroadMIA/">Broad MIA: Protein Language Models</a>-->
        <p class="card-header-title">May 2024: &nbsp; <span class="has-text-primary">Broad MIA: Protein Language Models</span></p>
<!--        <p class="card-header-item">May 2024</p>-->
<!--        <p class="card-footer-item">May 29, 2024</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Check out our Broad’s seminars on <a href="https://youtu.be/LcLmvtXHI1s?si=GABqGdFt5ze9leT_">Multimodal protein language models for deciphering protein function.</a></p>
</p>-->
            <p>Check out our Broad’s seminars on <a href="https://youtu.be/LcLmvtXHI1s?si=GABqGdFt5ze9leT_">Multimodal protein language models for deciphering protein function.</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2024/05/29/BroadMIA/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: May 29, 2024</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2024/05/28/KnowingAGene/">On Knowing a Gene in Cell Systems</a>-->
        <p class="card-header-title">May 2024: &nbsp; <span class="has-text-primary">On Knowing a Gene in Cell Systems</span></p>
<!--        <p class="card-header-item">May 2024</p>-->
<!--        <p class="card-footer-item">May 28, 2024</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>We shed light on <a href="https://www.sciencedirect.com/science/article/pii/S2405471224001236">distributional gene representations and their potential applications in biology to characterize gene function from a broader and more holistic perspective.</a></p>
</p>-->
            <p>We shed light on <a href="https://www.sciencedirect.com/science/article/pii/S2405471224001236">distributional gene representations and their potential applications in biology to characterize gene function from a broader and more holistic perspective.</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2024/05/28/KnowingAGene/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: May 28, 2024</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2024/04/04/BIomedicalAIAgents/">Biomedical AI Agents</a>-->
        <p class="card-header-title">Apr 2024: &nbsp; <span class="has-text-primary">Biomedical AI Agents</span></p>
<!--        <p class="card-header-item">Apr 2024</p>-->
<!--        <p class="card-footer-item">Apr 4, 2024</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>We envision <a href="https://arxiv.org/abs/2404.02831">‘AI scientists’ as systems capable of skeptical learning and reasoning that empower biomedical research through collaborative agents</a> that integrate machine learning tools with experimental platforms.</p>
</p>-->
            <p>We envision <a href="https://arxiv.org/abs/2404.02831">‘AI scientists’ as systems capable of skeptical learning and reasoning that empower biomedical research through collaborative agents</a> that integrate machine learning tools with experimental platforms.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2024/04/04/BIomedicalAIAgents/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Apr 4, 2024</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2024/03/23/EfficientMLSeminar/">Efficient ML Seminar Series</a>-->
        <p class="card-header-title">Mar 2024: &nbsp; <span class="has-text-primary">Efficient ML Seminar Series</span></p>
<!--        <p class="card-header-item">Mar 2024</p>-->
<!--        <p class="card-footer-item">Mar 23, 2024</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>We started a <a href="https://efficientml.org/">Harvard University Efficient ML Seminar Series</a>. Congrats to Jonathan for spearheading this initiative. <a href="https://www.harvardmagazine.com/2024/03/scaling-artificial-intelligence">Harvard Magazine</a> covered the first meeting focusing on LLMs.</p>
</p>-->
            <p>We started a <a href="https://efficientml.org/">Harvard University Efficient ML Seminar Series</a>. Congrats to Jonathan for spearheading this initiative. <a href="https://www.harvardmagazine.com/2024/03/scaling-artificial-intelligence">Harvard Magazine</a> covered the first meeting focusing on LLMs.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2024/03/23/EfficientMLSeminar/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Mar 23, 2024</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2024/03/04/UniTS/">UniTS - Unified Time Series Model</a>-->
        <p class="card-header-title">Mar 2024: &nbsp; <span class="has-text-primary">UniTS - Unified Time Series Model</span></p>
<!--        <p class="card-header-item">Mar 2024</p>-->
<!--        <p class="card-footer-item">Mar 4, 2024</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p><a href="https://arxiv.org/abs/2403.00131">UniTS is a unified time series model</a> that can process classification, forecasting, anomaly detection and imputation tasks within a single model with no task-specific modules. UniTS has zero-shot, few-shot, and prompt learning capabilities. <a href="https://zitniklab.hms.harvard.edu/projects/UniTS/">Project website.</a></p>
</p>-->
            <p><a href="https://arxiv.org/abs/2403.00131">UniTS is a unified time series model</a> that can process classification, forecasting, anomaly detection and imputation tasks within a single model with no task-specific modules. UniTS has zero-shot, few-shot, and prompt learning capabilities. <a href="https://zitniklab.hms.harvard.edu/projects/UniTS/">Project website.</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2024/03/04/UniTS/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Mar 4, 2024</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2024/03/02/WeintraubAward/">Weintraub Graduate Student Award</a>-->
        <p class="card-header-title">Mar 2024: &nbsp; <span class="has-text-primary">Weintraub Graduate Student Award</span></p>
<!--        <p class="card-header-item">Mar 2024</p>-->
<!--        <p class="card-footer-item">Mar 2, 2024</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Michelle receives the 2024 Harold M. Weintraub Graduate Student Award. The award recognizes exceptional achievement in graduate studies in biological sciences. <a href="https://dbmi.hms.harvard.edu/news/li-receives-weintraub-graduate-student-award">News Story.</a> Congratulations!</p>
</p>-->
            <p>Michelle receives the 2024 Harold M. Weintraub Graduate Student Award. The award recognizes exceptional achievement in graduate studies in biological sciences. <a href="https://dbmi.hms.harvard.edu/news/li-receives-weintraub-graduate-student-award">News Story.</a> Congratulations!</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2024/03/02/WeintraubAward/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Mar 2, 2024</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2024/03/01/PocketGen/">PocketGen - Generating Full-Atom Ligand-Binding Protein Pockets</a>-->
        <p class="card-header-title">Mar 2024: &nbsp; <span class="has-text-primary">PocketGen - Generating Full-Atom Ligand-Binding Protein Pockets</span></p>
<!--        <p class="card-header-item">Mar 2024</p>-->
<!--        <p class="card-footer-item">Mar 1, 2024</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p><a href="https://www.biorxiv.org/content/10.1101/2024.02.25.581968">PocketGen is a deep generative model</a> that generates residue sequence and full-atom structure of protein pockets, maximizing binding to ligands. <a href="https://zitniklab.hms.harvard.edu/projects/PocketGen/">Project website.</a></p>
</p>-->
            <p><a href="https://www.biorxiv.org/content/10.1101/2024.02.25.581968">PocketGen is a deep generative model</a> that generates residue sequence and full-atom structure of protein pockets, maximizing binding to ligands. <a href="https://zitniklab.hms.harvard.edu/projects/PocketGen/">Project website.</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2024/03/01/PocketGen/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Mar 1, 2024</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2024/02/28/SPECTRA-model-generalizability/">SPECTRA - Generalizability of Molecular AI</a>-->
        <p class="card-header-title">Feb 2024: &nbsp; <span class="has-text-primary">SPECTRA - Generalizability of Molecular AI</span></p>
<!--        <p class="card-header-item">Feb 2024</p>-->
<!--        <p class="card-footer-item">Feb 28, 2024</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p><a href="https://www.biorxiv.org/content/10.1101/2024.02.25.581982v1">SPECTRA is an approach for holistic evaluation of how AI models</a> generalize to new molecular datasets. <a href="https://zitniklab.hms.harvard.edu/projects/SPECTRA/">Project website.</a></p>
</p>-->
            <p><a href="https://www.biorxiv.org/content/10.1101/2024.02.25.581982v1">SPECTRA is an approach for holistic evaluation of how AI models</a> generalize to new molecular datasets. <a href="https://zitniklab.hms.harvard.edu/projects/SPECTRA/">Project website.</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2024/02/28/SPECTRA-model-generalizability/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Feb 28, 2024</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2024/02/02/KanebFellowship/">Kaneb Fellowship Award</a>-->
        <p class="card-header-title">Feb 2024: &nbsp; <span class="has-text-primary">Kaneb Fellowship Award</span></p>
<!--        <p class="card-header-item">Feb 2024</p>-->
<!--        <p class="card-footer-item">Feb 2, 2024</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>The lab receives the <a href="#">John and Virginia Kaneb Fellowship Award at Harvard Medical School</a> to enhance research progress in the lab.</p>

</p>-->
            <p>The lab receives the <a href="#">John and Virginia Kaneb Fellowship Award at Harvard Medical School</a> to enhance research progress in the lab.</p>


<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2024/02/02/KanebFellowship/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Feb 2, 2024</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2024/02/02/NSFCAREER/">NSF CAREER Award</a>-->
        <p class="card-header-title">Feb 2024: &nbsp; <span class="has-text-primary">NSF CAREER Award</span></p>
<!--        <p class="card-header-item">Feb 2024</p>-->
<!--        <p class="card-footer-item">Feb 2, 2024</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>The lab receives the <a href="#">NSF CAREER Award</a> for our research in geometric deep learning to facilitate algorithmic and scientific advances in therapeutics.</p>

</p>-->
            <p>The lab receives the <a href="#">NSF CAREER Award</a> for our research in geometric deep learning to facilitate algorithmic and scientific advances in therapeutics.</p>


<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2024/02/02/NSFCAREER/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Feb 2, 2024</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2024/02/01/HMSDeansAI/">Dean’s Innovation Award in AI</a>-->
        <p class="card-header-title">Feb 2024: &nbsp; <span class="has-text-primary">Dean’s Innovation Award in AI</span></p>
<!--        <p class="card-header-item">Feb 2024</p>-->
<!--        <p class="card-footer-item">Feb 1, 2024</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>The lab receives <a href="#">Dean’s Innovation Award for the Use of Artificial Intelligence in Research</a>. <a href="https://hms.harvard.edu/news/dean-announces-winners-inaugural-ai-grants">HMS News Story.</a></p>

</p>-->
            <p>The lab receives <a href="#">Dean’s Innovation Award for the Use of Artificial Intelligence in Research</a>. <a href="https://hms.harvard.edu/news/dean-announces-winners-inaugural-ai-grants">HMS News Story.</a></p>


<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2024/02/01/HMSDeansAI/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Feb 1, 2024</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2024/01/25/MultimodalLearning/">AI's Prospects in Nature Machine Intelligence</a>-->
        <p class="card-header-title">Jan 2024: &nbsp; <span class="has-text-primary">AI's Prospects in Nature Machine Intelligence</span></p>
<!--        <p class="card-header-item">Jan 2024</p>-->
<!--        <p class="card-footer-item">Jan 25, 2024</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>We discussed <a href="https://www.nature.com/articles/s42256-023-00784-5">AI’s 2024 prospects with Nature Machine Intelligence</a>, covering LLM progress, multimodal AI, multi-task agents, and how to bridge the digital divide across communities and world regions.</p>

</p>-->
            <p>We discussed <a href="https://www.nature.com/articles/s42256-023-00784-5">AI’s 2024 prospects with Nature Machine Intelligence</a>, covering LLM progress, multimodal AI, multi-task agents, and how to bridge the digital divide across communities and world regions.</p>


<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2024/01/25/MultimodalLearning/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Jan 25, 2024</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2024/01/01/PDGrapher/">Combinatorial Therapeutic Perturbations</a>-->
        <p class="card-header-title">Jan 2024: &nbsp; <span class="has-text-primary">Combinatorial Therapeutic Perturbations</span></p>
<!--        <p class="card-header-item">Jan 2024</p>-->
<!--        <p class="card-footer-item">Jan 1, 2024</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>New paper introducing <a href="https://www.biorxiv.org/content/10.1101/2024.01.03.573985">PDGrapher for combinatorial prediction of chemical and genetic perturbations</a> using causally-inspired neural networks.</p>

</p>-->
            <p>New paper introducing <a href="https://www.biorxiv.org/content/10.1101/2024.01.03.573985">PDGrapher for combinatorial prediction of chemical and genetic perturbations</a> using causally-inspired neural networks.</p>


<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2024/01/01/PDGrapher/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Jan 1, 2024</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2023/11/23/OpenPositions/">Next Generation of Therapeutics Commons</a>-->
        <p class="card-header-title">Nov 2023: &nbsp; <span class="has-text-primary">Next Generation of Therapeutics Commons</span></p>
<!--        <p class="card-header-item">Nov 2023</p>-->
<!--        <p class="card-footer-item">Nov 23, 2023</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>We are building the next generation of <a href="https://tdcommons.ai/">Therapeutics Commons</a>! We are seeking <a href="/postdoc-TDC/">outstanding fellows who will lead AI research to advance molecular drug design and clinical drug development.</a></p>

</p>-->
            <p>We are building the next generation of <a href="https://tdcommons.ai/">Therapeutics Commons</a>! We are seeking <a href="/postdoc-TDC/">outstanding fellows who will lead AI research to advance molecular drug design and clinical drug development.</a></p>


<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2023/11/23/OpenPositions/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Nov 23, 2023</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2023/10/24/SBDD/">Structure-Based Drug Design</a>-->
        <p class="card-header-title">Oct 2023: &nbsp; <span class="has-text-primary">Structure-Based Drug Design</span></p>
<!--        <p class="card-header-item">Oct 2023</p>-->
<!--        <p class="card-footer-item">Oct 24, 2023</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p><a href="https://arxiv.org/abs/2306.11768">Geometric deep learning has emerged as a valuable tool</a> for structure-based drug design, to generate and refine biomolecules by leveraging detailed three-dimensional geometric and molecular interaction information.</p>

</p>-->
            <p><a href="https://arxiv.org/abs/2306.11768">Geometric deep learning has emerged as a valuable tool</a> for structure-based drug design, to generate and refine biomolecules by leveraging detailed three-dimensional geometric and molecular interaction information.</p>


<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2023/10/24/SBDD/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Oct 24, 2023</p>-->
<!--    </footer>-->
</div>
    </div>
    

    <div class="column is-12">
        <div class="card">
            <header class="card-header">
        <!--        <a class="card-header-title" href=""></a>-->
            <p class="card-header-title"><span class="has-text-primary">Tweets</span></p>
        <!--        <p class="card-header-item"></p>-->
        <!--        <p class="card-footer-item"></p>-->
            </header>
        <div class="card-content">
        <!--        <div class="content">-->
        <!--            -->
        <!--            <p></p>-->
        <a class="twitter-timeline" data-width="500" data-height="500" data-theme="light" href="https://twitter.com/marinkazitnik?ref_src=twsrc%5Etfw" data-chrome="noheader transparent nofooter nofooter noborders">Tweets by marinkazitnik</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
        <!--        </div>-->
        <!--        <div class="has-text-centered">-->
        <!--            <a href="" class="button is-primary">Read more</a>-->
        <!--        </div>-->
            </div>
        </div>
    </div>
</div>




                </div>
                
            </div>
        </div>
    </section>
    
    <script src="/assets/js/app.js" type="text/javascript"></script><!-- footer scripts -->
<div style="background-color:#A41034">
        <div class="content is-normal has-text-centered">
            <p style="color:white;padding-top:20px;padding-bottom:20px;"><a href="https://scholar.harvard.edu/marinka" style="color:white"><b>Zitnik Lab</b></a>
                &nbsp;&middot;&nbsp; <a href="#" style="color:white"><b>Artificial Intelligence in Medicine and Science</b></a>
                &nbsp;&middot;&nbsp; <a href="https://harvard.edu" style="color:white"><b>Harvard</b></a>
                &nbsp;&middot;&nbsp; <a href="https://dbmi.hms.harvard.edu/" style="color:white"><b>Department of Biomedical Informatics</b></a></p>
        </div>
</div></body>
</html>

