
<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Datasets - Zitnik Lab</title>
    <link rel="stylesheet" href="/assets/css/app.css">
    <link rel="shortcut icon" type="image/png"
           href="/favicon.png" 
    />
    <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>
    <!-- Begin Jekyll SEO tag v2.6.1 -->
<title>Datasets | Zitnik Lab</title>
<meta name="generator" content="Jekyll v3.8.6" />
<meta property="og:title" content="Datasets" />
<meta name="author" content="Marinka Zitnik" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Artificial Intelligence (AI), Medicine, Science, and Drug Discovery at Harvard" />
<meta property="og:description" content="Artificial Intelligence (AI), Medicine, Science, and Drug Discovery at Harvard" />
<link rel="canonical" href="https://zitniklab.hms.harvard.edu/data/" />
<meta property="og:url" content="https://zitniklab.hms.harvard.edu/data/" />
<meta property="og:site_name" content="Zitnik Lab" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="Datasets" />
<meta name="twitter:site" content="@marinkazitnik" />
<meta name="twitter:creator" content="@Marinka Zitnik" />
<script type="application/ld+json">
{"url":"https://zitniklab.hms.harvard.edu/data/","author":{"@type":"Person","name":"Marinka Zitnik"},"headline":"Datasets","description":"Artificial Intelligence (AI), Medicine, Science, and Drug Discovery at Harvard","@type":"WebPage","@context":"https://schema.org"}</script>
<!-- End Jekyll SEO tag -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-162129505-1"></script>
<script>
  window['ga-disable-UA-162129505-1'] = window.doNotTrack === "1" || navigator.doNotTrack === "1" || navigator.doNotTrack === "yes" || navigator.msDoNotTrack === "1";
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());
  gtag('config', 'UA-162129505-1');
</script><!-- head scripts --></head>

  <body>
    
<nav class="navbar is-primary" >
    <div class="container">
        <div class="navbar-brand">
            <a href="/" class="navbar-item"><b>
                Zitnik Lab
            </b></a>
            <a role="button" class="navbar-burger burger" aria-label="menu" aria-expanded="false" data-target="navMenu">
                <span aria-hidden="true"></span>
                <span aria-hidden="true"></span>
                <span aria-hidden="true"></span>
            </a>
        </div>
        <div class="navbar-menu" id="navMenu">
            <div class="navbar-start">
<!--                <a href="/" class="navbar-item "><b>Home</b></a>-->
                
                
                    
                    <div class="navbar-item has-dropdown is-hoverable">
                        <a href="/#" class="navbar-link "><b>About</b></a>
                        <div class="navbar-dropdown">
                            
                            <a href="/bio/" class="navbar-item "><b>Bio</b></a>
                            
                            <a href="/contact/" class="navbar-item "><b>Contact</b></a>
                            
                            <a href="/talks/" class="navbar-item "><b>Recent Talks</b></a>
                            
                        </div>
                    </div>
                    
                
                    
                <a href="/research/" class="navbar-item "><b>Research</b></a>
                    
                
                    
                <a href="/publications/" class="navbar-item "><b>Publications</b></a>
                    
                
                    
                <a href="/people/" class="navbar-item "><b>Members</b></a>
                    
                
                    
                <a href="/meetings/" class="navbar-item "><b>Education</b></a>
                    
                
                    
                <a href="/DMAI/" class="navbar-item "><b>DMAI</b></a>
                    
                
                    
                <a href="/data/" class="navbar-item is-active"><b>Datasets</b></a>
                    
                
                    
                <a href="/software/" class="navbar-item "><b>ML Tools</b></a>
                    
                
                    
                <a href="https://zitniklab.hms.harvard.edu/TDC/" class="navbar-item "><b>TDC</b></a>
                    
                
                    
                <a href="/news/" class="navbar-item "><b>News</b></a>
                    
                
                    
                <a href="/jobs/" class="navbar-item "><b>Join Us</b></a>
                    
                
                
            </div>
        </div>
    </div>
</nav>

    
        <section class="hero  is-medium  is-bold is-primary"  style="background: url('/hero.jpg') no-repeat center center; background-size: cover;" >
    <div class="hero-body">
        <div class="container">
            <p class="title is-2">Datasets</p>
            <p class="subtitle is-3"></p>
            
        </div>
    </div>
</section>
    
    


    <section class="section">
        <div class="container">
            <div class="columns">
                
                <div class="column is-8">
                    
                    
    

    <div class="content">
        <p>
</p>
    </div>

    
        <section class="showcase">
<!--            <figure class="image  is-16by9 ">-->
<!--                <img src="" />-->
<!--            </figure>-->
            <div class="showcase-content">
                <div class="columns is-centered">
                    <div class="column is-8-desktop is-12-tablet">
                        <p class="title">PrimeKG</p>
                        <p class="subtitle">Precision Medicine Oriented Knowledge Graph</p>

                        
                        
                        <div class="content">
                            <p><p>PrimeKG is a precision medicine-oriented knowledge graph that provides a holistic view of diseases. It integrates 20 high-quality resources to describe 17,080 diseases with 5,050,249 relationships representing ten major biological scales, including disease-associated protein perturbations, biological processes and pathways, anatomical and phenotypic scale, and the entire range of approved and experimental drugs with their therapeutic action.</p>

<p>PrimeKG supports drug-disease prediction by including an abundance of ’indications’, ’contradictions’ and ’off-label use’ edges, which are usually missing in other knowledge graphs. We accompany PrimeKG’s graph structure with text descriptions of clinical guidelines for drugs and diseases to enable multi-modal analyses.</p>
</p>
                        </div>

                        
                        

                        

                        
                        <a href="https://zitniklab.hms.harvard.edu/projects/PrimeKG" class="button is-primary">
                            View the PrimeKG Website
                        </a>
                        
                        
                                                
                    </div>
                </div>

            </div>
        </section>
    
        <section class="showcase">
<!--            <figure class="image  is-16by9 ">-->
<!--                <img src="" />-->
<!--            </figure>-->
            <div class="showcase-content">
                <div class="columns is-centered">
                    <div class="column is-8-desktop is-12-tablet">
                        <p class="title">GraphXAI</p>
                        <p class="subtitle">Evaluating Explainability for Graph Neural Networks</p>

                        
                        
                        <div class="content">
                            <p><p>GraphXAI is a resource to systematically evaluate and benchmark the quality of GNN explanations. A key component is a novel and flexible synthetic dataset generator called ShapeGGen that automatically generates a variety of benchmark datasets (e.g., varying graph sizes, degree distributions, homophilic vs. heterophilic graphs) together with ground-truth explanations that address all known pitfalls of explainability methods.</p>
</p>
                        </div>

                        
                        

                        

                        
                        <a href="https://zitniklab.hms.harvard.edu/projects/GraphXAI/" class="button is-primary">
                            View the GraphXAI Website
                        </a>
                        
                        
                                                
                    </div>
                </div>

            </div>
        </section>
    
        <section class="showcase">
<!--            <figure class="image  is-16by9 ">-->
<!--                <img src="" />-->
<!--            </figure>-->
            <div class="showcase-content">
                <div class="columns is-centered">
                    <div class="column is-8-desktop is-12-tablet">
                        <p class="title">Physical Activity Monitoring Dataset</p>
                        <p class="subtitle">Dataset for Irregular Time Series Research</p>

                        
                        
                        <div class="content">
                            <p><p>We are developing representation learning techniques for complex time series dataset. In the <a href="https://zitniklab.hms.harvard.edu/projects/Raindrop/">Raindrop study (ICLR’22)</a>, we introduced a graph-guided network for irregularly sampled multivariate time series. The study includes a processed sensor dataset recording daily living activities of individuals.</p>
</p>
                        </div>

                        
                        

                        

                        
                        <a href="https://doi.org/10.6084/m9.figshare.19514347" class="button is-primary">
                            View the Physical Activity Monitoring Dataset
                        </a>
                        
                        
                                                
                    </div>
                </div>

            </div>
        </section>
    
        <section class="showcase">
<!--            <figure class="image  is-16by9 ">-->
<!--                <img src="" />-->
<!--            </figure>-->
            <div class="showcase-content">
                <div class="columns is-centered">
                    <div class="column is-8-desktop is-12-tablet">
                        <p class="title">Population-Scale Patient Safety Dataset</p>
                        <p class="subtitle">Adverse Events of Medications across Patient Groups and the Entire Range of Human Diseases and Approved Drugs</p>

                        
                        
                        <div class="content">
                            <p><p>We present a comprehensive catalog of 10,443,476 adverse event reports (involving 19,193 adverse events and 3,624 drugs) from the U.S. Food and Drug Administration (FDA) Adverse Event Reporting System (FAERS), collected from January 2013 to September 2020. The new resource can help discover relationships between drugs and safety events, especially in cases of rare events and effects within population subgroups that differ in their risks of specific clinical outcomes and are disproportionately affected by preventable inequities.</p>
</p>
                        </div>

                        
                        

                        

                        
                        <a href="https://zitniklab.hms.harvard.edu/projects/patient-safety/" class="button is-primary">
                            View the Patient Safety Dataset
                        </a>
                        
                        
                                                
                    </div>
                </div>

            </div>
        </section>
    
        <section class="showcase">
<!--            <figure class="image  is-16by9 ">-->
<!--                <img src="" />-->
<!--            </figure>-->
            <div class="showcase-content">
                <div class="columns is-centered">
                    <div class="column is-8-desktop is-12-tablet">
                        <p class="title">Subgraph Datasets</p>
                        <p class="subtitle">Datasets for Subgraph Representation Learning Research</p>

                        
                        
                        <div class="content">
                            <p><p>We design novel synthetic and real-world social and biological datasets, consisting of underlying base graphs and many labeled subgraphs. These datasets are ready to be used for benchmarking, systematic model evaluation and comparison.</p>
</p>
                        </div>

                        
                        

                        

                        
                        <a href="https://zitniklab.hms.harvard.edu/projects/SubGNN/" class="button is-primary">
                            View the SubGNN Website
                        </a>
                        
                        
                                                
                    </div>
                </div>

            </div>
        </section>
    
        <section class="showcase">
<!--            <figure class="image  is-16by9 ">-->
<!--                <img src="" />-->
<!--            </figure>-->
            <div class="showcase-content">
                <div class="columns is-centered">
                    <div class="column is-8-desktop is-12-tablet">
                        <p class="title">Therapeutics Data Commons</p>
                        <p class="subtitle">Machine Learning Datasets and Tasks for Drug Discovery and Development</p>

                        
                        
                        <div class="content">
                            <p><p>TDC is the first unifying framework to systematically access and evaluate machine learning across the entire range of therapeutics.</p>

<p>At its core, TDC is a collection of AI/ML-ready datasets and learning tasks to serve as a meeting point for domain and ML scientists. TDC also provides an ecosystem of tools, libraries, leaderboards, and community resources, including data functions, strategies for systematic model evaluation, meaningful data splits, data processors, and molecule generation oracles. All datasets and learning tasks are integrated and accessible via an open-source library.</p>
</p>
                        </div>

                        
                        

                        

                        
                        <a href="https://zitniklab.hms.harvard.edu/TDC" class="button is-primary">
                            View the TDC Website
                        </a>
                        
                        
                                                
                    </div>
                </div>

            </div>
        </section>
    
        <section class="showcase">
<!--            <figure class="image  is-16by9 ">-->
<!--                <img src="" />-->
<!--            </figure>-->
            <div class="showcase-content">
                <div class="columns is-centered">
                    <div class="column is-8-desktop is-12-tablet">
                        <p class="title">BioSNAP</p>
                        <p class="subtitle">Stanford Biomedical Network Dataset Collection</p>

                        
                        
                        <div class="content">
                            <p><p>BioSNAP is a collection diverse biomedical networks, inclusing protein-protein interaction networks, single-cell similarity networks, drug-drug interaction networks.</p>

<p>BioSNAP datasets contain metadata on graphs and node features, and can be easily linked to external repositories of biological knowledge.</p>
</p>
                        </div>

                        
                        

                        

                        
                        <a href="http://snap.stanford.edu/biodata/" class="button is-primary">
                            View the BioSNAP Website
                        </a>
                        
                        
                                                
                    </div>
                </div>

            </div>
        </section>
    
        <section class="showcase">
<!--            <figure class="image  is-16by9 ">-->
<!--                <img src="" />-->
<!--            </figure>-->
            <div class="showcase-content">
                <div class="columns is-centered">
                    <div class="column is-8-desktop is-12-tablet">
                        <p class="title">Fair Graph Datasets</p>
                        <p class="subtitle">Graph datasets comprising of high-stakes decisions in criminal justice and financial lending domains</p>

                        
                        
                        <div class="content">
                            <p><p>Graph datasets comprise of critical decisions in criminal justice (if a defendant should be released on bail)
and financial lending (if an individual should be given loan) domains. These attributed graphs contain sensitive/protected attributes,
which makes them suitable for studying algorithmic fairness.</p>
</p>
                        </div>

                        
                        

                        

                        
                        <a href="https://zitniklab.hms.harvard.edu/projects/NIFTY/" class="button is-primary">
                            View the NIFTY website
                        </a>
                        
                        
                                                
                    </div>
                </div>

            </div>
        </section>
    
        <section class="showcase">
<!--            <figure class="image  is-16by9 ">-->
<!--                <img src="" />-->
<!--            </figure>-->
            <div class="showcase-content">
                <div class="columns is-centered">
                    <div class="column is-8-desktop is-12-tablet">
                        <p class="title">OGB</p>
                        <p class="subtitle">The Open Graph Benchmark</p>

                        
                        
                        <div class="content">
                            <p><p>OGB is a collection of benchmark datasets, data loaders, and evaluators for graph machine learning. Datasets cover a variety of graph machine learning tasks and real-world applications.</p>

<p>The OGB data loaders are fully compatible with popular graph deep learning frameworks, including <a href="https://pytorch-geometric.readthedocs.io/">Pytorch Geometric</a> and <a href="https://www.dgl.ai/">DGL</a>. They provide automatic dataset downloading, standardized dataset splits, and unified performance evaluation.</p>
</p>
                        </div>

                        
                        

                        

                        
                        <a href="https://ogb.stanford.edu/" class="button is-primary">
                            View the OGB Website
                        </a>
                        
                        
                                                
                    </div>
                </div>

            </div>
        </section>
    
        <section class="showcase">
<!--            <figure class="image  is-16by9 ">-->
<!--                <img src="" />-->
<!--            </figure>-->
            <div class="showcase-content">
                <div class="columns is-centered">
                    <div class="column is-8-desktop is-12-tablet">
                        <p class="title">Disease pathways</p>
                        <p class="subtitle">Disease pathways overlaid on the human interactome</p>

                        
                        
                        <div class="content">
                            <p>
</p>
                        </div>

                        
                        

                        

                        
                        <a href="http://snap.stanford.edu/pathways/" class="button is-primary">
                            View Disease Pathway Dataset
                        </a>
                        
                        
                                                
                    </div>
                </div>

            </div>
        </section>
    
        <section class="showcase">
<!--            <figure class="image  is-16by9 ">-->
<!--                <img src="" />-->
<!--            </figure>-->
            <div class="showcase-content">
                <div class="columns is-centered">
                    <div class="column is-8-desktop is-12-tablet">
                        <p class="title">Multimodal cancer network</p>
                        <p class="subtitle">Multimodal network centered on genes frequently mutated in cancer patients</p>

                        
                        
                        <div class="content">
                            <p><p>The multimodal cancer network integrates information on chemicals, diseases, molecular functions, genes, and protein.</p>

<p>The dataset has 21 types of biologically meaningful associations (edge types): chemical-chemical, chemical-protein, disease-chemical, disease-disease, disease-function, disease-gene, function-function, gene-gene (split into 6 edge types by interaction type), gene-protein, protein-function, and protein-protein interactions.</p>

<p>The network has 20 K nodes and 3.4 M edges.</p>
</p>
                        </div>

                        
                        

                        

                        
                        <a href="http://snap.stanford.edu/mambo/" class="button is-primary">
                            View the Multimodal Cancer Network
                        </a>
                        
                        
                                                
                    </div>
                </div>

            </div>
        </section>
    
        <section class="showcase">
<!--            <figure class="image  is-16by9 ">-->
<!--                <img src="" />-->
<!--            </figure>-->
            <div class="showcase-content">
                <div class="columns is-centered">
                    <div class="column is-8-desktop is-12-tablet">
                        <p class="title">Giga-scale biological network</p>
                        <p class="subtitle"></p>

                        
                        
                        <div class="content">
                            <p><p>The giga-scale biological network is one of the largest networks ever constructed in biology. The network integrates protein and genetic interaction data from more than two thousand species.</p>

<p>The network has 10 M nodes and 2.3 B edges.</p>
</p>
                        </div>

                        
                        

                        

                        
                        <a href="http://snap.stanford.edu/mambo/" class="button is-primary">
                            View the Giga-Scale Biological Network
                        </a>
                        
                        
                                                
                    </div>
                </div>

            </div>
        </section>
    
        <section class="showcase">
<!--            <figure class="image  is-16by9 ">-->
<!--                <img src="" />-->
<!--            </figure>-->
            <div class="showcase-content">
                <div class="columns is-centered">
                    <div class="column is-8-desktop is-12-tablet">
                        <p class="title">Tree of life</p>
                        <p class="subtitle">Protein interactomes across the tree of life</p>

                        
                        
                        <div class="content">
                            <p><p>The dataset contains protein interactomes from 1,840 species across the tree of life. The dataset contains rich metadata about proteins, including their homology relationships</p>

<p>The dataset also contains metadata about species, including taxonomy of species, phylogenetic reltionships, and ecological information on environments and habitats in which species live.</p>
</p>
                        </div>

                        
                        

                        

                        
                        <a href="https://snap.stanford.edu/tree-of-life/" class="button is-primary">
                            View the Tree of Life dataset
                        </a>
                        
                        
                                                
                    </div>
                </div>

            </div>
        </section>
    
        <section class="showcase">
<!--            <figure class="image  is-16by9 ">-->
<!--                <img src="" />-->
<!--            </figure>-->
            <div class="showcase-content">
                <div class="columns is-centered">
                    <div class="column is-8-desktop is-12-tablet">
                        <p class="title">Polypharmacy network</p>
                        <p class="subtitle">Network of drugs, proteins, and side effects</p>

                        
                        
                        <div class="content">
                            <p><p>The polypharmacy network is a highly multi-relational network, consisting of protein-protein interactions, drug-protein targets, and drug-drug interactions encoded by polypharmacy side effects.</p>

<p>The network has 20 K nodes and 5 M edges, which are split into 1 K distinct edge types.</p>
</p>
                        </div>

                        
                        

                        

                        
                        <a href="http://snap.stanford.edu/decagon/" class="button is-primary">
                            View the Polypharmacy Network
                        </a>
                        
                        
                                                
                    </div>
                </div>

            </div>
        </section>
    
        <section class="showcase">
<!--            <figure class="image  is-16by9 ">-->
<!--                <img src="" />-->
<!--            </figure>-->
            <div class="showcase-content">
                <div class="columns is-centered">
                    <div class="column is-8-desktop is-12-tablet">
                        <p class="title">Tissue-specific protein dataset</p>
                        <p class="subtitle"></p>

                        
                        
                        <div class="content">
                            <p><p>The dataset contains protein-protein interaction networks specific to 107 human tissues, a tissue hierarchy of anatomical relationships between tissues, and tissue-specific gene-function annotations.</p>
</p>
                        </div>

                        
                        

                        

                        
                        <a href="http://snap.stanford.edu/ohmnet/" class="button is-primary">
                            View the Tissue-Specific Protein Dataset
                        </a>
                        
                        
                                                
                    </div>
                </div>

            </div>
        </section>
    
        <section class="showcase">
<!--            <figure class="image  is-16by9 ">-->
<!--                <img src="" />-->
<!--            </figure>-->
            <div class="showcase-content">
                <div class="columns is-centered">
                    <div class="column is-8-desktop is-12-tablet">
                        <p class="title">Human knowledge network</p>
                        <p class="subtitle"></p>

                        
                        
                        <div class="content">
                            <p><p>The human knowledge network contains interactions between proteins, diseases, biological processes, side effects, and drugs.</p>

<p>The network has 98 K nodes and 8 M edges, which are split into 42 distinct types of biologically relevant molecular interactions.</p>
</p>
                        </div>

                        
                        

                        

                        
                        <a href="http://snap.stanford.edu/nqe/" class="button is-primary">
                            View the Human Knowledge Network
                        </a>
                        
                        
                                                
                    </div>
                </div>

            </div>
        </section>
    

                    
<div class="content">
    

</div>
                </div>
                
                <div class="column is-4-desktop is-12-tablet">
                    <p class="title is-4">Latest News</p>

<div class="columns is-multiline">
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/12/14/SHEPHERD/">Can we shorten rare disease diagnostic odyssey?</a>-->
        <p class="card-header-title">Dec 2022: &nbsp; <span class="has-text-primary">Can we shorten rare disease diagnostic odyssey?</span></p>
<!--        <p class="card-header-item">Dec 2022</p>-->
<!--        <p class="card-footer-item">Dec 14, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>New preprint! <a href="https://www.medrxiv.org/content/10.1101/2022.12.07.22283238">Geometric deep learning for diagnosing patients with rare genetic diseases.</a> Implications for the use of deep learning on datasets with very few labels. Thankful for this collaboration with <a href="https://zaklab.org/">Zak Lab.</a> <a href="/projects/SHEPHERD/">Project website.</a></p>
</p>-->
            <p>New preprint! <a href="https://www.medrxiv.org/content/10.1101/2022.12.07.22283238">Geometric deep learning for diagnosing patients with rare genetic diseases.</a> Implications for the use of deep learning on datasets with very few labels. Thankful for this collaboration with <a href="https://zaklab.org/">Zak Lab.</a> <a href="/projects/SHEPHERD/">Project website.</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/12/14/SHEPHERD/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Dec 14, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/11/15/HarvardMedicineTDC/">Can AI transform the way we discover new drugs?</a>-->
        <p class="card-header-title">Nov 2022: &nbsp; <span class="has-text-primary">Can AI transform the way we discover new drugs?</span></p>
<!--        <p class="card-header-item">Nov 2022</p>-->
<!--        <p class="card-footer-item">Nov 15, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Our <a href="https://hms.harvard.edu/news/can-ai-transform-way-we-discover-new-drugs">conversation with Harvard Medicine News</a> highlights recent developments and new features in <a href="https://tdcommons.ai/">Therapeutics Data Commons.</a></p>
</p>-->
            <p>Our <a href="https://hms.harvard.edu/news/can-ai-transform-way-we-discover-new-drugs">conversation with Harvard Medicine News</a> highlights recent developments and new features in <a href="https://tdcommons.ai/">Therapeutics Data Commons.</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/11/15/HarvardMedicineTDC/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Nov 15, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/10/31/GRL-NatureBiomedEng/">New Paper in Nature Biomedical Engineering</a>-->
        <p class="card-header-title">Oct 2022: &nbsp; <span class="has-text-primary">New Paper in Nature Biomedical Engineering</span></p>
<!--        <p class="card-header-item">Oct 2022</p>-->
<!--        <p class="card-footer-item">Oct 31, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>New paper on <a href="https://www.nature.com/articles/s41551-022-00942-x">graph representation learning in biomedicine and healthcare</a> published in Nature Biomedical Engineering.</p>
</p>-->
            <p>New paper on <a href="https://www.nature.com/articles/s41551-022-00942-x">graph representation learning in biomedicine and healthcare</a> published in Nature Biomedical Engineering.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/10/31/GRL-NatureBiomedEng/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Oct 31, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/09/21/TDC-NatureChemBio/">New Paper in Nature Chemical Biology</a>-->
        <p class="card-header-title">Sep 2022: &nbsp; <span class="has-text-primary">New Paper in Nature Chemical Biology</span></p>
<!--        <p class="card-header-item">Sep 2022</p>-->
<!--        <p class="card-footer-item">Sep 21, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Our paper on <a href="https://www.nature.com/articles/s41589-022-01131-2">artificial intelligence foundation for therapeutic science</a> is published in Nature Chemical Biology.</p>
</p>-->
            <p>Our paper on <a href="https://www.nature.com/articles/s41589-022-01131-2">artificial intelligence foundation for therapeutic science</a> is published in Nature Chemical Biology.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/09/21/TDC-NatureChemBio/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Sep 21, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/09/14/TF-C-NeurIPS/">Self-Supervised Pre-Training at NeurIPS 2022</a>-->
        <p class="card-header-title">Sep 2022: &nbsp; <span class="has-text-primary">Self-Supervised Pre-Training at NeurIPS 2022</span></p>
<!--        <p class="card-header-item">Sep 2022</p>-->
<!--        <p class="card-footer-item">Sep 14, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>New paper on self-supervised contrastive pre-training accepted at <a href="https://nips.cc/Conferences/2022">NeurIPS 2022.</a> <a href="https://zitniklab.hms.harvard.edu/projects/TF-C/">Project page.</a> Thankful for this collaboration with the <a href="https://www.ll.mit.edu/">Lincoln National Laboratory</a>.</p>
</p>-->
            <p>New paper on self-supervised contrastive pre-training accepted at <a href="https://nips.cc/Conferences/2022">NeurIPS 2022.</a> <a href="https://zitniklab.hms.harvard.edu/projects/TF-C/">Project page.</a> Thankful for this collaboration with the <a href="https://www.ll.mit.edu/">Lincoln National Laboratory</a>.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/09/14/TF-C-NeurIPS/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Sep 14, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/09/06/BestPaperAwardIEEEVIS/">Best Paper Honorable Mention Award at IEEE VIS</a>-->
        <p class="card-header-title">Sep 2022: &nbsp; <span class="has-text-primary">Best Paper Honorable Mention Award at IEEE VIS</span></p>
<!--        <p class="card-header-item">Sep 2022</p>-->
<!--        <p class="card-footer-item">Sep 6, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Our paper on <a href="https://osf.io/yhdpv">user-centric AI of drug repurposing</a> received the <a href="http://ieeevis.org/year/2022/info/awards/best-paper-awards">Best Paper Honorable Mention Award</a> at IEEE VIS 2022. Thankful for this collaboration with <a href="http://gehlenborglab.org/">Gehlenborg Lab.</a></p>
</p>-->
            <p>Our paper on <a href="https://osf.io/yhdpv">user-centric AI of drug repurposing</a> received the <a href="http://ieeevis.org/year/2022/info/awards/best-paper-awards">Best Paper Honorable Mention Award</a> at IEEE VIS 2022. Thankful for this collaboration with <a href="http://gehlenborglab.org/">Gehlenborg Lab.</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/09/06/BestPaperAwardIEEEVIS/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Sep 6, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/09/05/MGLPreprint/">Multimodal Representation Learning with Graphs</a>-->
        <p class="card-header-title">Sep 2022: &nbsp; <span class="has-text-primary">Multimodal Representation Learning with Graphs</span></p>
<!--        <p class="card-header-item">Sep 2022</p>-->
<!--        <p class="card-footer-item">Sep 5, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>New preprint! We present the <a href="https://arxiv.org/abs/2209.03299">blueprint for graph-centric multimodal learning.</a></p>
</p>-->
            <p>New preprint! We present the <a href="https://arxiv.org/abs/2209.03299">blueprint for graph-centric multimodal learning.</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/09/05/MGLPreprint/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Sep 5, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/08/24/ISMB22-Tutorial-Recording/">On Graph AI for Precision Medicine</a>-->
        <p class="card-header-title">Aug 2022: &nbsp; <span class="has-text-primary">On Graph AI for Precision Medicine</span></p>
<!--        <p class="card-header-item">Aug 2022</p>-->
<!--        <p class="card-footer-item">Aug 24, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>The recording of our <a href="https://zitniklab.hms.harvard.edu/biomedgraphml/">tutorial on using graph AI to advance precision medicine</a> is available. <a href="https://iscb.junolive.co/ismb2022/library/search/OnDemand_ismb2022_tutorial_vt4">Tune into four hours of interactive lectures</a> about state-of-the-art graph AI methods and applications in precision medicine.</p>
</p>-->
            <p>The recording of our <a href="https://zitniklab.hms.harvard.edu/biomedgraphml/">tutorial on using graph AI to advance precision medicine</a> is available. <a href="https://iscb.junolive.co/ismb2022/library/search/OnDemand_ismb2022_tutorial_vt4">Tune into four hours of interactive lectures</a> about state-of-the-art graph AI methods and applications in precision medicine.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/08/24/ISMB22-Tutorial-Recording/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Aug 24, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/08/22/GraphXAI/">Evaluating Explainability for GNNs</a>-->
        <p class="card-header-title">Aug 2022: &nbsp; <span class="has-text-primary">Evaluating Explainability for GNNs</span></p>
<!--        <p class="card-header-item">Aug 2022</p>-->
<!--        <p class="card-footer-item">Aug 22, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>New preprint! We introduce a resource for <a href="https://arxiv.org/abs/2208.09339">broad evaluation of the quality and reliability of GNN explanations</a>, addressing challenges and providing solutions for GNN explainability. <a href="https://zitniklab.hms.harvard.edu/projects/GraphXAI/">Project website.</a></p>
</p>-->
            <p>New preprint! We introduce a resource for <a href="https://arxiv.org/abs/2208.09339">broad evaluation of the quality and reliability of GNN explanations</a>, addressing challenges and providing solutions for GNN explainability. <a href="https://zitniklab.hms.harvard.edu/projects/GraphXAI/">Project website.</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/08/22/GraphXAI/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Aug 22, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/07/16/NeurIPS-NewFrontiersGraphs/">New Frontiers in Graph Learning at NeurIPS</a>-->
        <p class="card-header-title">Jul 2022: &nbsp; <span class="has-text-primary">New Frontiers in Graph Learning at NeurIPS</span></p>
<!--        <p class="card-header-item">Jul 2022</p>-->
<!--        <p class="card-footer-item">Jul 16, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Excited to organize the <a href="https://glfrontiers.github.io/">New Frontiers in Graph Learning</a> workshop at NeurIPS.</p>
</p>-->
            <p>Excited to organize the <a href="https://glfrontiers.github.io/">New Frontiers in Graph Learning</a> workshop at NeurIPS.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/07/16/NeurIPS-NewFrontiersGraphs/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Jul 16, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/07/16/NeurIPS-AI4Science/">AI4Science at NeurIPS</a>-->
        <p class="card-header-title">Jul 2022: &nbsp; <span class="has-text-primary">AI4Science at NeurIPS</span></p>
<!--        <p class="card-header-item">Jul 2022</p>-->
<!--        <p class="card-footer-item">Jul 16, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>We are excited to host the <a href="https://ai4sciencecommunity.github.io/index.html">AI4Science meeting</a> at NeurIPS discussing AI-driven scientific discovery, implementation and verification of AI in science, the influence AI has on the conduct of science, and more.</p>
</p>-->
            <p>We are excited to host the <a href="https://ai4sciencecommunity.github.io/index.html">AI4Science meeting</a> at NeurIPS discussing AI-driven scientific discovery, implementation and verification of AI in science, the influence AI has on the conduct of science, and more.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/07/16/NeurIPS-AI4Science/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Jul 16, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/07/09/ISMB-Tutorial-Materials/">Graph AI for Precision Medicine at ISMB</a>-->
        <p class="card-header-title">Jul 2022: &nbsp; <span class="has-text-primary">Graph AI for Precision Medicine at ISMB</span></p>
<!--        <p class="card-header-item">Jul 2022</p>-->
<!--        <p class="card-footer-item">Jul 9, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Check out our half-day tutorial with resources on <a href="/biomedgraphml/">methods and applications in graph representation learning for precision medicine.</a></p>
</p>-->
            <p>Check out our half-day tutorial with resources on <a href="/biomedgraphml/">methods and applications in graph representation learning for precision medicine.</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/07/09/ISMB-Tutorial-Materials/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Jul 9, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/07/04/SummerInterns/">Welcoming Fellows and Summer Students</a>-->
        <p class="card-header-title">Jul 2022: &nbsp; <span class="has-text-primary">Welcoming Fellows and Summer Students</span></p>
<!--        <p class="card-header-item">Jul 2022</p>-->
<!--        <p class="card-footer-item">Jul 4, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Welcoming a research fellow Julia Balla and three Summer students, Nicholas Ho, Satvik Tripathi, and Isuru Herath.</p>
</p>-->
            <p>Welcoming a research fellow Julia Balla and three Summer students, Nicholas Ho, Satvik Tripathi, and Isuru Herath.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/07/04/SummerInterns/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Jul 4, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/06/20/ContrastivePreTrainingTimeSeries/">Broadly Generalizable Pre-Training Approach</a>-->
        <p class="card-header-title">Jun 2022: &nbsp; <span class="has-text-primary">Broadly Generalizable Pre-Training Approach</span></p>
<!--        <p class="card-header-item">Jun 2022</p>-->
<!--        <p class="card-footer-item">Jun 20, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Excited to share a <a href="https://arxiv.org/abs/2206.08496">preprint on self-supervised method for pre-training</a>. <a href="/projects/TF-C">Project website</a> with evaluation on eight datasets, including electrodiagnostic testing, human daily activity recognition, and health state monitoring.</p>
</p>-->
            <p>Excited to share a <a href="https://arxiv.org/abs/2206.08496">preprint on self-supervised method for pre-training</a>. <a href="/projects/TF-C">Project website</a> with evaluation on eight datasets, including electrodiagnostic testing, human daily activity recognition, and health state monitoring.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/06/20/ContrastivePreTrainingTimeSeries/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Jun 20, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/06/07/NewPostdocs/">Welcoming New Postdocs</a>-->
        <p class="card-header-title">Jun 2022: &nbsp; <span class="has-text-primary">Welcoming New Postdocs</span></p>
<!--        <p class="card-header-item">Jun 2022</p>-->
<!--        <p class="card-footer-item">Jun 7, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Excited to welcome George Dasoulas and Huan He, new postdocs joining us this Summer.</p>
</p>-->
            <p>Excited to welcome George Dasoulas and Huan He, new postdocs joining us this Summer.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/06/07/NewPostdocs/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Jun 7, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/05/20/DasoulasHDSIFellow/">George Named the 2022 Wojcicki Troper Fellow</a>-->
        <p class="card-header-title">May 2022: &nbsp; <span class="has-text-primary">George Named the 2022 Wojcicki Troper Fellow</span></p>
<!--        <p class="card-header-item">May 2022</p>-->
<!--        <p class="card-footer-item">May 20, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Congratulations to <a href="https://www.hdsi.news/post/the-2022-harvard-data-science-initiative-postdoctoral-fellows">George Dasoulas, our incoming postdoctoral fellow, on being named the 2022 Wojcicki Troper HDSI Postdoctoral Fellow</a>. We are delighted to welcome George in our group.</p>
</p>-->
            <p>Congratulations to <a href="https://www.hdsi.news/post/the-2022-harvard-data-science-initiative-postdoctoral-fellows">George Dasoulas, our incoming postdoctoral fellow, on being named the 2022 Wojcicki Troper HDSI Postdoctoral Fellow</a>. We are delighted to welcome George in our group.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/05/20/DasoulasHDSIFellow/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: May 20, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/05/01/PrimeKGPreprint/">New preprint on PrimeKG</a>-->
        <p class="card-header-title">May 2022: &nbsp; <span class="has-text-primary">New preprint on PrimeKG</span></p>
<!--        <p class="card-header-item">May 2022</p>-->
<!--        <p class="card-footer-item">May 1, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p><a href="https://www.biorxiv.org/content/10.1101/2022.05.01.489928">New preprint on building knowledge graphs</a> to enable precision medicine applications.</p>
</p>-->
            <p><a href="https://www.biorxiv.org/content/10.1101/2022.05.01.489928">New preprint on building knowledge graphs</a> to enable precision medicine applications.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/05/01/PrimeKGPreprint/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: May 1, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/05/01/PrimeKG/">Building KGs to Support Precision Medicine</a>-->
        <p class="card-header-title">May 2022: &nbsp; <span class="has-text-primary">Building KGs to Support Precision Medicine</span></p>
<!--        <p class="card-header-item">May 2022</p>-->
<!--        <p class="card-footer-item">May 1, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>We created <a href="https://zitniklab.hms.harvard.edu/projects/PrimeKG/">PrimeKG, a precision medicine-oriented knowledge graph to support graph AI analyses in medicine.</a></p>
</p>-->
            <p>We created <a href="https://zitniklab.hms.harvard.edu/projects/PrimeKG/">PrimeKG, a precision medicine-oriented knowledge graph to support graph AI analyses in medicine.</a></p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/05/01/PrimeKG/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: May 1, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/04/21/CellSystemsCover/">Webster on the Cover of Cell Systems</a>-->
        <p class="card-header-title">Apr 2022: &nbsp; <span class="has-text-primary">Webster on the Cover of Cell Systems</span></p>
<!--        <p class="card-header-item">Apr 2022</p>-->
<!--        <p class="card-footer-item">Apr 21, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Webster is on the <a href="/img/webster-cell-systems-cover.jpg">cover of April issue</a> of <a href="https://www.cell.com/cell-systems/issue?pii=S2405-4712(21)X0005-1">Cell Systems</a>. <a href="https://doi.org/10.1016/j.cels.2021.12.005">Webster uses cell viability changes following gene perturbation</a> to automatically learn cellular functions and pathways from data.</p>
</p>-->
            <p>Webster is on the <a href="/img/webster-cell-systems-cover.jpg">cover of April issue</a> of <a href="https://www.cell.com/cell-systems/issue?pii=S2405-4712(21)X0005-1">Cell Systems</a>. <a href="https://doi.org/10.1016/j.cels.2021.12.005">Webster uses cell viability changes following gene perturbation</a> to automatically learn cellular functions and pathways from data.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/04/21/CellSystemsCover/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Apr 21, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    
    <div class="column is-12">
        <div class="card">
    
    <header class="card-header">
<!--        <a class="card-header-title" href="/2022/04/20/NASASpaceBiology/">NASA Space Biology</a>-->
        <p class="card-header-title">Apr 2022: &nbsp; <span class="has-text-primary">NASA Space Biology</span></p>
<!--        <p class="card-header-item">Apr 2022</p>-->
<!--        <p class="card-footer-item">Apr 20, 2022</p>-->
    </header>
    
    <div class="card-content">
<!--        <div class="content">-->
<!--            -->
<!--            <p><p>Dr. Zitnik will serve on the Science Working Group at <a href="https://science.nasa.gov/">NASA Space Biology</a>.</p>
</p>-->
            <p>Dr. Zitnik will serve on the Science Working Group at <a href="https://science.nasa.gov/">NASA Space Biology</a>.</p>

<!--        </div>-->
<!--        <div class="has-text-centered">-->
<!--            <a href="/2022/04/20/NASASpaceBiology/" class="button is-primary">Read more</a>-->
<!--        </div>-->
    </div>
<!--    <footer class="card-footer">-->
<!--        <p class="card-footer-item">Published: Apr 20, 2022</p>-->
<!--    </footer>-->
</div>
    </div>
    

    <div class="column is-12">
        <div class="card">
            <header class="card-header">
        <!--        <a class="card-header-title" href=""></a>-->
            <p class="card-header-title"><span class="has-text-primary">Tweets</span></p>
        <!--        <p class="card-header-item"></p>-->
        <!--        <p class="card-footer-item"></p>-->
            </header>
        <div class="card-content">
        <!--        <div class="content">-->
        <!--            -->
        <!--            <p></p>-->
        <a class="twitter-timeline" data-width="500" data-height="500" data-theme="light" href="https://twitter.com/marinkazitnik?ref_src=twsrc%5Etfw" data-chrome="noheader transparent nofooter nofooter noborders">Tweets by marinkazitnik</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
        <!--        </div>-->
        <!--        <div class="has-text-centered">-->
        <!--            <a href="" class="button is-primary">Read more</a>-->
        <!--        </div>-->
            </div>
        </div>
    </div>
</div>




                </div>
                
            </div>
        </div>
    </section>
    
    <script src="/assets/js/app.js" type="text/javascript"></script><!-- footer scripts -->
<div style="background-color:#A41034">
        <div class="content is-normal has-text-centered">
            <p style="color:white;padding-top:20px;padding-bottom:20px;"><a href="https://scholar.harvard.edu/marinka" style="color:white"><b>Zitnik Lab</b></a>
                &nbsp;&middot;&nbsp; <a href="#" style="color:white"><b>Artificial Intelligence in Medicine and Science</b></a>
                &nbsp;&middot;&nbsp; <a href="https://harvard.edu" style="color:white"><b>Harvard</b></a>
                &nbsp;&middot;&nbsp; <a href="https://dbmi.hms.harvard.edu/" style="color:white"><b>Department of Biomedical Informatics</b></a></p>
        </div>
</div></body>
</html>

